<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Publications | Complex Disease Epigenetics Group</title> <meta name="author" content="Complex Disease Epigenetics Group"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="epigenetics, exeter, complex disease epigenetics"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css"> <script src="https://kit.fontawesome.com/cfa1627e95.js" crossorigin="anonymous"></script> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/favicon.ico"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://aspides-js.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <a href="https://medicine.exeter.ac.uk/" rel="external nofollow noopener" target="_blank"> <img src="/assets/img/uofelogo.svg" width="150px"> </a> <a href="/"> <img src="/assets/img/epigenomicslogo.png" width="150px"> </a> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About</a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">Research</a> </li> <li class="nav-item "> <a class="nav-link" href="/data/">Data</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">People</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/joinus/">Join Us</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/contactus/">Contact Us</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000886075100039" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2022.07.154" rel="external nofollow noopener" target="_blank">Long Read Transcriptome Sequencing Reveals Isoform Diversity Across Human Neurodevelopment</a></div> <div class="author"> Rosemary Bamford, Aaron R. Jeffries, Emma Walker, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Szi Kay Leung, Gina Commin, Jonathan P. Davies, Emma Dempster, Eilis Hannon, Jonathan Mill' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2022 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2022.07.154" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2022.07.154" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000886075100015" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2022.07.130" rel="external nofollow noopener" target="_blank">Leveraging a Bespoke Cortical Transcriptome to Assess The Role of Alternative Splicing in Schizophrenia</a></div> <div class="author"> Eilis Hannon, Aaron R. Jeffries, Stefania Policicchio, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Joana Viana, Szi Kay Leung, Emma Dempster, Jonathan Mill' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2022 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2022.07.130" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2022.07.130" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000886075100059" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2022.07.174" rel="external nofollow noopener" target="_blank">IDENTIFYING CELL-TYPE-SPECIFIC EPIGENETIC VARIATION IN THE CORTEX ASSOCIATED WITH SCHIZOPHRENIA</a></div> <div class="author"> Eilis Hannon, Jonathan Davies, Barry Chioza, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Stefania Policicchio, Joe Burrage, Gina Commin, Aaron R. Jeffries, Leo Schalkwyk, Emma Dempster, Jonathan Mill' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2022 </div> <div class="periodical"> World Congress of Psychiatric Genetics (WCPG), Florence, ITALY, SEP 13-17, 2022 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2022.07.174" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2022.07.174" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000860106600023" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1212/WNL.0000000000200898" rel="external nofollow noopener" target="_blank">Association of Pace of Aging Measured by Blood-Based DNA Methylation With Age-Related Cognitive Impairment and Dementia</a></div> <div class="author"> Karen Sugden, Avshalom Caspi, Maxwell L. Elliott, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Kyle J. Bourassa, Kartik Chamarti, David L. Corcoran, Ahmad R. Hariri, Renate M. Houts, Meeraj Kothari, Stephen Kritchevsky, George A. Kuchel, Jonathan S. Mill, Benjamin S. Williams, Daniel W. Belsky, Terrie E. Moffitt, Alzheimer’s Dis Neuroimaging Initi' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>NEUROLOGY</em>, Sep 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1212/WNL.0000000000200898" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1212/WNL.0000000000200898" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background and Objectives DNA methylation algorithms are increasingly used to estimate biological aging; however, how these proposed measures of whole-organism biological aging relate to aging in the brain is not known. We used data from the Alzheimer’s Disease Neuroimaging Initiative (ADNI) and the Framingham Heart Study (FHS) Offspring Cohort to test the association between blood-based DNA methylation measures of biological aging and cognitive impairment and dementia in older adults. Methods We tested 3 “generations” of DNA methylation age algorithms (first generation: Horvath and Hannum clocks; second generation: PhenoAge and GrimAge; and third generation: DunedinPACE, Dunedin Pace of Aging Calculated from the Epigenome) against the following measures of cognitive impairment in ADNI: clinical diagnosis of dementia and mild cognitive impairment, scores on Alzheimer disease (AD) / Alzheimer disease and related dementias (ADRD) screening tests (Alzheimer’s Disease Assessment Scale, Mini-Mental State Examination, and Montreal Cognitive Assessment), and scores on cognitive tests (Rey Auditory Verbal Learning Test, Logical Memory test, and Trail Making Test). In an independent replication in the FHS Offspring Cohort, we further tested the longitudinal association between the DNA methylation algorithms and the risk of developing dementia. Results In ADNI (N = 649 individuals), the first-generation (Horvath and Hannum DNA methylation age clocks) and the second-generation (PhenoAge and GrimAge) DNA methylation measures of aging were not consistently associated with measures of cognitive impairment in older adults. By contrast, a third-generation measure of biological aging, DunedinPACE, was associated with clinical diagnosis of Alzheimer disease (beta [95% CI] = 0.28 [0.08-0.47]), poorer scores on Alzheimer disease/ADRD screening tests (beta [Robust SE] = -0.10 [0.04] to 0.08[0.04]), and cognitive tests (beta [Robust SE] = -0.12 [0.04] to 0.10 [0.03]). The association between faster pace of aging, as measured by DunedinPACE, and risk of developing dementia was confirmed in a longitudinal analysis of the FHS Offspring Cohort (N = 2,264 individuals, hazard ratio [95% CI] = 1.27 [1.07-1.49]). Discussion Third-generation blood-based DNA methylation measures of aging could prove valuable for measuring differences between individuals in the rate at which they age and in their risk for cognitive decline, and for evaluating interventions to slow aging.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000858076500013" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41467-022-33394-7" rel="external nofollow noopener" target="_blank">DNA methylation signatures of Alzheimer’s disease neuropathology in the cortex are primarily driven by variation in non-neuronal cell-types</a></div> <div class="author"> Gemma Shireby, Emma L. Dempster, Stefania Policicchio, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Rebecca G. Smith, Ehsan Pishva, Barry Chioza, Jonathan P. Davies, Joe Burrage, Katie Lunnon, Dorothea Seiler Vellame, Seth Love, Alan Thomas, Keeley Brookes, Kevin Morgan, Paul Francis, Eilis Hannon, Jonathan Mill' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Sep 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41467-022-33394-7" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-022-33394-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alzheimer’s disease (AD) is a chronic neurodegenerative disease characterized by the progressive accumulation of amyloid-beta and neurofibrillary tangles of tau in the neocortex. We profiled DNA methylation in two regions of the cortex from 631 donors, performing an epigenome-wide association study of multiple measures of AD neuropathology. We meta-analyzed our results with those from previous studies of DNA methylation in AD cortex (total n = 2013 donors), identifying 334 cortical differentially methylated positions (DMPs) associated with AD pathology including methylomic variation at loci not previously implicated in dementia. We subsequently profiled DNA methylation in NeuN+ (neuronal-enriched), SOX10+ (oligodendrocyte-enriched) and NeuN-/SOX10- (microglia- and astrocyte-enriched) nuclei, finding that the majority of DMPs identified in ‘bulk’ cortex tissue reflect DNA methylation differences occurring in non-neuronal cells. Our study highlights the power of utilizing multiple measures of neuropathology to identify epigenetic signatures of AD and the importance of characterizing disease-associated variation in purified cell-types. Here the authors identify differences in cortical DNA methylation associated with Alzheimer’s disease pathology, and profiling nuclei from specific cell-types, find that most of these differences reflect variation occurring in non-neuronal cells.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000843205300009" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41467-022-32619-z" rel="external nofollow noopener" target="_blank">Epigenome-wide association study of human frontal cortex identifies differential methylation in Lewy body pathology</a></div> <div class="author"> Lasse Pihlstrom, Gemma Shireby, Hanneke Geut, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Sandra Pilar Henriksen, Annemieke J. M. Rozemuller, Jon-Anders Tunold, Eilis Hannon, Paul Francis, Alan J. Thomas, Seth Love, Jonathan Mill, Wilma D. J. Berg, Mathias Toft' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Aug 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41467-022-32619-z" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-022-32619-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Parkinson’s disease (PD) and dementia with Lewy bodies (DLB) are closely related progressive disorders with no available disease-modifying therapy, neuropathologically characterized by intraneuronal aggregates of misfolded alpha-synuclein. To explore the role of DNA methylation changes in PD and DLB pathogenesis, we performed an epigenome-wide association study (EWAS) of 322 postmortem frontal cortex samples and replicated results in an independent set of 200 donors. We report novel differentially methylated replicating loci associated with Braak Lewy body stage near TMCC2, SFMBT2, AKAP6 and PHYHIP. Differentially methylated probes were independent of known PD genetic risk alleles. Meta-analysis provided suggestive evidence for a differentially methylated locus within the chromosomal region affected by the PD-associated 22q11.2 deletion. Our findings elucidate novel disease pathways in PD and DLB and generate hypotheses for future molecular studies of Lewy body pathology. Parkinson’s disease and dementia with Lewy bodies are closely related neurodegenerative disorders, although the epigenetic similarities are not well known. Here, the authors study Lewy pathology and DNA methylation in postmortem human frontal cortex, identifying differentially methylated genomic loci.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000812900100001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1002/ajmg.b.32905" rel="external nofollow noopener" target="_blank">Functional characterization of the schizophrenia associated gene AS3MT identifies a role in neuronal development</a></div> <div class="author"> Sam J. Washer, Robert Flynn, Asami Oguro-Ando, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Eilis Hannon, Joe Burrage, Aaron Jeffries, Jonathan Mill, Emma L. Dempster' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>AMERICAN JOURNAL OF MEDICAL GENETICS PART B-NEUROPSYCHIATRIC GENETICS</em>, Jul 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/ajmg.b.32905" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/ajmg.b.32905" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Genome-wide association studies (GWAS) have identified multiple genomic regions associated with schizophrenia, although many variants reside in noncoding regions characterized by high linkage disequilibrium (LD) making the elucidation of molecular mechanisms challenging. A genomic region on chromosome 10q24 has been consistently associated with schizophrenia with risk attributed to the AS3MT gene. Although AS3MT is hypothesized to play a role in neuronal development and differentiation, work to fully understand the function of this gene has been limited. In this study we explored the function of AS3MT using a neuronal cell line (SH-SY5Y). We confirm previous findings of isoform specific expression of AS3MT during SH-SY5Y differentiation toward neuronal fates. Using CRISPR-Cas9 gene editing we generated AS3MT knockout SH-SY5Y cell lines and used RNA-seq to identify significant changes in gene expression in pathways associated with neuronal development, inflammation, extracellular matrix formation, and RNA processing, including dysregulation of other genes strongly implicated in schizophrenia. We did not observe any morphological changes in cell size and neurite length following neuronal differentiation and MAP2 immunocytochemistry. These results provide novel insights into the potential role of AS3MT in brain development and identify pathways through which genetic variation in this region may confer risk for schizophrenia.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000805861000003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neurobiolaging.2022.04.003" rel="external nofollow noopener" target="_blank">Methylome analysis of ALS patients and presymptomatic mutation carriers in blood cells</a></div> <div class="author"> Wolfgang P. Ruf, Eilis Hannon, Axel Freischmidt, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Veselin Grozdanov, David Brenner, Kathrin Mueller, Antje Knehr, Kornelia Guenther, Johannes Dorst, Ole Ammerpohl, Karin M. Danzer, Jonathan Mill, Albert C. Ludolph, Jochen H. Weishaupt' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF AGING</em>, Aug 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2022.04.003" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2022.04.003" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Amyotrophic lateral sclerosis (ALS) is a fatal motoneuron disease with a monogenic cause in approximately 10% of cases. However, familial clustering of disease without inheritance in a Mendelian manner and the broad range of phenotypes suggest the presence of epigenetic mechanisms. Hence, we performed an epigenome-wide association study on sporadic, symptomatic and presymptomatic familial ALS cases with mutations in C9ORF72 and FUS and healthy controls studying DNA methylation in blood cells. We found differentially methylated DNA positions (DMPs) and regions embedding DMPs associated with either disease status, C9ORF72 or FUS mutation status. One DMP reached methylome-wide significance and is attributed to a region encoding a long non-coding RNA (LOC389247). Furthermore, we could demonstrate co-localization of DMPs with an ALS-associated GWAS region near the SCN7A/SCN9A and XIRP2 genes. Finally, a classifier model that predicts disease status (ALS, healthy) classified all but one presymptomatic mutation carrier as healthy, suggesting that the presence of ALS symptoms rather than the presence of ALS-associated genetic mutations is associated with blood cell DNA methylation. (c) 2022 Elsevier Inc. All rights reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000781132500001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1111/ejn.15661" rel="external nofollow noopener" target="_blank">A comparison of blood and brain-derived ageing and inflammation-related DNA methylation signatures and their association with microglial burdens</a></div> <div class="author"> Anna J. Stevenson, Daniel L. McCartney, Danni A. Gadd, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Gemma Shireby, Robert F. Hillary, Declan King, Makis Tzioras, Nicola Wrobel, Sarah McCafferty, Lee Murphy, Barry W. McColl, Paul Redmond, Adele M. Taylor, Sarah E. Harris, Tom C. Russ, Andrew M. McIntosh, Jonathan Mill, Colin Smith, Ian J. Deary, Simon R. Cox, Riccardo E. Marioni, Tara L. Spires-Jones' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>EUROPEAN JOURNAL OF NEUROSCIENCE</em>, Nov 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1111/ejn.15661" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1111/ejn.15661" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Inflammation and ageing-related DNA methylation patterns in the blood have been linked to a variety of morbidities, including cognitive decline and neurodegenerative disease. However, it is unclear how these blood-based patterns relate to patterns within the brain and how each associates with central cellular profiles. In this study, we profiled DNA methylation in both the blood and in five post mortem brain regions (BA17, BA20/21, BA24, BA46 and hippocampus) in 14 individuals from the Lothian Birth Cohort 1936. Microglial burdens were additionally quantified in the same brain regions. DNA methylation signatures of five epigenetic ageing biomarkers (’epigenetic clocks’), and two inflammatory biomarkers (methylation proxies for C-reactive protein and interleukin-6) were compared across tissues and regions. Divergent associations between the inflammation and ageing signatures in the blood and brain were identified, depending on region assessed. Four out of the five assessed epigenetic age acceleration measures were found to be highest in the hippocampus (beta range = 0.83-1.14, p &lt;= 0.02). The inflammation-related DNA methylation signatures showed no clear variation across brain regions. Reactive microglial burdens were found to be highest in the hippocampus (beta = 1.32, p = 5 x 10(-4)); however, the only association identified between the blood- and brain-based methylation signatures and microglia was a significant positive association with acceleration of one epigenetic clock (termed DNAm PhenoAge) averaged over all five brain regions (beta = 0.40, p = 0.002). This work highlights a potential vulnerability of the hippocampus to epigenetic ageing and provides preliminary evidence of a relationship between DNA methylation signatures in the brain and differences in microglial burdens.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000760127200002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1126/scitranslmed.abj0264" rel="external nofollow noopener" target="_blank">Genome-wide study of DNA methylation shows alterations in metabolic, inflammatory, and cholesterol pathways in ALS</a></div> <div class="author"> Paul J. Hop, Ramona A. J. Zwamborn, Eilis Hannon, and <span class="more-authors" title="click to view 82 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '82 more authors' ? 'Gemma L. Shireby, Marta F. Nabais, Emma M. Walker, Wouter Rheenen, Joke J. F. A. Vugt, Annelot M. Dekker, Henk-Jan Westeneng, Gijs H. P. Tazelaar, Kristel R. Eijk, Matthieu Moisse, Denis Baird, Ahmad Al Khleifat, Alfredo Iacoangeli, Nicola Ticozzi, Antonia Ratti, Jonathan Cooper-Knock, Karen E. Morrison, Pamela J. Shaw, A. Nazli Basak, Adriano Chio, Andrea Calvo, Cristina Moglia, Antonio Canosa, Maura Brunetti, Maurizio Grassano, Marc Gotkine, Yossef Lerner, Michal Zabari, Patrick Vourc’h, Philippe Corcia, Philippe Couratier, Jesus S. Mora Pardina, Teresa Salas, Patrick Dion, Jay P. Ross, Robert D. Henderson, Susan Mathers, Pamela A. McCombe, Merrilee Needham, Garth Nicholson, Dominic B. Rowe, Roger Pamphlett, Karen A. Mather, Perminder S. Sachdev, Sarah Furlong, Fleur C. Garton, Anjali K. Henders, Tian Lin, Shyuan T. Ngo, Frederik J. Steyn, Leanne Wallace, Kelly L. Williams, Miguel Mitne Neto, Ruben J. Cauchi, Ian P. Blair, Matthew C. Kiernan, Vivian Drory, Monica Povedano, Mamede Carvalho, Susana Pinto, Markus Weber, Guy A. Rouleau, Vincenzo Silani, John E. Landers, Christopher E. Shaw, Peter M. Andersen, Allan F. McRae, Michael A. Es, R. Jeroen Pasterkamp, Naomi R. Wray, Russell L. McLaughlin, Orla Hardiman, Kevin P. Kenna, Ellen Tsai, Heiko Runz, Ammar Al-Chalabi, Leonard H. Berg, Philip Van Damme, Jonathan Mill, Jan H. Veldink, BIOS Consortium, Brain MEND Consortium' : '82 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">82 more authors</span> </div> <div class="periodical"> <em>SCIENCE TRANSLATIONAL MEDICINE</em>, Feb 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1126/scitranslmed.abj0264" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1126/scitranslmed.abj0264" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease with an estimated heritability between 40 and 50%. DNA methylation patterns can serve as proxies of (past) exposures and disease progression, as well as providing a potential mechanism that mediates genetic or environmental risk. Here, we present a blood-based epigenome-wide association study meta-analysis in 9706 samples passing stringent quality control (6763 patients, 2943 controls). We identified a total of 45 differentially methylated positions (DMPs) annotated to 42 genes, which are enriched for pathways and traits related to metabolism, cholesterol biosynthesis, and immunity. We then tested 39 DNA methylation-based proxies of putative ALS risk factors and found that high-density lipoprotein cholesterol, body mass index, white blood cell proportions, and alcohol intake were independently associated with ALS. Integration of these results with our latest genome-wide association study showed that cholesterol biosynthesis was potentially causally related to ALS. Last, DNA methylation at several DMPs and blood cell proportion estimates derived from DNA methylation data were associated with survival rate in patients, suggesting that they might represent indicators of underlying disease processes potentially amenable to therapeutic interventions.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000750327700001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41588-022-01020-3" rel="external nofollow noopener" target="_blank">Common and rare variant association analyses in amyotrophic lateral sclerosis identify 15 risk loci with distinct genetic architectures and neuron-specific biology (vol 53, pg 1636, 2021)</a></div> <div class="author"> Wouter Rheenen, Rick A. A. Spek, Mark K. Bakker, and <span class="more-authors" title="click to view 282 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '282 more authors' ? 'Joke J. F. A. Vugt, Paul J. Hop, Ramona A. J. Zwamborn, Niek Klein, Harm-Jan Westra, Olivier B. Bakker, Patrick Deelen, Gemma Shireby, Eilis Hannon, Matthieu Moisse, Denis Baird, Restuadi Restuadi, Egor Dolzhenko, Annelot M. Dekker, Klara Gawor, Henk-Jan Westeneng, Gijs H. P. Tazelaar, Kristel R. Eijk, Maarten Kooyman, Ross P. Byrne, Mark Doherty, Mark Heverin, Ahmad Al Khleifat, Alfredo Iacoangeli, Aleksey Shatunov, Nicola Ticozzi, Johnathan Cooper-Knock, Bradley N. Smith, Marta Gromicho, Siddharthan Chandran, Suvankar Pal, Karen E. Morrison, Pamela J. Shaw, John Hardy, Richard W. Orrell, Michael Sendtner, Thomas Meyer, Nazli Basak, Anneke J. Kooi, Antonia Ratti, Isabella Fogh, Cinzia Gellera, Giuseppe Lauria, Stefania Corti, Cristina Cereda, Daisy Sproviero, Sandra D’Alfonso, Gianni Soraru, Gabriele Siciliano, Massimiliano Filosto, Alessandro Padovani, Adriano Chio, Andrea Calvo, Cristina Moglia, Maura Brunetti, Antonio Canosa, Maurizio Grassano, Ettore Beghi, Elisabetta Pupillo, Giancarlo Logroscino, Beatrice Nefussy, Alma Osmanovic, Angelica Nordin, Yossef Lerner, Michal Zabari, Marc Gotkine, Robert H. Baloh, Shaughn Bell, Patrick Vourc’h, Philippe Corcia, Philippe Couratier, Stephanie Millecamps, Vincent Meininger, Francois Salachas, Jesus S. Mora Pardina, Abdelilah Assialioui, Ricardo Rojas-Garcia, Patrick A. Dion, Jay P. Ross, Albert C. Ludolph, Jochen H. Weishaupt, David Brenner, Axel Freischmidt, Gilbert Bensimon, Alexis Brice, Alexandra Durr, Christine A. M. Payan, Safa Saker-Delye, Nicholas W. Wood, Simon Topp, Rosa Rademakers, Lukas Tittmann, Wolfgang Lieb, Andre Franke, Stephan Ripke, Alice Braun, Julia Kraft, David C. Whiteman, Catherine M. Olsen, Andre G. Uitterlinden, Albert Hofman, Marcella Rietschel, Sven Cichon, Markus M. Nothen, Philippe Amouyel, Giancarlo Comi, Nilo Riva, Christian Lunetta, Francesca Gerardi, Maria Sofia Cotelli, Fabrizio Rinaldi, Luca Chiveri, Maria Cristina Guaita, Patrizia Perrone, Mauro Ceroni, Luca Diamanti, Carlo Ferrarese, Lucio Tremolizzo, Maria Luisa Delodovici, Giorgio Bono, Umberto Manera, Rosario Vasta, Alessandro Bombaci, Federico Casale, Giuseppe Fuda, Paolina Salamone, Barbara Iazzolino, Laura Peotta, Paolo Cugnasco, Giovanni De Marco, Maria Claudia Torrieri, Francesca Palumbo, Salvatore Gallone, Marco Barberis, Luca Sbaiz, Salvatore Gentile, Alessandro Mauro, Letizia Mazzini, Fabiola De Marchi, Lucia Corrado, Sandra D’Alfonso, Antonio Bertolotto, Maurizio Gionco, Daniela Leotta, Enrico Odddenino, Daniele Imperiale, Roberto Cavallo, Pietro Pignatta, Marco De Mattei, Claudio Geda, Diego Maria Papurello, Graziano Gusmaroli, Cristoforo Comi, Carmelo Labate, Luigi Ruiz, Delfina Ferrandi, Eugenia Rota, Marco Aguggia, Nicoletta Di Vito, Piero Meineri, Paolo Ghiglione, Nicola Launaro, Michele Dotta, Alessia Di Sapio, Guido Giardini, Cinzia Tiloca, Silvia Peverelli, Franco Taroni, Viviana Pensato, Barbara Castellotti, Giacomo P. Comi, Roberto Del Bo, Mauro Ceroni, Stella Gagliardi, Lucia Corrado, Letizia Mazzini, Flavia Raggi, Costanza Simoncini, Annalisa Lo Gerfo, Maurizio Inghilleri, Alessandra Ferlini, Isabella L. Simone, Bruno Passarella, Vito Guerra, Stefano Zoccolella, Cecilia Nozzoli, Ciro Mundi, Maurizio Leone, Michele Zarrelli, Filippo Tamma, Francesco Valluzzi, Gianluigi Calabrese, Giovanni Boero, Augusto Rini, Bryan J. Traynor, Andrew B. Singleton, Miguel Mitne Neto, Ruben J. Cauchi, Roel A. Ophoff, Martina Wiedau-Pazos, Catherine Lomen-Hoerth, Vivianna M. Deerlin, Julian Grosskreutz, Annekathrin Roediger, Nayana Gaur, Alexander Joerk, Tabea Barthel, Erik Theele, Benjamin Ilse, Beatrice Stubendorff, Otto W. Witte, Robert Steinbach, Christian A. Huebner, Caroline Graff, Lev Brylev, Vera Fominykh, Vera Demeshonok, Anastasia Ataulina, Boris Rogelj, Blaz Koritnik, Janez Zidar, Metka Ravnik-Glavac, Damjan Glavac, Zorica Stevic, Vivian Drory, Monica Povedano, Ian P. Blair, Matthew C. Kiernan, Beben Benyamin, Robert D. Henderson, Sarah Furlong, Susan Mathers, Pamela A. McCombe, Merrilee Needham, Shyuan T. Ngo, Garth A. Nicholson, Roger Pamphlett, Dominic B. Rowe, Frederik J. Steyn, Kelly L. Williams, Karen A. Mather, Perminder S. Sachdev, Anjali K. Henders, Leanne Wallace, Mamede Carvalho, Susana Pinto, Susanne Petri, Markus Weber, Guy A. Rouleau, Vincenzo Silani, Charles J. Curtis, Gerome Breen, Jonathan D. Glass, Robert H. Brown, John E. Landers, Christopher E. Shaw, Peter M. Andersen, Ewout J. N. Groen, Michael A. Es, R. Jeroen Pasterkamp, Dongsheng Fan, Fleur C. Garton, Allan F. McRae, George Davey Smith, Tom R. Gaunt, Michael A. Eberle, Jonathan Mill, Russell L. McLaughlin, Orla Hardiman, Kevin P. Kenna, Naomi R. Wray, Ellen Tsai, Heiko Runz, Lude Franke, Ammar Al-Chalabi, Philip Van Damme, Leonard H. Berg, Jan H. Veldink, SLALOM Consortium, PARALS Consortium, SLAGEN Consortium, SLAP Consortium' : '282 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">282 more authors</span> </div> <div class="periodical"> <em>NATURE GENETICS</em>, Mar 2022 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41588-022-01020-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41588-022-01020-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000760311100001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.7554/eLife.73420;%2010.7554/eLife.73420.sa0;%2010.7554/eLife.73420.sa1;%0A%20%20%2010.7554/eLife.73420.sa2" rel="external nofollow noopener" target="_blank">DunedinPACE, a DNA methylation biomarker of the pace of aging</a></div> <div class="author"> Daniel W. Belsky, Avshalom Caspi, David L. Corcoran, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Karen Sugden, Richie Poulton, Louise Arseneault, Andrea Baccarelli, Kartik Chamarti, Xu Gao, Eilis Hannon, Hona Lee Harrington, Renate Houts, Meeraj Kothari, Dayoon Kwon, Jonathan Mill, Joel Schwartz, Pantel Vokonas, Cuicui Wang, Benjamin S. Williams, Terrie E. Moffitt' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>ELIFE</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.7554/eLife.73420; 10.7554/eLife.73420.sa0; 10.7554/eLife.73420.sa1; 10.7554/eLife.73420.sa2" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.7554/eLife.73420; 10.7554/eLife.73420.sa0; 10.7554/eLife.73420.sa1; 10.7554/eLife.73420.sa2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Measures to quantify changes in the pace of biological aging in response to intervention are needed to evaluate geroprotective interventions for humans. Previously, we showed that quantification of the pace of biological aging from a DNA-methylation blood test was possible (Belsky et al., 2020). Here, we report a next-generation DNA-methylation biomarker of Pace of Aging, DunedinPACE (for Pace of Aging Calculated from the Epigenome). Methods: We used data from the Dunedin Study 1972-1973 birth cohort tracking within-individual decline in 19 indicators of organ-system integrity across four time points spanning two decades to model Pace of Aging. We distilled this two-decade Pace of Aging into a single-time-point DNA-methylation blood-test using elastic-net regression and a DNA-methylation dataset restricted to exclude probes with low test-retest reliability. We evaluated the resulting measure, named DunedinPACE, in five additional datasets. Results: DunedinPACE showed high test-retest reliability, was associated with morbidity, disability, and mortality, and indicated faster aging in young adults with childhood adversity. DunedinPACE effect-sizes were similar to GrimAge Clock effect-sizes. In analysis of incident morbidity, disability, and mortality, DunedinPACE and added incremental prediction beyond GrimAge. Conclusions: DunedinPACE is a novel blood biomarker of the pace of aging for gerontology and geroscience. Funding: This research was supported by US-National Institute on Aging grants AG032282, AG061378, AG066887, and UK Medical Research Council grant MR/P005918/1.</p> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000727387900007" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41588-021-00973-1" rel="external nofollow noopener" target="_blank">Common and rare variant association analyses in amyotrophic lateral sclerosis identify 15 risk loci with distinct genetic architectures and neuron-specific biology</a></div> <div class="author"> Wouter Rheenen, Rick A. A. Spek, Mark K. Bakker, and <span class="more-authors" title="click to view 282 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '282 more authors' ? 'Joke J. F. A. Vugt, Paul J. Hop, Ramona A. J. Zwamborn, Niek Klein, Harm-Jan Westra, Olivier B. Bakker, Patrick Deelen, Gemma Shireby, Eilis Hannon, Matthieu Moisse, Denis Baird, Restuadi Restuadi, Egor Dolzhenko, Annelot M. Dekker, Klara Gawor, Henk-Jan Westeneng, Gijs H. P. Tazelaar, Kristel R. Eijk, Maarten Kooyman, Ross P. Byrne, Mark Doherty, Mark Heverin, Ahmad Al Khleifat, Alfredo Iacoangeli, Aleksey Shatunov, Nicola Ticozzi, Johnathan Cooper-Knock, Bradley N. Smith, Marta Gromicho, Siddharthan Chandran, Suvankar Pal, Karen E. Morrison, Pamela J. Shaw, John Hardy, Richard W. Orrell, Michael Sendtner, Thomas Meyer, Nazli Basak, Anneke J. Kooi, Antonia Ratti, Isabella Fogh, Cinzia Gellera, Giuseppe Lauria, Stefania Corti, Cristina Cereda, Daisy Sproviero, Sandra D’Alfonso, Gianni Soraru, Gabriele Siciliano, Massimiliano Filosto, Alessandro Padovani, Adriano Chio, Andrea Calvo, Cristina Moglia, Maura Brunetti, Antonio Canosa, Maurizio Grassano, Ettore Beghi, Elisabetta Pupillo, Giancarlo Logroscino, Beatrice Nefussy, Alma Osmanovic, Angelica Nordin, Yossef Lerner, Michal Zabari, Marc Gotkine, Robert H. Baloh, Shaughn Bell, Patrick Vourc’h, Philippe Corcia, Philippe Couratier, Stephanie Millecamps, Vincent Meininger, Francois Salachas, Jesus S. Mora Pardina, Abdelilah Assialioui, Ricardo Rojas-Garcia, Patrick A. Dion, Jay P. Ross, Albert C. Ludolph, Jochen H. Weishaupt, David Brenner, Axel Freischmidt, Gilbert Bensimon, Alexis Brice, Alexandra Durr, Christine A. M. Payan, Safa Saker-Delye, Nicholas W. Wood, Simon Topp, Rosa Rademakers, Lukas Tittmann, Wolfgang Lieb, Andre Franke, Stephan Ripke, Alice Braun, Julia Kraft, David C. Whiteman, Catherine M. Olsen, Andre G. Uitterlinden, Albert Hofman, Marcella Rietschel, Sven Cichon, Markus M. Nothen, Philippe Amouyel, Giancarlo Comi, Nilo Riva, Christian Lunetta, Francesca Gerardi, Maria Sofia Cotelli, Fabrizio Rinaldi, Luca Chiveri, Maria Cristina Guaita, Patrizia Perrone, Mauro Ceroni, Luca Diamanti, Carlo Ferrarese, Lucio Tremolizzo, Maria Luisa Delodovici, Giorgio Bono, Umberto Manera, Rosario Vasta, Alessandro Bombaci, Federico Casale, Giuseppe Fuda, Paolina Salamone, Barbara Iazzolino, Laura Peotta, Paolo Cugnasco, Giovanni De Marco, Maria Claudia Torrieri, Francesca Palumbo, Salvatore Gallone, Marco Barberis, Luca Sbaiz, Salvatore Gentile, Alessandro Mauro, Letizia Mazzini, Fabiola De Marchi, Lucia Corrado, Sandra D’Alfonso, Antonio Bertolotto, Maurizio Gionco, Daniela Leotta, Enrico Odddenino, Daniele Imperiale, Roberto Cavallo, Pietro Pignatta, Marco De Mattei, Claudio Geda, Diego Maria Papurello, Graziano Gusmaroli, Cristoforo Comi, Carmelo Labate, Luigi Ruiz, Delfina Ferrandi, Eugenia Rota, Marco Aguggia, Nicoletta Di Vito, Piero Meineri, Paolo Ghiglione, Nicola Launaro, Michele Dotta, Alessia Di Sapio, Guido Giardini, Cinzia Tiloca, Silvia Peverelli, Franco Taroni, Viviana Pensato, Barbara Castellotti, Giacomo P. Comi, Roberto Del Bo, Mauro Ceroni, Stella Gagliardi, Lucia Corrado, Letizia Mazzini, Flavia Raggi, Costanza Simoncini, Annalisa Lo Gerfo, Maurizio Inghilleri, Alessandra Ferlini, Isabella L. Simone, Bruno Passarella, Vito Guerra, Stefano Zoccolella, Cecilia Nozzoli, Ciro Mundi, Maurizio Leone, Michele Zarrelli, Filippo Tamma, Francesco Valluzzi, Gianluigi Calabrese, Giovanni Boero, Augusto Rini, Bryan J. Traynor, Andrew B. Singleton, Miguel Mitne Neto, Ruben J. Cauchi, Roel A. Ophoff, Martina Wiedau-Pazos, Catherine Lomen-Hoerth, Vivianna M. Deerlin, Julian Grosskreutz, Annekathrin Roediger, Nayana Gaur, Alexander Joerk, Tabea Barthel, Erik Theele, Benjamin Ilse, Beatrice Stubendorff, Otto W. Witte, Robert Steinbach, Christian A. Huebner, Caroline Graff, Lev Brylev, Vera Fominykh, Vera Demeshonok, Anastasia Ataulina, Boris Rogelj, Blaz Koritnik, Janez Zidar, Metka Ravnik-Glavac, Damjan Glavac, Zorica Stevic, Vivian Drory, Monica Povedano, Ian P. Blair, Matthew C. Kiernan, Beben Benyamin, Robert D. Henderson, Sarah Furlong, Susan Mathers, Pamela A. McCombe, Merrilee Needham, Shyuan T. Ngo, Garth A. Nicholson, Roger Pamphlett, Dominic B. Rowe, Frederik J. Steyn, Kelly L. Williams, Karen A. Mather, Perminder S. Sachdev, Anjali K. Henders, Leanne Wallace, Mamede Carvalho, Susana Pinto, Susanne Petri, Markus Weber, Guy A. Rouleau, Vincenzo Silani, Charles J. Curtis, Gerome Breen, Jonathan D. Glass, Robert H. Brown, John E. Landers, Christopher E. Shaw, Peter M. Andersen, Ewout J. N. Groen, Michael A. Es, R. Jeroen Pasterkamp, Dongsheng Fan, Fleur C. Garton, Allan F. McRae, George Davey Smith, Tom R. Gaunt, Michael A. Eberle, Jonathan Mill, Russell L. McLaughlin, Orla Hardiman, Kevin P. Kenna, Naomi R. Wray, Ellen Tsai, Heiko Runz, Lude Franke, Ammar Al-Chalabi, Philip Van Damme, Leonard H. Berg, Jan H. Veldink, SLALOM Consortium, PARALS Consortium, SLAGEN Consortium, SLAP Consortium' : '282 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">282 more authors</span> </div> <div class="periodical"> <em>NATURE GENETICS</em>, Dec 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41588-021-00973-1" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41588-021-00973-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>A cross-ancestry genome-wide association meta-analysis of amyotrophic lateral sclerosis (ALS) including 29,612 patients with ALS and 122,656 controls identifies 15 risk loci with distinct genetic architectures and neuron-specific biology. Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease with a lifetime risk of one in 350 people and an unmet need for disease-modifying therapies. We conducted a cross-ancestry genome-wide association study (GWAS) including 29,612 patients with ALS and 122,656 controls, which identified 15 risk loci. When combined with 8,953 individuals with whole-genome sequencing (6,538 patients, 2,415 controls) and a large cortex-derived expression quantitative trait locus (eQTL) dataset (MetaBrain), analyses revealed locus-specific genetic architectures in which we prioritized genes either through rare variants, short tandem repeats or regulatory effects. ALS-associated risk loci were shared with multiple traits within the neurodegenerative spectrum but with distinct enrichment patterns across brain regions and cell types. Of the environmental and lifestyle risk factors obtained from the literature, Mendelian randomization analyses indicated a causal role for high cholesterol levels. The combination of all ALS-associated signals reveals a role for perturbations in vesicle-mediated transport and autophagy and provides evidence for cell-autonomous disease initiation in glutamatergic neurons.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000720347900007" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.celrep.2021.110022" rel="external nofollow noopener" target="_blank">Full-length transcript sequencing of human and mouse cerebral cortex identifies widespread isoform diversity and alternative splicing</a></div> <div class="author"> Szi Kay Leung, Aaron R. Jeffries, Isabel Castanho, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Ben T. Jordan, Karen Moore, Jonathan P. Davies, Emma L. Dempster, Nicholas J. Bray, Paul O’Neill, Elizabeth Tseng, Zeshan Ahmed, David A. Collier, Erin D. Jeffery, Shyam Prabhakar, Leonard Schalkwyk, Connor Jops, Michael J. Gandal, Gloria M. Sheynkman, Eilis Hannon, Jonathan Mill' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>CELL REPORTS</em>, Nov 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.celrep.2021.110022" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.celrep.2021.110022" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alternative splicing is a post-transcriptional regulatory mechanism producing distinct mRNA molecules from a single pre-mRNA with a prominent role in the development and function of the central nervous system. We used long-read isoform sequencing to generate full-length transcript sequences in the human and mouse cortex. We identify novel transcripts not present in existing genome annotations, including transcripts mapping to putative novel (unannotated) genes and fusion transcripts incorporating exons from multiple genes. Global patterns of transcript diversity are similar between human and mouse cortex, although certain genes are characterized by striking differences between species. We also identify developmental changes in alternative splicing, with differential transcript usage between human fetal and adult cortex. Our data confirm the importance of alternative splicing in the cortex, dramatically increasing transcriptional diversity and representing an important mechanism underpinning gene regulation in the brain. We provide transcript-level data for human and mouse cortex as a resource to the scientific community.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000705103600001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1002/ajmg.b.32877" rel="external nofollow noopener" target="_blank">Sites of active gene regulation in the prenatal frontal cortex and their role in neuropsychiatric disorders</a></div> <div class="author"> Manuela R. Kouakou, Darren Cameron, Eilis Hannon, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Emma L. Dempster, Jonathan Mill, Matthew J. Hill, Nicholas J. Bray' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>AMERICAN JOURNAL OF MEDICAL GENETICS PART B-NEUROPSYCHIATRIC GENETICS</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/ajmg.b.32877" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/ajmg.b.32877" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Common genetic variation appears to largely influence risk for neuropsychiatric disorders through effects on gene regulation. It is therefore possible to shed light on the biology of these conditions by testing for enrichment of associated genetic variation within regulatory genomic regions operating in specific tissues or cell types. Here, we have used the assay for transposase-accessible chromatin with high-throughput sequencing (ATAC-Seq) to map open chromatin (an index of active regulatory genomic regions) in bulk tissue, NeuN+ and NeuN- nuclei from the prenatal human frontal cortex, and tested enrichment of single-nucleotide polymorphism (SNP) heritability for five neuropsychiatric disorders (autism spectrum disorder, attention deficit hyperactivity disorder [ADHD], bipolar disorder, major depressive disorder, and schizophrenia) within these regions. We observed significant enrichment of SNP heritability for ADHD, major depressive disorder, and schizophrenia within open chromatin regions (OCRs) mapped in bulk fetal frontal cortex, and for all five tested neuropsychiatric conditions when we restricted these sites to those overlapping histone modifications indicative of enhancers (H3K4me1) or promoters (H3K4me3) in fetal brain. SNP heritability for neuropsychiatric disorders was significantly enriched in OCRs identified in fetal frontal cortex NeuN- as well as NeuN+ nuclei overlapping fetal brain H3K4me1 or H3K4me3 sites. We additionally demonstrate the utility of our mapped OCRs for prioritizing potentially functional SNPs at genome-wide significant risk loci for neuropsychiatric disorders. Our data provide evidence for an early neurodevelopmental component to a range of neuropsychiatric conditions and highlight an important role for regulatory genomic regions active within both NeuN+ and NeuN- cells of the prenatal brain.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000701980200006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41467-021-25583-7" rel="external nofollow noopener" target="_blank">Identical twins carry a persistent epigenetic signature of early genome programming</a></div> <div class="author"> Jenny Dongen, Scott D. Gordon, Allan F. McRae, and <span class="more-authors" title="click to view 35 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '35 more authors' ? 'Veronika V. Odintsova, Hamdi Mbarek, Charles E. Breeze, Karen Sugden, Sara Lundgren, Juan E. Castillo-Fernandez, Eilis Hannon, Terrie E. Moffitt, Fiona A. Hagenbeek, Catharina E. M. Beijsterveldt, Jouke Jan Hottenga, Pei-Chien Tsai, Josine L. Min, Gibran Hemani, Erik A. Ehli, Franziska Paul, Claudio D. Stern, Bastiaan T. Heijmans, P. Eline Slagboom, Lucia Daxinger, Silvere M. Maarel, E. J. C. Geus, Gonneke Willemsen, Grant W. Montgomery, Bruno Reversade, Miina Ollikainen, Jaakko Kaprio, Tim D. Spector, Jordana T. Bell, Jonathan Mill, Avshalom Caspi, Nicholas G. Martin, Dorret Boomsma, Bios Consortium, Genetics DNA Methylation Consortiu' : '35 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">35 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41467-021-25583-7" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-021-25583-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The mechanisms underlying how monozygotic (or identical) twins arise are yet to be determined. Here, the authors investigate this in an epigenome-wide association study, showing that monozygotic twinning has a characteristic DNA methylation signature in adult somatic tissues. Monozygotic (MZ) twins and higher-order multiples arise when a zygote splits during pre-implantation stages of development. The mechanisms underpinning this event have remained a mystery. Because MZ twinning rarely runs in families, the leading hypothesis is that it occurs at random. Here, we show that MZ twinning is strongly associated with a stable DNA methylation signature in adult somatic tissues. This signature spans regions near telomeres and centromeres, Polycomb-repressed regions and heterochromatin, genes involved in cell-adhesion, WNT signaling, cell fate, and putative human metastable epialleles. Our study also demonstrates a never-anticipated corollary: because identical twins keep a lifelong molecular signature, we can retrospectively diagnose if a person was conceived as monozygotic twin.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000704035500114" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2021.07.127" rel="external nofollow noopener" target="_blank">CELL-TYPE-SPECIFIC PATTERNS OF DNA METHYLATION IN THE DEVELOPING HUMAN BRAIN</a></div> <div class="author"> Jonathan Davies, Alice Franklin, Gina Commin, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Emma Walker, Stefania Policicchio, Aaron Jeffries, Joe Burrage, Barry Chioza, Jinyue Liu, Nick Bray, Shyam Prabhakar, Eilis Hannon, Emma Dempster, Jonathan Mill' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2021 </div> <div class="periodical"> World Congress of Psychiatric Genetics (WCPG), ELECTR NETWORK, OCT 11-15, 2021 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2021.07.127" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2021.07.127" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000704035500379" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2021.08.216" rel="external nofollow noopener" target="_blank">LEVERAGING DNA METHYLATION QUANTITATIVE-TRAIT LOCI TO CHARACTERIZE THE RELATIONSHIP BETWEEN METHYLOMIC VARIATION, GENE EXPRESSION, AND PSYCHIATRIC DISEASE</a></div> <div class="author"> Gemma Shireby, Eilis Hannon, Gina Commin, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Joe Burrage, Jonathan Davies, Stefania Policicchio, Leo Schalkwyk, Emma Dempster, Jonathan Mill' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2021 </div> <div class="periodical"> World Congress of Psychiatric Genetics (WCPG), ELECTR NETWORK, OCT 11-15, 2021 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2021.08.216" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2021.08.216" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000694666000011" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41588-021-00923-x" rel="external nofollow noopener" target="_blank">Genomic and phenotypic insights from an atlas of genetic effects on DNA methylation</a></div> <div class="author"> Josine L. Min, Gibran Hemani, Eilis Hannon, and <span class="more-authors" title="click to view 154 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '154 more authors' ? 'Koen F. Dekkers, Juan Castillo-Fernandez, Rene Luijk, Elena Carnero-Montoro, Daniel J. Lawson, Kimberley Burrows, Matthew Suderman, Andrew D. Bretherick, Tom G. Richardson, Johanna Klughammer, Valentina Iotchkova, Gemma Sharp, Ahmad Al Khleifat, Aleksey Shatunov, Alfredo Iacoangeli, Wendy L. McArdle, Karen M. Ho, Ashish Kumar, Cilia Soderhall, Carolina Soriano-Tarraga, Eva Giralt-Steinhauer, Nabila Kazmi, Dan Mason, Allan F. McRae, David L. Corcoran, Karen Sugden, Silva Kasela, Alexia Cardona, Felix R. Day, Giovanni Cugliari, Clara Viberti, Simonetta Guarrera, Michael Lerro, Richa Gupta, Sailalitha Bollepalli, Pooja Mandaviya, Yanni Zeng, Toni-Kim Clarke, Rosie M. Walker, Vanessa Schmoll, Darina Czamara, Carlos Ruiz-Arenas, Faisal Rezwan, Riccardo E. Marioni, Tian Lin, Yvonne Awaloff, Marine Germain, Dylan Aissi, Ramona Zwamborn, Kristel Eijk, Annelot Dekker, Jenny Dongen, Jouke-Jan Hottenga, Gonneke Willemsen, Cheng-Jian Xu, Guillermo Barturen, Francesc Catala-Moll, Martin Kerick, Carol Wang, Phillip Melton, Hannah R. Elliott, Jean Shin, Manon Bernard, Idil Yet, Melissa Smarts, Tyler Gorrie-Stone, Chris Shaw, Ammar Al Chalabi, Susan M. Ring, Goran Pershagen, Erik Melen, Jordi Jimenez-Conde, Jaume Roquer, Deborah A. Lawlor, John Wright, Nicholas G. Martin, Grant W. Montgomery, Terrie E. Moffitt, Richie Poulton, Tonu Esko, Lili Milani, Andres Metspalu, John R. B. Perry, Ken K. Ong, Nicholas J. Wareham, Giuseppe Matullo, Carlotta Sacerdote, Salvatore Panico, Avshalom Caspi, Louise Arseneault, France Gagnon, Miina Ollikainen, Jaakko Kaprio, Janine F. Felix, Fernando Rivadeneira, Henning Tiemeier, Marinus H. IJzendoorn, Andre G. Uitterlinden, Vincent W. V. Jaddoe, Chris Hale, Andrew M. McIntosh, Kathryn L. Evans, Alison Murray, Katri Raikkonen, Jari Lahti, Ellen A. Nohr, Thorkild I. A. Sorensen, Torben Hansen, Camilla S. Morgen, Elisabeth B. Binder, Susanne Lucae, Juan Ramon Gonzalez, Mariona Bustamante, Jordi Sunyer, John W. Holloway, Wilfried Karmaus, Hongmei Zhang, Ian J. Deary, Naomi R. Wray, John M. Starr, Marian Beekman, Diana VanHeemst, P. Eline Slagboom, Pierre-Emmanuel Morange, David-Alexandre Tregouet, Jan H. Veldink, Gareth E. Davies, Eco J. C. Geus, Dorret Boomsma, Judith M. Vonk, Bert Brunekreef, Gerard H. Koppelman, Marta E. Alarcon-Riquelme, Rae-Chi Huang, Craig E. Pennell, Joyce Meurs, M. Arfan Ikram, Alun D. Hughes, Therese Tillin, Nish Chaturvedi, Zdenka Pausova, Tomas Paus, Timothy D. Spector, Meena Kumari, Leonard C. Schalkwyk, Peter M. Visscher, George Davey Smith, Christoph Bock, Tom R. Gaunt, Jordana T. Bell, Bastiaan T. Heijmans, Jonathan Mill, Caroline L. Relton, BIOS Consortium' : '154 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">154 more authors</span> </div> <div class="periodical"> <em>NATURE GENETICS</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41588-021-00923-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41588-021-00923-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>DNA methylation quantitative trait locus (mQTL) analyses on 32,851 participants identify genetic variants associated with DNA methylation at 420,509 sites in blood, resulting in a database of &gt;270,000 independent mQTLs. Characterizing genetic influences on DNA methylation (DNAm) provides an opportunity to understand mechanisms underpinning gene regulation and disease. In the present study, we describe results of DNAm quantitative trait locus (mQTL) analyses on 32,851 participants, identifying genetic variants associated with DNAm at 420,509 DNAm sites in blood. We present a database of &gt;270,000 independent mQTLs, of which 8.5% comprise long-range (trans) associations. Identified mQTL associations explain 15-17% of the additive genetic variance of DNAm. We show that the genetic architecture of DNAm levels is highly polygenic. Using shared genetic control between distal DNAm sites, we constructed networks, identifying 405 discrete genomic communities enriched for genomic annotations and complex traits. Shared genetic variants are associated with both DNAm levels and complex diseases, but only in a minority of cases do these associations reflect causal relationships from DNAm to trait or vice versa, indicating a more complex genotype-phenotype map than previously anticipated.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000680546900001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-021-01496-3" rel="external nofollow noopener" target="_blank">A machine learning case-control classifier for schizophrenia based on DNA methylation in blood</a></div> <div class="author"> Chathura J. Gunasekara, Eilis Hannon, Harry MacKay, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Cristian Coarfa, Andrew McQuillin, David St Clair, Jonathan Mill, Robert A. Waterland' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-021-01496-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-021-01496-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Epigenetic dysregulation is thought to contribute to the etiology of schizophrenia (SZ), but the cell type-specificity of DNA methylation makes population-based epigenetic studies of SZ challenging. To train an SZ case-control classifier based on DNA methylation in blood, therefore, we focused on human genomic regions of systemic interindividual epigenetic variation (CoRSIVs), a subset of which are represented on the Illumina Human Methylation 450K (HM450) array. HM450 DNA methylation data on whole blood of 414 SZ cases and 433 non-psychiatric controls were used as training data for a classification algorithm with built-in feature selection, sparse partial least squares discriminate analysis (SPLS-DA); application of SPLS-DA to HM450 data has not been previously reported. Using the first two SPLS-DA dimensions we calculated a “risk distance” to identify individuals with the highest probability of SZ. The model was then evaluated on an independent HM450 data set on 353 SZ cases and 322 non-psychiatric controls. Our CoRSIV-based model classified 303 individuals as cases with a positive predictive value (PPV) of 80%, far surpassing the performance of a model based on polygenic risk score (PRS). Importantly, risk distance (based on CoRSIV methylation) was not associated with medication use, arguing against reverse causality. Risk distance and PRS were positively correlated (Pearson r = 0.28, P = 1.28 x 10(-12)), and mediational analysis suggested that genetic effects on SZ are partially mediated by altered methylation at CoRSIVs. Our results indicate two innate dimensions of SZ risk: one based on genetic, and the other on systemic epigenetic variants.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000669604000002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s12864-021-07675-2" rel="external nofollow noopener" target="_blank">DNA methylation-based sex classifier to predict sex and identify sex chromosome aneuploidy</a></div> <div class="author"> Yucheng Wang, Eilis Hannon, Olivia A. Grant, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Tyler J. Gorrie-Stone, Meena Kumari, Jonathan Mill, Xiaojun Zhai, Klaus D. McDonald-Maier, Leonard C. Schalkwyk' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>BMC GENOMICS</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s12864-021-07675-2" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-021-07675-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Sex is an important covariate of epigenome-wide association studies due to its strong influence on DNA methylation patterns across numerous genomic positions. Nevertheless, many samples on the Gene Expression Omnibus (GEO) frequently lack a sex annotation or are incorrectly labelled. Considering the influence that sex imposes on DNA methylation patterns, it is necessary to ensure that methods for filtering poor samples and checking of sex assignment are accurate and widely applicable. Results: Here we presented a novel method to predict sex using only DNA methylation beta values, which can be readily applied to almost all DNA methylation datasets of different formats (raw IDATs or text files with only signal intensities) uploaded to GEO. We identified 4345 significantly (p&lt;0.01) sex-associated CpG sites present on both 450K and EPIC arrays, and constructed a sex classifier based on the two first principal components of the DNA methylation data of sex-associated probes mapped on sex chromosomes. The proposed method is constructed using whole blood samples and exhibits good performance across a wide range of tissues. We further demonstrated that our method can be used to identify samples with sex chromosome aneuploidy, this function is validated by five Turner syndrome cases and one Klinefelter syndrome case. Conclusions: This proposed sex classifier not only can be used for sex predictions but also applied to identify samples with sex chromosome aneuploidy, and it is freely and easily accessible by calling the ‘estimateSex’ function from the newest wateRmelon Bioconductor package (https://github.com/schalkwyk/wateRmelon).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000668631000003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13041-021-00810-w" rel="external nofollow noopener" target="_blank">Novel epigenetic clock for fetal brain development predicts prenatal age for cellular stem cell models and derived neurons</a></div> <div class="author"> Leonard C. Steg, Gemma L. Shireby, Jennifer Imm, and <span class="more-authors" title="click to view 22 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '22 more authors' ? 'Jonathan P. Davies, Alice Franklin, Robert Flynn, Seema C. Namboori, Akshay Bhinge, Aaron R. Jeffries, Joe Burrage, Grant W. A. Neilson, Emma M. Walker, Leo W. Perfect, Jack Price, Grainne McAlonan, Deepak P. Srivastava, Nicholas J. Bray, Emma L. Cope, Kimberley M. Jones, Nicholas D. Allen, Ehsan Pishva, Emma L. Dempster, Katie Lunnon, Jonathan Mill, Eilis Hannon' : '22 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">22 more authors</span> </div> <div class="periodical"> <em>MOLECULAR BRAIN</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13041-021-00810-w" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13041-021-00810-w" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Induced pluripotent stem cells (iPSCs) and their differentiated neurons (iPSC-neurons) are a widely used cellular model in the research of the central nervous system. However, it is unknown how well they capture age-associated processes, particularly given that pluripotent cells are only present during the earliest stages of mammalian development. Epigenetic clocks utilize coordinated age-associated changes in DNA methylation to make predictions that correlate strongly with chronological age. It has been shown that the induction of pluripotency rejuvenates predicted epigenetic age. As existing clocks are not optimized for the study of brain development, we developed the fetal brain clock (FBC), a bespoke epigenetic clock trained in human prenatal brain samples in order to investigate more precisely the epigenetic age of iPSCs and iPSC-neurons. The FBC was tested in two independent validation cohorts across a total of 194 samples, confirming that the FBC outperforms other established epigenetic clocks in fetal brain cohorts. We applied the FBC to DNA methylation data from iPSCs and embryonic stem cells and their derived neuronal precursor cells and neurons, finding that these cell types are epigenetically characterized as having an early fetal age. Furthermore, while differentiation from iPSCs to neurons significantly increases epigenetic age, iPSC-neurons are still predicted as being fetal. Together our findings reiterate the need to better understand the limitations of existing epigenetic clocks for answering biological research questions and highlight a limitation of iPSC-neurons as a cellular model of age-related diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000684816800002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.nbd.2021.105428" rel="external nofollow noopener" target="_blank">The association of epigenetic clocks in brain tissue with brain pathologies and common aging phenotypes</a></div> <div class="author"> Francine Grodstein, Bernardo Lemos, Lei Yu, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Hans-Ulrich Klein, Artemis Iatrou, Aron S. Buchman, Gemma L. Shireby, Jonathan Mill, Julie A. Schneider, Philip L. De Jager, David A. Bennett' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF DISEASE</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.nbd.2021.105428" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.nbd.2021.105428" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Epigenetic clocks are calculated by combining DNA methylation states across select CpG sites to estimate biologic age, and have been noted as the most successful markers of biologic aging to date. Yet, limited research has considered epigenetic clocks calculated in brain tissue. We used DNA methylation states in dorsolateral prefrontal cortex specimens from 721 older participants of the Religious Orders Study and Rush Memory and Aging Project, to calculate DNA methylation age using four established epigenetic clocks: Hannum, Horvath, PhenoAge, GrimAge, and a new Cortical clock. The four established clocks were trained in blood samples (Hannum, PhenoAge, GrimAge) or using 51 human tissue and cell types (Horvath); the recent Cortical clock is the first trained in postmortem cortical tissue. Participants were recruited beginning in 1994 (Religious Orders Study) and 1997 (Memory and Aging Project), and followed annually with questionnaires and clinical evaluations; brain specimens were obtained for 80-90% of participants. Mean age at death was 88.0 (SD 6.7) years. We used linear regression, logistic regression, and linear mixed models, to examine relations of epigenetic clock ages to neuropathologic and clinical aging phenotypes, controlling for chronologic age, sex, education, and depressive symptomatology. Hannum, Horvath, PhenoAge and Cortical clock ages were related to pathologic diagnosis of Alzheimer’s disease (AD), as well as to A beta load (a hallmark pathology of Alzheimer’s disease). However, associations were substantially stronger for the Cortical than other clocks; for example, each standard deviation (SD) increase in Hannum, Horvath, and PhenoAge clock age was related to approximately 30% greater likelihood of pathologic AD (all p &lt; 0.05), while each SD increase in Cortical age was related to 90% greater likelihood of pathologic AD (odds ratio = 1.91, 95% confidence interval 1.38, 2.62). Moreover, Cortical age was significantly related to other AD pathology (eg, mean tau tangle density, p = 0.003), and to odds of neocortical Lewy body pathology (for each SD increase in Cortical age, odds ratio = 2.00, 95% confidence 1.27, 3.17), although no clocks were related to cerebrovascular neuropathology. Cortical age was the only epigenetic clock significantly associated with the clinical phenotypes examined, from dementia to cognitive decline (5 specific cognitive systems, and a global cognitive measure averaging 17 tasks) to Parkinsonian signs. Overall, our findings provide evidence of the</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000663793500003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s12864-021-07721-z" rel="external nofollow noopener" target="_blank">Characterizing the properties of bisulfite sequencing data: maximizing power and sensitivity to identify between-group differences in DNA methylation</a></div> <div class="author"> Dorothea Seiler Vellame, Isabel Castanho, Aisha Dahir, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Jonathan Mill, Eilis Hannon' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>BMC GENOMICS</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s12864-021-07721-z" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-021-07721-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background The combination of sodium bisulfite treatment with highly-parallel sequencing is a common method for quantifying DNA methylation across the genome. The power to detect between-group differences in DNA methylation using bisulfite-sequencing approaches is influenced by both experimental (e.g. read depth, missing data and sample size) and biological (e.g. mean level of DNA methylation and difference between groups) parameters. There is, however, no consensus about the optimal thresholds for filtering bisulfite sequencing data with implications for the reproducibility of findings in epigenetic epidemiology. Results We used a large reduced representation bisulfite sequencing (RRBS) dataset to assess the distribution of read depth across DNA methylation sites and the extent of missing data. To investigate how various study variables influence power to identify DNA methylation differences between groups, we developed a framework for simulating bisulfite sequencing data. As expected, sequencing read depth, group size, and the magnitude of DNA methylation difference between groups all impacted upon statistical power. The influence on power was not dependent on one specific parameter, but reflected the combination of study-specific variables. As a resource to the community, we have developed a tool, POWEREDBiSeq, which utilizes our simulation framework to predict study-specific power for the identification of DNAm differences between groups, taking into account user-defined read depth filtering parameters and the minimum sample size per group. Conclusions Our data-driven approach highlights the importance of filtering bisulfite-sequencing data by minimum read depth and illustrates how the choice of threshold is influenced by the specific study design and the expected differences between groups being compared. The POWEREDBiSeq tool, which can be applied to different types of bisulfite sequencing data (e.g. RRBS, whole genome bisulfite sequencing (WGBS), targeted bisulfite sequencing and amplicon-based bisulfite sequencing), can help users identify the level of data filtering needed to optimize power and aims to improve the reproducibility of bisulfite sequencing studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000663747800007" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41467-021-23243-4" rel="external nofollow noopener" target="_blank">A meta-analysis of epigenome-wide association studies in Alzheimer’s disease highlights novel differentially methylated loci across cortex</a></div> <div class="author"> Rebecca G. Smith, Ehsan Pishva, Gemma Shireby, and <span class="more-authors" title="click to view 25 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '25 more authors' ? 'Adam R. Smith, Janou A. Y. Roubroeks, Eilis Hannon, Gregory Wheildon, Diego Mastroeni, Gilles Gasparoni, Matthias Riemenschneider, Armin Giese, Andrew J. Sharp, Leonard Schalkwyk, Vahram Haroutunian, Wolfgang Viechtbauer, Daniel L. A. Hove, Michael Weedon, Danielle Brokaw, Paul T. Francis, Alan J. Thomas, Seth Love, Kevin Morgan, Joern Walter, Paul D. Coleman, David A. Bennett, Philip L. De Jager, Jonathan Mill, Katie Lunnon' : '25 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">25 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41467-021-23243-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-021-23243-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Epigenome-wide association studies of Alzheimer’s disease have highlighted neuropathology-associated DNA methylation differences, although existing studies have been limited in sample size and utilized different brain regions. Here, we combine data from six DNA methylomic studies of Alzheimer’s disease (N=1453 unique individuals) to identify differential methylation associated with Braak stage in different brain regions and across cortex. We identify 236 CpGs in the prefrontal cortex, 95 CpGs in the temporal gyrus and ten CpGs in the entorhinal cortex at Bonferroni significance, with none in the cerebellum. Our cross-cortex meta-analysis (N=1408 donors) identifies 220 CpGs associated with neuropathology, annotated to 121 genes, of which 84 genes have not been previously reported at this significance threshold. We have replicated our findings using two further DNA methylomic datasets consisting of a further &gt;600 unique donors. The meta-analysis summary statistics are available in our online data resource (www.epigenomicslab.com/ad-meta-analysis/). Although epigenome-wide association studies of Alzheimer’s disease have highlighted neuropathology-associated DNA methylation differences, previous studies have been limited in sample size and brain region used. Here, the authors combine data from six DNA methylomic studies of Alzheimer’s disease (N=1453 unique individuals) to identify differentially methylated loci across cortex.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000660063100002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.cub.2021.03.062" rel="external nofollow noopener" target="_blank">A central role for anterior cingulate cortex in the control of pathological aggression</a></div> <div class="author"> Sabrina Heukelum, Kerli Tulva, Femke E. Geers, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Sanne Dulm, I. Hyun Ruisch, Jonathan Mill, Joana F. Viana, Christian F. Beckmann, Jan K. Buitelaar, Geert Poelmans, Jeffrey C. Glennon, Brent A. Vogt, Martha N. Havenith, Arthur S. C. Franca' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>CURRENT BIOLOGY</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.cub.2021.03.062" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2021.03.062" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Controlling aggression is a crucial skill in social species like rodents and humans and has been associated with anterior cingulate cortex (ACC). Here, we directly link the failed regulation of aggression in BALB/cJ mice to ACC hypofunction. We first show that ACC in BALB/cJ mice is structurally degraded: neuron density is decreased, with pervasive neuron death and reactive astroglia. Gene-set enrichment analysis suggested that this process is driven by neuronal degeneration, which then triggers toxic astrogliosis. cFos expression across ACC indicated functional consequences: during aggressive encounters, ACC was engaged in control mice, but not BALB/cJ mice. Chemogenetically activating ACC during aggressive encounters drastically suppressed pathological aggression but left species-typical aggression intact. The network effects of our chemogenetic perturbation suggest that this behavioral rescue is mediated by suppression of amygdala and hypothalamus and activation of mediodorsal thalamus. Together, these findings highlight the central role of ACC in curbing pathological aggression.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000633385500001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13059-021-02275-5" rel="external nofollow noopener" target="_blank">Meta-analysis of genome-wide DNA methylation identifies shared associations across neurodegenerative disorders</a></div> <div class="author"> Marta F. Nabais, Simon M. Laws, Tian Lin, and <span class="more-authors" title="click to view 69 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '69 more authors' ? 'Costanza L. Vallerga, Nicola J. Armstrong, Ian P. Blair, John B. Kwok, Karen A. Mather, George D. Mellick, Perminder S. Sachdev, Leanne Wallace, Anjali K. Henders, Ramona A. J. Zwamborn, Paul J. Hop, Katie Lunnon, Ehsan Pishva, Janou A. Y. Roubroeks, Hilkka Soininen, Magda Tsolaki, Patrizia Mecocci, Simon Lovestone, Iwona Kloszewska, Bruno Vellas, Sarah Furlong, Fleur C. Garton, Robert D. Henderson, Susan Mathers, Pamela A. McCombe, Merrilee Needham, Shyuan T. Ngo, Garth Nicholson, Roger Pamphlett, Dominic B. Rowe, Frederik J. Steyn, Kelly L. Williams, Tim J. Anderson, Steven R. Bentley, John Dalrymple-Alford, Javed Fowder, Jacob Gratten, Glenda Halliday, Ian B. Hickie, Martin Kennedy, Simon J. G. Lewis, Grant W. Montgomery, John Pearson, Toni L. Pitcher, Peter Silburn, Futao Zhang, Peter M. Visscher, Jian Yang, Anna J. Stevenson, Robert F. Hillary, Riccardo E. Marioni, Sarah E. Harris, Ian J. Deary, Ashley R. Jones, Aleksey Shatunov, Alfredo Iacoangeli, Wouter Rheenen, Leonard H. Berg, Pamela J. Shaw, Cristopher E. Shaw, Karen E. Morrison, Ammar Al-Chalabi, Jan H. Veldink, Eilis Hannon, Jonathan Mill, Naomi R. Wray, Allan F. McRae, Australian Imaging Biomarkers Life, Alzheimer’s Dis Neuroimaging Initi' : '69 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">69 more authors</span> </div> <div class="periodical"> <em>GENOME BIOLOGY</em>, Mar 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-021-02275-5" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-021-02275-5" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background People with neurodegenerative disorders show diverse clinical syndromes, genetic heterogeneity, and distinct brain pathological changes, but studies report overlap between these features. DNA methylation (DNAm) provides a way to explore this overlap and heterogeneity as it is determined by the combined effects of genetic variation and the environment. In this study, we aim to identify shared blood DNAm differences between controls and people with Alzheimer’s disease, amyotrophic lateral sclerosis, and Parkinson’s disease. Results We use a mixed-linear model method (MOMENT) that accounts for the effect of (un)known confounders, to test for the association of each DNAm site with each disorder. While only three probes are found to be genome-wide significant in each MOMENT association analysis of amyotrophic lateral sclerosis and Parkinson’s disease (and none with Alzheimer’s disease), a fixed-effects meta-analysis of the three disorders results in 12 genome-wide significant differentially methylated positions. Predicted immune cell-type proportions are disrupted across all neurodegenerative disorders. Protein inflammatory markers are correlated with profile sum-scores derived from disease-associated immune cell-type proportions in a healthy aging cohort. In contrast, they are not correlated with MOMENT DNAm-derived profile sum-scores, calculated using effect sizes of the 12 differentially methylated positions as weights. Conclusions We identify shared differentially methylated positions in whole blood between neurodegenerative disorders that point to shared pathogenic mechanisms. These shared differentially methylated positions may reflect causes or consequences of disease, but they are unlikely to reflect cell-type proportion differences.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000631085600002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1371/journal.pgen.1009443" rel="external nofollow noopener" target="_blank">Assessing the co-variability of DNA methylation across peripheral cells and tissues: Implications for the interpretation of findings in epigenetic epidemiology</a></div> <div class="author"> Eilis Hannon, Georgina Mansell, Emma Walker, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Marta F. Nabais, Joe Burrage, Agnieszka Kepa, Janis Best-Lane, Anna Rose, Suzanne Heck, Terrie E. Moffitt, Avshalom Caspi, Louise Arseneault, Jonathan Mill' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>PLOS GENETICS</em>, Mar 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1371/journal.pgen.1009443" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pgen.1009443" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Most epigenome-wide association studies (EWAS) quantify DNA methylation (DNAm) in peripheral tissues such as whole blood to identify positions in the genome where variation is statistically associated with a trait or exposure. As whole blood comprises a mix of cell types, it is unclear whether trait-associated DNAm variation is specific to an individual cellular population. We collected three peripheral tissues (whole blood, buccal epithelial and nasal epithelial cells) from thirty individuals. Whole blood samples were subsequently processed using fluorescence-activated cell sorting (FACS) to purify five constituent cell-types (monocytes, granulocytes, CD4(+) T cells, CD8(+) T cells, and B cells). DNAm was profiled in all eight sample-types from each individual using the Illumina EPIC array. We identified significant differences in both the level and variability of DNAm between different sample types, and DNAm data-derived estimates of age and smoking were found to differ dramatically across sample types from the same individual. We found that for the majority of loci variation in DNAm in individual blood cell types was only weakly predictive of variance in DNAm measured in whole blood, although the proportion of variance explained was greater than that explained by either buccal or nasal epithelial samples. Covariation across sample types was much higher for DNAm sites influenced by genetic factors. Overall, we observe that DNAm variation in whole blood is additively influenced by a combination of the major blood cell types. For a subset of sites, however, variable DNAm detected in whole blood can be attributed to variation in a single blood cell type providing potential mechanistic insight about EWAS findings. Our results suggest that associations between whole blood DNAm and traits or exposures reflect differences in multiple cell types and our data will facilitate the interpretation of findings in epigenetic epidemiology. Author summary As epigenetic variation is cell-type specific, an ongoing challenge in epigenetic epidemiology is how to interpret studies performed using bulk tissue (for example, whole blood) which comprises a mix of different cell types. In this study, we identified major differences in DNA methylation (DNAm) across multiple peripheral tissues and different blood cell types, with each sample type being characterized by a unique signature across multiple genomic loci. We demonstrate how these differences influence commonly used prediction scores derived from DNAm data for age and tobacco smoking, with estimates for the same individual being highly variable across tissues and cell types. Our results enabled us to assess the extent to which variable DNAm in each individual blood cell type relates to variation measured in whole blood. We found that although individual blood cell types predict more of the variation in DNAm in whole blood compared to buccal and nasal epithelial cells, the actual proportion of variance explained is relatively small, except for at sites where DNAm is under genetic control. Our data indicate that for most sites variation in multiple blood cell types additively combines to drive variation in DNAm measured in whole blood. Of note, for a subset of sites, variation in DNAm detected in whole blood can be attributed to a specific blood cell type, potentially facilitating the interpretation of EWAS findings.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000636043300001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.7554/eLife.58430" rel="external nofollow noopener" target="_blank">DNA methylation meta-analysis reveals cellular alterations in psychosis and markers of treatment-resistant schizophrenia</a></div> <div class="author"> Eilis Hannon, Emma L. Dempster, Georgina Mansell, and <span class="more-authors" title="click to view 45 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '45 more authors' ? 'Joe Burrage, Nick Bass, Marc M. Bohlken, Aiden Corvin, Charles J. Curtis, David Dempster, Marta Di Forti, Timothy G. Dinan, Gary Donohoe, Fiona Gaughran, Michael Gill, Amy Gillespie, Cerisse Gunasinghe, Hilleke E. Hulshoff, Christina M. Hultman, Viktoria Johansson, Rene S. Kahn, Jaakko Kaprio, Gunter Kenis, Kaarina Kowalec, James MacCabe, Colm McDonald, Andrew McQuillin, Derek W. Morris, Kieran C. Murphy, Colette J. Mustard, Igor Nenadic, Michael C. O’Donovan, Diego Quattrone, Alexander L. Richards, Bart Pf Rutten, David St Clair, Sebastian Therman, Timothea Toulopoulou, Jim Van Os, John L. Waddington, Patrick Sullivan, Evangelos Vassos, Gerome Breen, David Andrew Collier, Robin M. Murray, Leonard S. Schalkwyk, Jonathan Mill, Wellcome Trust Case Control Consor, CRESTAR Consortium' : '45 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">45 more authors</span> </div> <div class="periodical"> <em>ELIFE</em>, Feb 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.7554/eLife.58430" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.7554/eLife.58430" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>We performed a systematic analysis of blood DNA methylation profiles from 4483 participants from seven independent cohorts identifying differentially methylated positions (DMPs) associated with psychosis, schizophrenia, and treatment-resistant schizophrenia. Psychosis cases were characterized by significant differences in measures of blood cell proportions and elevated smoking exposure derived from the DNA methylation data, with the largest differences seen in treatment-resistant schizophrenia patients. We implemented a stringent pipeline to meta-analyze epigenome-wide association study (EWAS) results across datasets, identifying 95 DMPs associated with psychosis and 1048 DMPs associated with schizophrenia, with evidence of colocalization to regions nominated by genetic association studies of disease. Many schizophrenia-associated DNA methylation differences were only present in patients with treatment-resistant schizophrenia, potentially reflecting exposure to the atypical antipsychotic clozapine. Our results highlight how DNA methylation data can be leveraged to identify physiological (e.g., differential cell counts) and environmental (e.g., smoking) factors associated with psychosis and molecular biomarkers of treatment-resistant schizophrenia.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000616287700001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.2144/fsoa-2020-0161" rel="external nofollow noopener" target="_blank">The histone modification H3K4me3 is altered at the ANK1 locus in Alzheimer’s disease brain</a></div> <div class="author"> Adam R. Smith, Rebecca G. Smith, Ruby Macdonald, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Sarah J. Marzi, Joe Burrage, Claire Troakes, Safa Al-Sarraj, Jonathan Mill, Katie Lunnon' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>FUTURE SCIENCE OA</em>, Apr 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.2144/fsoa-2020-0161" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.2144/fsoa-2020-0161" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Lay abstract Alzheimer’s disease affects millions of people worldwide and numerous genetic association studies have been undertaken in the disease in recent years. The expression of genes can be altered by epigenetic processes, which include modifications to the DNA or histone proteins. Several studies have now reported increased DNA methylation in the ANK1 gene in Alzheimer’s disease brain samples. However, to date no studies have explored histone modifications in this gene in the disease. Here, we show decreased levels of the H3K4me3 modification in regions of the ANK1 gene in the entorhinal cortex of Alzheimer’s disease brain samples, which is a marker of active gene expression. This study further supports a role for epigenetic modifications in the ANK1 gene in Alzheimer’s disease pathology. Several epigenome-wide association studies of DNA methylation have highlighted altered DNA methylation in the ANK1 gene in Alzheimer’s disease (AD) brain samples. However, no study has specifically examined ANK1 histone modifications in the disease. We use chromatin immunoprecipitation-qPCR to quantify tri-methylation at histone 3 lysine 4 (H3K4me3) and 27 (H3K27me3) in the ANK1 gene in entorhinal cortex from donors with high (n = 59) or low (n = 29) Alzheimer’s disease pathology. We demonstrate decreased levels of H3K4me3, a marker of active gene transcription, with no change in H3K27me3, a marker of inactive genes. H3K4me3 is negatively correlated with DNA methylation in specific regions of the ANK1 gene. Our study suggests that the ANK1 gene shows altered epigenetic marks indicative of reduced gene activation in Alzheimer’s disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000654037100001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41380-020-00987-x" rel="external nofollow noopener" target="_blank">DNA methylation signatures of aggression and closely related constructs: A meta-analysis of epigenome-wide studies across the lifespan</a></div> <div class="author"> Jenny Dongen, Fiona A. Hagenbeek, Matthew Suderman, and <span class="more-authors" title="click to view 86 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '86 more authors' ? 'Peter J. Roetman, Karen Sugden, Andreas G. Chiocchetti, Khadeeja Ismail, Rosa H. Mulder, Jonathan D. Hafferty, Mark J. Adams, Rosie M. Walker, Stewart W. Morris, Jari Lahti, Leanne K. Kupers, Georgia Escaramis, Silvia Alemany, Marc Jan Bonder, Mandy Meijer, Hill F. Ip, Rick Jansen, Bart M. L. Baselmans, Priyanka Parmar, Estelle Lowry, Fabian Streit, Lea Sirignano, Tabea S. Send, Josef Frank, Juulia Jylhava, Yunzhang Wang, Pashupati Prasad Mishra, Olivier F. Colins, David L. Corcoran, Richie Poulton, Jonathan Mill, Eilis Hannon, Louise Arseneault, Tellervo Korhonen, Eero Vuoksimaa, Janine F. Felix, Marian J. Bakermans-Kranenburg, Archie Campbell, Darina Czamara, Elisabeth Binder, Eva Corpeleijn, Juan R. Gonzalez, Regina Grazuleviciene, Kristine B. Gutzkow, Jorunn Evandt, Marina Vafeiadi, Marieke Klein, Dennis Meer, Lannie Ligthart, Cornelis Kluft, Gareth E. Davies, Christian Hakulinen, Liisa Keltikangas-Jarvinen, Barbara Franke, Christine M. Freitag, Kerstin Konrad, Amaia Hervas, Aranzazu Fernandez-Rivas, Agnes Vetro, Olli Raitakari, Terho Lehtimaki, Robert Vermeiren, Timo Strandberg, Katri Raikkonen, Harold Snieder, Stephanie H. Witt, Michael Deuschle, Nancy L. Pedersen, Sara Hagg, Jordi Sunyer, Lude Franke, Jaakko Kaprio, Miina Ollikainen, Terrie E. Moffitt, Henning Tiemeier, Marinus H. IJzendoorn, Caroline Relton, Martine Vrijheid, Sylvain Sebert, Marjo-Riitta Jarvelin, Avshalom Caspi, Kathryn L. Evans, Andrew M. McIntosh, Meike Bartels, Dorret Boomsma, BIOS Consortium' : '86 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">86 more authors</span> </div> <div class="periodical"> <em>MOLECULAR PSYCHIATRY</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41380-020-00987-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41380-020-00987-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>DNA methylation profiles of aggressive behavior may capture lifetime cumulative effects of genetic, stochastic, and environmental influences associated with aggression. Here, we report the first large meta-analysis of epigenome-wide association studies (EWAS) of aggressive behavior (N = 15,324 participants). In peripheral blood samples of 14,434 participants from 18 cohorts with mean ages ranging from 7 to 68 years, 13 methylation sites were significantly associated with aggression (alpha = 1.2 x 10(-7); Bonferroni correction). In cord blood samples of 2425 children from five cohorts with aggression assessed at mean ages ranging from 4 to 7 years, 83% of these sites showed the same direction of association with childhood aggression (r = 0.74, p = 0.006) but no epigenome-wide significant sites were found. Top-sites (48 at a false discovery rate of 5% in the peripheral blood meta-analysis or in a combined meta-analysis of peripheral blood and cord blood) have been associated with chemical exposures, smoking, cognition, metabolic traits, and genetic variation (mQTLs). Three genes whose expression levels were associated with top-sites were previously linked to schizophrenia and general risk tolerance. At six CpGs, DNA methylation variation in blood mirrors variation in the brain. On average 44% (range = 3-82%) of the aggression-methylation association was explained by current and former smoking and BMI. These findings point at loci that are sensitive to chemical exposures with potential implications for neuronal functions. We hope these results to be a starting point for studies leading to applications as peripheral biomarkers and to reveal causal relationships with aggression and related traits.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000603824700001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1080/15592294.2020.1853317" rel="external nofollow noopener" target="_blank">DNA methylation signatures of adolescent victimization: analysis of a longitudinal monozygotic twin sample</a></div> <div class="author"> Radhika Kandaswamy, Eilis Hannon, Louise Arseneault, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Georgina Mansell, Karen Sugden, Benjamin Williams, Joe Burrage, James R. Staley, Ehsan Pishva, Aisha Dahir, Susanna Roberts, Andrea Danese, Jonathan Mill, Helen L. Fisher, Chloe C. Y. Wong' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>EPIGENETICS</em>, Nov 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1080/15592294.2020.1853317" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1080/15592294.2020.1853317" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Accumulating evidence suggests that individuals exposed to victimization at key developmental stages may have different epigenetic fingerprints compared to those exposed to no/minimal stressful events, however results are inconclusive. This study aimed to strengthen causal inference regarding the impact of adolescent victimization on the epigenome by controlling for genetic variation, age, gender, and shared environmental exposures. We conducted longitudinal epigenome-wide association analyses (EWAS) on DNA methylation (DNAm) profiles of 118 monozygotic (MZ) twin pairs from the Environmental Risk study with and without severe adolescent victimization generated using buccal DNA collected at ages 5, 10 and 18, and the Illumina EPIC array. Additionally, we performed cross-sectional EWAS on age-18 blood and buccal DNA from the same individuals to elucidate tissue-specific signatures of severe adolescent victimization. Our analyses identified 20 suggestive differentially methylated positions (DMPs) (P &lt; 5e-05), with altered DNAm trajectories between ages 10-18 associated with severe adolescent victimization ( increment Beta range = -5.5%-5.3%). Age-18 cross-sectional analyses revealed 72 blood ( increment Beta range = -2.2%-3.4%) and 42 buccal ( increment Beta range = -3.6%-4.6%) suggestive severe adolescent victimization-associated DMPs, with some evidence of convergent signals between these two tissue types. Downstream regional analysis identified significant differentially methylated regions (DMRs) in LGR6 and ANK3 (Sidak P = 5e-09 and 4.07e-06), and one upstream of CCL27 (Sidak P = 2.80e-06) in age-18 blood and buccal EWAS, respectively. Our study represents the first longitudinal MZ twin analysis of DNAm and severe adolescent victimization, providing initial evidence for altered DNA methylomic signatures in individuals exposed to adolescent victimization.</p> </div> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000713118700031" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/nargab/lqaa105" rel="external nofollow noopener" target="_blank">Cross-reactive probes on Illumina DNA methylation arrays: a large study on ALS shows that a cautionary approach is warranted in interpreting epigenome-wide association studies</a></div> <div class="author"> Paul J. Hop, Ramona A. J. Zwamborn, Eilis J. Hannon, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Annelot M. Dekker, Kristel R. Eijk, Emma M. Walker, Alfredo Iacoangeli, Ashley R. Jones, Aleksey Shatunov, Ahmad Al Khleifat, Sarah Opie-Martin, Christopher E. Shaw, Karen E. Morrison, Pamela J. Shaw, Russell L. McLaughlin, Orla Hardiman, Ammar Al-Chalabi, Leonard H. Berg, Jonathan Mill, Jan H. Veldink' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>NAR GENOMICS AND BIOINFORMATICS</em>, Dec 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/nargab/lqaa105" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/nargab/lqaa105" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Illumina DNA methylation arrays are a widely used tool for performing genome-wide DNA methylation analyses. However, measurements obtained from these arrays may be affected by technical artefacts that result in spurious associations if left unchecked. Cross-reactivity represents one of the major challenges, meaning that probes may map to multiple regions in the genome. Although several studies have reported on this issue, few studies have empirically examined the impact of cross-reactivity in an epigenome-wide association study (EWAS). In this paper, we report on cross-reactivity issues that we discovered in a large EWAS on the presence of the C9orf72 repeat expansion in ALS patients. Specifically, we found that that the majority of the significant probes inadvertently cross-hybridized to the C9orf72 locus. Importantly, these probes were not flagged as cross-reactive in previous studies, leading to novel insights into the extent to which cross-reactivity can impact EWAS. Our findings are particularly relevant for epigenetic studies into diseases associated with repeat expansions and other types of structural variation. More generally however, considering that most spurious associations were not excluded based on pre-defined sets of cross-reactive probes, we believe that the presented data-driven flag and consider approach is relevant for any type of EWAS.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000615920000033" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/brain/awaa334" rel="external nofollow noopener" target="_blank">Recalibrating the epigenetic clock: implications for assessing biological age in the human cortex</a></div> <div class="author"> Gemma L. Shireby, Jonathan P. Davies, Paul T. Francis, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Joe Burrage, Emma M. Walker, Grant W. A. Neilson, Aisha Dahir, Alan J. Thomas, Seth Love, Rebecca G. Smith, Katie Lunnon, Meena Kumari, Leonard C. Schalkwyk, Kevin Morgan, Keeley Brookes, Eilis Hannon, Jonathan Mill' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>BRAIN</em>, Dec 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/brain/awaa334" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/brain/awaa334" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Human DNA methylation data have been used to develop biomarkers of ageing, referred to as ‘epigenetic clocks’, which have been widely used to identify differences between chronological age and biological age in health and disease including neurodegeneration, dementia and other brain phenotypes. Existing DNA methylation clocks have been shown to be highly accurate in blood but are less precise when used in older samples or in tissue types not included in training the model, including brain. We aimed to develop a novel epigenetic clock that performs optimally in human cortex tissue and has the potential to identify phenotypes associated with biological ageing in the brain. We generated an extensive dataset of human cortex DNA methylation data spanning the life course (n=1397, ages = 1 to 108 years). This dataset was split into ‘training’ and ‘testing’ samples (training: n=1047; testing: n=350). DNA methylation age estimators were derived using a transformed version of chronological age on DNA methylation at specific sites using elastic net regression, a supervised machine learning method. The cortical clock was subsequently validated in a novel independent human cortex dataset (n=1221, ages = 41 to 104 years) and tested for specificity in a large whole blood dataset (n=1175, ages = 28 to 98 years). We identified a set of 347 DNA methylation sites that, in combination, optimally predict age in the human cortex. The sum of DNA methylation levels at these sites weighted by their regression coefficients provide the cortical DNA methylation clock age estimate. The novel clock dramatically outperformed previously reported clocks in additional cortical datasets. Our findings suggest that previous associations between predicted DNA methylation age and neurodegenerative phenotypes might represent false positives resulting from clocks not robustly calibrated to the tissue being tested and for phenotypes that become manifest in older ages. The age distribution and tissue type of samples included in training datasets need to be considered when building and applying epigenetic clock algorithms to human epidemiological or disease cohorts.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000583312900004" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neurobiolaging.2020.06.023" rel="external nofollow noopener" target="_blank">An epigenome-wide association study of Alzheimer’s disease blood highlights robust DNA hypermethylation in the HOXB6 gene</a></div> <div class="author"> Janou A. Y. Roubroeks, Adam R. Smith, Rebecca G. Smith, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Ehsan Pishva, Zina Ibrahim, Martina Sattlecker, Eilis J. Hannon, Iwona Kloszewska, Patrizia Mecocci, Hilkka Soininen, Magda Tsolaki, Bruno Vellas, Lars-Olof Wahlund, Dag Aarsland, Petroula Proitsi, Angela Hodges, Simon Lovestone, Stephen J. Newhouse, Richard J. B. Dobson, Jonathan Mill, Daniel L. A. Hove, Katie Lunnon' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF AGING</em>, Nov 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2020.06.023" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2020.06.023" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>A growing number of epigenome-wide association studies have demonstrated a role for DNA methylation in the brain in Alzheimer’s disease. With the aim of exploring peripheral biomarker potential, we have examined DNA methylation patterns in whole blood collected from 284 individuals in the Add-NeuroMed study, which included 89 nondemented controls, 86 patients with Alzheimer’s disease, and 109 individuals with mild cognitive impairment, including 38 individuals who progressed to Alzheimer’s disease within 1 year. We identified significant differentially methylated regions, including 12 adjacent hypermethylated probes in the HOXB6 gene in Alzheimer’s disease, which we validated using pyrosequencing. Using weighted gene correlation network analysis, we identified comethylated modules of genes that were associated with key variables such as APOE genotype and diagnosis. In summary, this study represents the first large-scale epigenome-wide association study of Alzheimer’s disease and mild cognitive impairment using blood. We highlight the differences in various loci and pathways in early disease, suggesting that these patterns relate to cognitive decline at an early stage. (C) 2020 The Authors. Published by Elsevier Inc.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000581004100122" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">DNA methylation signatures of a broad spectrum of aggressive behavior: a meta-analysis of epigenome-wide studies across the lifespan</a></div> <div class="author"> Jenny Dongen, Fiona A. Hagenbeek, Matthew Suderman, and <span class="more-authors" title="click to view 85 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '85 more authors' ? 'Peter Roetman, Karen Sugden, Andreas G. Chiocchetti, Khadeeja Ismail, Rosa H. Mulder, Jonathan Hafferty, Mark J. Adams, Rosie M. Walker, Stewart W. Morris, Jari Lahti, Leanne K. Kupers, Georgia Escaramis, Silvia Alemany, Marc Jan Bonder, Mandy Meijer, Hill F. Ip, Rick Jansen, Bart M. L. Baselmans, Priyanka Parmar, Estelle Lowry, Fabian Streit, Lea Sirignano, Tabea Send, Josef Frank, Juulia Jylhava, Yunzhang Wang, Pashupati Prasad Mishra, Olivier F. Colins, David Corcoran, Richie Poulton, Jonathan Mill, Eilis J. Hannon, Louise Arseneault, Tellervo Korhonen, Eero Vuoksimaa, Janine Felix, Marian Bakermans-Kranenburg, Archie Campbell, Darina Czamara, Elisabeth Binder, Eva Corpeleijn, Juan Ramon Gonzalez, Regina Grazuleviciene, Kristine B. Gutzkow, Jorunn Evandt, Marina Vafeiadi, Marieke Klein, Dennis Meer, Lannie Ligthart, Cornelis Kluft, Gareth E. Davies, Christian Hakulinen, Liisa Keltikangas-Jarvinen, Barbara Franke, Christine M. Freitag, Kerstin Konrad, Amaia Hervas, Aranzazu Fernandez-Rivas, Agnes Vetro, Olli Raitakari, Terho Lehtimaki, Robert Vermeiren, Timo Strandberg, Katri Raikkonen, Harold Snieder, Stephanie H. Witt, Michael Deuschle, Nancy L. Pedersen, Sara Hagg, Jordi Sunyer, Lude Franke, Jaakko Kaprio, Miina Ollikainen, Terrie E. Moffitt, Henning Tiemeier, Marinus H. Ijzendoorn, Caroline Relton, Martine Vrijheid, Sylvain Sebert, Marjo-Riitta Jarvelin, Avshalom Caspi, Kathryn L. Evans, Andrew M. McIntosh, Meike Bartels, Dorret Boomsma' : '85 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">85 more authors</span> </div> <div class="periodical"> <em>BEHAVIOR GENETICS</em>, Nov 2020 </div> <div class="periodical"> 50th Annual Meeting of the Behavior-Genetics-Association (BGA), ELECTR NETWORK, JUN 26, 2020 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000573737400005" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41467-020-18526-1" rel="external nofollow noopener" target="_blank">Integrative genomics identifies a convergent molecular subtype that links epigenomic with transcriptomic differences in autism</a></div> <div class="author"> Gokul Ramaswami, Hyejung Won, Michael J. Gandal, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Jillian Haney, Jerry C. Wang, Chloe C. Y. Wong, Wenjie Sun, Shyam Prabhakar, Jonathan Mill, Daniel H. Geschwind' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Sep 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41467-020-18526-1" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-020-18526-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Autism spectrum disorder (ASD) is a phenotypically and genetically heterogeneous neurodevelopmental disorder. Despite this heterogeneity, previous studies have shown patterns of molecular convergence in post-mortem brain tissue from autistic subjects. Here, we integrate genome-wide measures of mRNA expression, miRNA expression, DNA methylation, and histone acetylation from ASD and control brains to identify a convergent molecular subtype of ASD with shared dysregulation across both the epigenome and transcriptome. Focusing on this convergent subtype, we substantially expand the repertoire of differentially expressed genes in ASD and identify a component of upregulated immune processes that are associated with hypomethylation. We utilize eQTL and chromosome conformation datasets to link differentially acetylated regions with their cognate genes and identify an enrichment of ASD genetic risk variants in hyperacetylated noncoding regulatory regions linked to neuronal genes. These findings help elucidate how diverse genetic risk factors converge onto specific molecular processes in ASD. Autism spectrum disorder (ASD) is a neurodevelopmental disorder characterized by impaired social interactions with repetitive and restrictive behaviours. Here the authors integrate mRNA expression, miRNA expression, DNA methylation, and histone acetylation datasets from a collection of post mortem brain tissues and identify a convergent molecular subtype of ASD.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000566846900008" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1111/bpa.12879" rel="external nofollow noopener" target="_blank">The molecular etiology of Alzheimer’s disease</a></div> <div class="author"> Adam R. Smith, <em>Jonathan Mill</em>, and Katie Lunnon</div> <div class="periodical"> <em>BRAIN PATHOLOGY</em>, Sep 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1111/bpa.12879" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1111/bpa.12879" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alzheimer’s disease (AD) is a growing global healthcare epidemic. Owing to advances in technology, genome-scale studies of various layers of molecular information have been undertaken in recent years and robust variation in key loci have now been published and reproduced by others. This mini-symposium highlights four key areas of current research in the field of molecular biology in AD, including articles focused on large-scale genomic profiling, epigenetic research, integrative multi-omic approaches and how these can be appropriately modeled to address reverse causality. This mini-symposium provides a timely update on research focused on elucidating the molecular etiology of AD to date and highlights new methodological advances that could enable neuroscientists to identify novel therapeutic targets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000558637500016" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.stem.2020.06.018" rel="external nofollow noopener" target="_blank">Tissue-Biased Expansion of DNMT3A-Mutant Clones in a Mosaic Individual Is Associated with Conserved Epigenetic Erosion</a></div> <div class="author"> Ayala Tovy, Jaime M. Reyes, Michael C. Gundry, and <span class="more-authors" title="click to view 21 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '21 more authors' ? 'Lorenzo Brunetti, Henry Lee-Six, Mia Petljak, Hyun Jung Park, Anna G. Guzman, Carina Rosas, Aaron R. Jeffries, Emma Baple, Jonathan Mill, Andrew H. Crosby, Valerie Sency, Baozhong Xin, Heather E. Machado, Danielle Castillo, Jeffrey N. Weitzel, Wei Li, Michael R. Stratton, Peter J. Campbell, Heng Wang, Mathijs A. Sanders, Margaret A. Goodell' : '21 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">21 more authors</span> </div> <div class="periodical"> <em>CELL STEM CELL</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.stem.2020.06.018" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.stem.2020.06.018" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>DNA methyltransferase 3A (DNMT3A) is the most commonly mutated gene in clonal hematopoiesis (CH). Somatic DNMT3A mutations arise in hematopoietic stem cells (HSCs) many years before malignancies develop, but difficulties in comparing their impact before malignancy with wild-type cells have limited the understanding of their contributions to transformation. To circumvent this limitation, we derived normal and DNMT3A mutant lymphoblastoid cell lines from a germline mosaic individual in whom these cells co-existed for nearly 6 decades. Mutant cells dominated the blood system, but not other tissues. Deep sequencing revealed similar mutational burdens and signatures in normal and mutant clones, while epigenetic profiling uncovered the focal erosion of DNA methylation at oncogenic regulatory regions in mutant clones. These regions overlapped with those sensitive to DNMT3A loss after DNMT3A ablation in HSCs and in leukemia samples. These results suggest that DNMT3A maintains a conserved DNA methylation pattern, the erosion of which provides a distinct competitive advantage to hematopoietic cells.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000556543100045" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1126/sciadv.abc5801" rel="external nofollow noopener" target="_blank">Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia</a></div> <div class="author"> David H. Brann, Tatsuya Tsukahara, Caleb Weinreb, and <span class="more-authors" title="click to view 22 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '22 more authors' ? 'Marcela Lipovsek, Koen Berge, Boying Gong, Rebecca Chance, Iain C. Macaulay, Hsin-Jung Chou, Russell B. Fletcher, Diya Das, Kelly Street, Hector Roux Bezieux, Yoon-Gi Choi, Davide Risso, Sandrine Dudoit, Elizabeth Purdom, Jonathan Mill, Ralph Abi Hachem, Hiroaki Matsunami, Darren W. Logan, Bradley J. Goldstein, Matthew S. Grubb, John Ngai, Sandeep Robert Datta' : '22 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">22 more authors</span> </div> <div class="periodical"> <em>SCIENCE ADVANCES</em>, Jul 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1126/sciadv.abc5801" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.abc5801" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Altered olfactory function is a common symptom of COVID-19 (coronavirus disease 2019), but its etiology is unknown. A key question is whether SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2; CoV-2)-the causal agent in COVID-19-affects olfaction directly, by infecting olfactory sensory neurons or their targets in the olfactory bulb, or indirectly, by perturbing support cells. Bulk and single-cell RNA sequencing revealed that support and stem cells in the human and mouse olfactory epithelium and vascular pericytes in the mouse olfactory bulb express angiotensin-converting enzyme 2 (ACE2), which is essential for CoV-2 cell entry. In contrast, ACE2 was not detected in either olfactory sensory neurons or olfactory bulb neurons. Immunostaining confirmed these results and revealed pervasive expression of ACE2 protein in dorsally located olfactory epithelial sustentacular cells and mouse olfactory bulb pericytes. These findings suggest that CoV-2 infection of non-neuronal cell types leads to olfactory dysfunction in patients with COVID-19.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000544062000001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-020-0860-4" rel="external nofollow noopener" target="_blank">Epigenome-wide association study of attention-deficit/hyperactivity disorder in adults</a></div> <div class="author"> Paula Rovira, Cristina Sanchez-Mora, Mireia Pagerols, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Vanesa Richarte, Montserrat Corrales, Christian Fadeuilhe, Laura Vilar-Ribo, Lorena Arribas, Gemma Shireby, Eilis Hannon, Jonathan Mill, Miquel Casas, Josep Antoni Ramos-Quiroga, Maria Soler Artigas, Marta Ribases' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Jun 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-020-0860-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-020-0860-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Attention-deficit/hyperactivity disorder (ADHD) is a highly heritable neurodevelopmental disorder that often persists into adulthood. There is growing evidence that epigenetic dysregulation participates in ADHD. Given that only a limited number of epigenome-wide association studies (EWASs) of ADHD have been conducted so far and they have mainly focused on pediatric and population-based samples, we performed an EWAS in a clinical sample of adults with ADHD. We report one CpG site and four regions differentially methylated between patients and controls, which are located in or near genes previously involved in autoimmune diseases, cancer or neuroticism. Our sensitivity analyses indicate that smoking status is not responsible for these results and that polygenic risk burden for ADHD does not greatly impact the signatures identified. Additionally, we show an overlap of our EWAS findings with genetic signatures previously described for ADHD and with epigenetic signatures for smoking behavior and maternal smoking. These findings support a role of DNA methylation in ADHD and emphasize the need for additional efforts in larger samples to clarify the role of epigenetic mechanisms on ADHD across the lifespan.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000540432200001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1001/jamanetworkopen.2020.6095" rel="external nofollow noopener" target="_blank">Association of Neighborhood Disadvantage in Childhood With DNA Methylation in Young Adulthood</a></div> <div class="author"> Aaron Reuben, Karen Sugden, Louise Arseneault, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'David L. Corcoran, Andrea Danese, Helen L. Fisher, Terrie E. Moffitt, Joanne B. Newbury, Candice Odgers, Joey Prinz, Line J. H. Rasmussen, Ben Williams, Jonathan Mill, Avshalom Caspi' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>JAMA NETWORK OPEN</em>, Jun 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1001/jamanetworkopen.2020.6095" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1001/jamanetworkopen.2020.6095" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Importance DNA methylation has been proposed as an epigenetic mechanism by which the childhood neighborhood environment may have implications for the genome that compromise adult health. Objective To ascertain whether childhood neighborhood socioeconomic disadvantage is associated with differences in DNA methylation by age 18 years. Design, Setting, and Participants This longitudinal cohort study analyzed data from the Environmental Risk (E-Risk) Longitudinal Twin Study, a nationally representative birth cohort of children born between 1994 and 1995 in England and Wales and followed up from age 5 to 18 years. Data analysis was performed from March 15, 2019, to June 30, 2019. Exposures High-resolution neighborhood data (indexing deprivation, dilapidation, disconnection, and dangerousness) collected across childhood. Main Outcomes and Measures DNA methylation in whole blood was drawn at age 18 years. Associations between neighborhood socioeconomic disadvantage and methylation were tested using 3 prespecified approaches: (1) testing probes annotated to candidate genes involved in biological responses to growing up in socioeconomically disadvantaged neighborhoods and investigated in previous epigenetic research (stress reactivity-related and inflammation-related genes), (2) polyepigenetic scores indexing differential methylation in phenotypes associated with growing up in disadvantaged neighborhoods (obesity, inflammation, and smoking), and (3) a theory-free epigenome-wide association study. Results A total of 1619 participants (806 female individuals [50%]) had complete neighborhood and DNA methylation data. Children raised in socioeconomically disadvantaged neighborhoods exhibited differential DNA methylation in genes involved in inflammation (beta = 0.12; 95% CI, 0.06-0.19; P &lt; .001) and smoking (beta = 0.18; 95% CI, 0.11-0.25; P &lt; .001) but not obesity (beta = 0.05; 95% CI, -0.01 to 0.11; P = .12). An epigenome-wide association study identified multiple CpG sites at an arraywide significance level of P &lt; 1.16 x 10(-7) in genes involved in the metabolism of hydrocarbons. Associations between neighborhood disadvantage and methylation were small but robust to family-level socioeconomic factors and to individual-level tobacco smoking. Conclusions and Relevance Children raised in more socioeconomically disadvantaged neighborhoods appeared to enter young adulthood epigenetically distinct from their less disadvantaged peers. This finding suggests that epigenetic regulation may be a mechanism by which the childhood neighborhood environment alters adult health. Question Is childhood neighborhood disadvantage associated with differential DNA methylation? Findings In this cohort study of 1619 children in Great Britain, exposure to neighborhood socioeconomic disadvantage during childhood was associated with differential DNA methylation at age 18 years in genes involved in inflammation, exposure to tobacco smoke, and metabolism of toxic air pollutants. Meaning The study found that children who were raised in socioeconomically disadvantaged neighborhoods appeared to enter young adulthood epigenetically distinct from their more advantaged peers. This cohort study traces the biological responses and associated phenotypes of an upbringing in a socially and economically disadvantaged environment.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000653823200002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.patter.2020.100014" rel="external nofollow noopener" target="_blank">Patterns of Reliability: Assessing the Reproducibility and Integrity of DNA Methylation Measurement</a></div> <div class="author"> Karen Sugden, Eilis J. Hannon, Louise Arseneault, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Daniel W. Belsky, David L. Corcoran, Helen L. Fisher, Renate M. Houts, Radhika Kandaswamy, Terrie E. Moffitt, Richie Poulton, Joseph A. Prinz, Line J. H. Rasmussen, Benjamin S. Williams, Chloe C. Y. Wong, Jonathan Mill, Avshalom Caspi' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>PATTERNS</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.patter.2020.100014" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.patter.2020.100014" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>DNA methylation plays an important role in both normal human development and risk of disease. The most utilized method of assessing DNA methylation uses BeadChips, generating an epigenome-wide “snapshot” of &gt;450,000 observations (probe measurements) per assay. However, the reliability of each of these measurements is not equal, and little consideration is paid to consequences for research. We correlated repeat measurements of the same DNA samples using the Illumina HumanMethylation450K and the Infinium Methylation-EPIC BeadChips in 350 blood DNA samples. Probes that were reliably measured were more heritable and showed consistent associations with environmental exposures, gene expression, and greater cross-tissue concordance. Unreliable probes were less replicable and generated an unknown volume of false negatives. This serves as a lesson for working with DNA methylation data, but the lessons are equally applicable to working with other data: as we advance toward generating increasingly greater volumes of data, failure to document reliability risks harming reproducibility.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000541448900001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.7554/eLife.54870" rel="external nofollow noopener" target="_blank">Quantification of the pace of biological aging in humans through a blood test, the DunedinPoAm DNA methylation algorithm</a></div> <div class="author"> Daniel W. Belsky, Avshalom Caspi, Louise Arseneault, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Andrea Baccarelli, David L. Corcoran, Xu Gao, Eiliss Hannon, Hona Lee Harrington, Line J. H. Rasmussen, Renate Houts, Kim Huffman, William E. Kraus, Dayoon Kwon, Jonathan Mill, Carl F. Pieper, Joseph A. Prinz, Richie Poulton, Joel Schwartz, Karen Sugden, Pantel Vokonas, Benjamin S. Williams, Terrie E. Moffitt' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>ELIFE</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.7554/eLife.54870" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.7554/eLife.54870" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Biological aging is the gradual, progressive decline in system integrity that occurs with advancing chronological age, causing morbidity and disability. Measurements of the pace of aging are needed as surrogate endpoints in trials of therapies designed to prevent disease by slowing biological aging. We report a blood-DNA-methylation measure that is sensitive to variation in pace of biological aging among individuals born the same year. We first modeled change-over-time in 18 biomarkers tracking organ-system integrity across 12 years of follow-up in n = 954 members of the Dunedin Study born in 1972-1973. Rates of change in each biomarker over ages 26-38 years were composited to form a measure of aging-related decline, termed Pace-of-Aging. Elastic-net regression was used to develop a DNA-methylation predictor of Pace-of-Aging, called DunedinPoAm for Dunedin(P)ace(o)f(A)ging(m)ethylation. Validation analysis in cohort studies and the CALERIE trial provide proof-of-principle for DunedinPoAm as a single-time-point measure of a person’s pace of biological aging.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000522552900009" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neurobiolaging.2020.01.001" rel="external nofollow noopener" target="_blank">Psychosis-associated DNA methylomic variation in Alzheimer’s disease cortex</a></div> <div class="author"> Ehsan Pishva, Byron Creese, Adam R. Smith, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Wolfgang Viechtbauer, Petroula Proitsi, Daniel L. A. Hove, Clive Ballard, Jonathan Mill, Katie Lunnon' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF AGING</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2020.01.001" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2020.01.001" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Psychotic symptoms are a common and debilitating feature of Alzheimer’s disease (AD) and are associated with a more rapid course of decline. Current evidence from postmortem and neuroimaging studies implicates frontal, temporal, and parietal lobes, with reported disruptions in monoaminergic pathways. However, the molecular mechanisms underlying this remain unclear. In the present study, we investigated methylomic variation associated with AD psychosis in 3 key brain regions implicated in the etiology of psychosis (prefrontal cortex, entorhinal cortex, and superior temporal gyrus) in postmortem brain samples from 29 AD donors with psychosis and 18 matched AD donors without psychosis. We identified psychosis-associated methylomic changes in a number of loci, with these genes being enriched in known schizophrenia-associated genetic and epigenetic variants. One of these known loci resided in the AS3MT gene-previously implicated in schizophrenia in a large GWAS meta-analysis. We used bisulfite-pyrosequencing to confirm hypomethylation across 4 neighboring CpG sites in the ASM3T gene. Finally, our regional analysis nominated multiple CpG sites in TBX15 and WT1, which are genes that have been previously implicated in AD. Thus one potential implication from our study is whether psychosis-associated variation drives reported associations in AD case-control studies. (C) 2020 Published by Elsevier Inc.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000563459800005" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41598-020-62262-x" rel="external nofollow noopener" target="_blank">Major surgery induces acute changes in measured DNA methylation associated with immune response pathways</a></div> <div class="author"> Ryoichi Sadahiro, Bridget Knight, Ffion James, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Eilis Hannon, John Charity, Ian R. Daniels, Joe Burrage, Olivia Knox, Bethany Crawford, Neil J. Smart, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>SCIENTIFIC REPORTS</em>, Apr 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41598-020-62262-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-020-62262-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Surgery is an invasive procedure evoking acute inflammatory and immune responses that can influence risk for postoperative complications including cognitive dysfunction and delirium. Although the specific mechanisms driving these responses have not been well-characterized, they are hypothesized to involve the epigenetic regulation of gene expression. We quantified genome-wide levels of DNA methylation in peripheral blood mononuclear cells (PBMCs) longitudinally collected from a cohort of elderly patients undergoing major surgery, comparing samples collected at baseline to those collected immediately post-operatively and at discharge from hospital. We identified acute changes in measured DNA methylation at sites annotated to immune system genes, paralleling changes in serum-levels of markers including C-reactive protein (CRP) and Interleukin 6 (IL-6) measured in the same individuals. Many of the observed changes in measured DNA methylation were consistent across different types of major surgery, although there was notable heterogeneity between surgery types at certain loci. The acute changes in measured DNA methylation induced by surgery are relatively stable in the postoperative period, generally persisting until discharge from hospital. Our results highlight the dramatic alterations in gene regulation induced by invasive surgery, primarily reflecting upregulation of the immune system in response to trauma, wound healing and anaesthesia.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000525861400015" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/schbul/sbz056" rel="external nofollow noopener" target="_blank">Independent Methylome-Wide Association Studies of Schizophrenia Detect Consistent Case-Control Differences</a></div> <div class="author"> Robin F. Chan, Andrey A. Shabalin, Carolina Montano, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Eilis Hannon, Christina M. Hultman, Margaret D. Fallin, Andrew P. Feinberg, Jonathan Mill, Edwin J. C. G. Oord, Karolina A. Aberg' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>SCHIZOPHRENIA BULLETIN</em>, Mar 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/schbul/sbz056" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/schbul/sbz056" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Methylome-wide association studies (MWASs) are promising complements to sequence variation studies. We used existing sequencing-based methylation data, which assayed the majority of all 28 million CpGs in the human genome, to perform an MWAS for schizophrenia in blood, while controlling for cell-type heterogeneity with a recently generated platform-specific reference panel. Next, we compared the MWAS results with findings from 3 existing large-scale array-based schizophrenia methylation studies in blood that assayed up to similar to 450 000 CpGs. Our MWAS identified 22 highly significant loci ( P &lt; 5 x 10(-8)) and 852 suggestively significant loci (P &lt; 1 x 10(-5)). The top finding (P = 5.62 x 10(-11), q = 0.001) was located in MFN2, which encodes mitofusin-2 that regulates Ca2+ transfer from the endoplasmic reticulum to mitochondria in cooperation with DISC1. The second-most significant site (P = 1.38 x 10(-9), q = 0.013) was located in ALDH1A2, which encodes an enzyme for astrocyte-derived retinoic acid-a key neuronal morphogen with relevance for schizophrenia. Although the most significant MWAS findings were not assayed on the arrays, we observed significant enrichment of overlapping findings with 2 of the 3 array datasets (P = 0.0315, 0.0045, 0.1946). Overrepresentation analysis of Gene Ontology terms for the genes in the significant overlaps suggested high similarity in the biological functions detected by the different datasets. Top terms were related to immune and/or stress responses, cell adhesion and motility, and a broad range of processes essential for neurodevelopment.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000516908200001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41525-020-0118-3" rel="external nofollow noopener" target="_blank">Significant out-of-sample classification from methylation profile scoring for amyotrophic lateral sclerosis</a></div> <div class="author"> Marta F. Nabais, Tian Lin, Beben Benyamin, and <span class="more-authors" title="click to view 34 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '34 more authors' ? 'Kelly L. Williams, Fleur C. Garton, Anna A. E. Vinkhuyzen, Futao Zhang, Costanza L. Vallerga, Restuadi Restuadi, Anna Freydenzon, Ramona A. J. Zwamborn, Paul J. Hop, Matthew R. Robinson, Jacob Gratten, Peter M. Visscher, Eilis Hannon, Jonathan Mill, Matthew A. Brown, Nigel G. Laing, Karen A. Mather, Perminder S. Sachdev, Shyuan T. Ngo, Frederik J. Steyn, Leanne Wallace, Anjali K. Henders, Merrilee Needham, Jan H. Veldink, Susan Mathers, Garth Nicholson, Dominic B. Rowe, Robert D. Henderson, Pamela A. McCombe, Roger Pamphlett, Jian Yang, Ian P. Blair, Allan F. McRae, Naomi R. Wray' : '34 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">34 more authors</span> </div> <div class="periodical"> <em>NPJ GENOMIC MEDICINE</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41525-020-0118-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41525-020-0118-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>We conducted DNA methylation association analyses using Illumina 450K data from whole blood for an Australian amyotrophic lateral sclerosis (ALS) case-control cohort (782 cases and 613 controls). Analyses used mixed linear models as implemented in the OSCA software. We found a significantly higher proportion of neutrophils in cases compared to controls which replicated in an independent cohort from the Netherlands (1159 cases and 637 controls). The OSCA MOMENT linear mixed model has been shown in simulations to best account for confounders. When combined in a methylation profile score, the 25 most-associated probes identified by MOMENT significantly classified case-control status in the Netherlands sample (area under the curve, AUC = 0.65, CI95% = [0.62-0.68], p = 8.3 x 10(-22)). The maximum AUC achieved was 0.69 (CI95% = [0.66-0.71], p = 4.3 x 10(-34)) when cell-type proportion was included in the predictor.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000517152200001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-020-0752-7" rel="external nofollow noopener" target="_blank">Genome-wide DNA methylation meta-analysis in the brains of suicide completers</a></div> <div class="author"> Stefania Policicchio, Sam Washer, Joana Viana, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Artemis Iatrou, Joe Burrage, Eilis Hannon, Gustavo Turecki, Zachary Kaminsky, Jonathan Mill, Emma L. Dempster, Therese M. Murphy' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-020-0752-7" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-020-0752-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Suicide is the second leading cause of death globally among young people representing a significant global health burden. Although the molecular correlates of suicide remains poorly understood, it has been hypothesised that epigenomic processes may play a role. The objective of this study was to identify suicide-associated DNA methylation changes in the human brain by utilising previously published and unpublished methylomic datasets. We analysed prefrontal cortex (PFC, n = 211) and cerebellum (CER, n = 114) DNA methylation profiles from suicide completers and non-psychiatric, sudden-death controls, meta-analysing data from independent cohorts for each brain region separately. We report evidence for altered DNA methylation at several genetic loci in suicide cases compared to controls in both brain regions with suicide-associated differentially methylated positions enriched among functional pathways relevant to psychiatric phenotypes and suicidality, including nervous system development (PFC) and regulation of long-term synaptic depression (CER). In addition, we examined the functional consequences of variable DNA methylation within a PFC suicide-associated differentially methylated region (PSORS1C3 DMR) using a dual luciferase assay and examined expression of nearby genes. DNA methylation within this region was associated with decreased expression of firefly luciferase but was not associated with expression of nearby genes, PSORS1C3 and POU5F1. Our data suggest that suicide is associated with DNA methylation, offering novel insights into the molecular pathology associated with suicidality.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000513920800031" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.celrep.2020.01.063" rel="external nofollow noopener" target="_blank">Transcriptional Signatures of Tau and Amyloid Neuropathology</a></div> <div class="author"> Isabel Castanho, Tracey K. Murray, Eilis Hannon, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Aaron Jeffries, Emma Walker, Emma Laing, Hedley Baulf, Joshua Harvey, Lauren Bradshaw, Andrew Randall, Karen Moore, Paul O’Neill, Katie Lunnon, David A. Collier, Zeshan Ahmed, Michael J. O’Neill, Jonathan Mill' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>CELL REPORTS</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.celrep.2020.01.063" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.celrep.2020.01.063" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alzheimer’s disease (AD) is associated with the intracellular aggregation of hyperphosphorylated tau and the accumulation of b-amyloid in the neocortex. We use transgenic mice harboring human tau (rTg4510) and amyloid precursor protein (J20) mutations to investigate transcriptional changes associated with the progression of tau and amyloid pathology. rTg4510 mice are characterized by widespread transcriptional differences in the entorhinal cortex with changes paralleling neuropathological burden across multiple brain regions. Differentially expressed transcripts overlap with genes identified in genetic studies of familial and sporadic AD. Systems-level analyses identify discrete co-expression networks associated with the progressive accumulation of tau that are enriched for genes and pathways previously implicated in AD pathology and overlap with co-expression networks identified in human AD cortex. Our data provide further evidence for an immune-response component in the accumulation of tau and reveal molecular pathways associated with the progression of AD neuropathology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000510945300001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41537-019-0092-x" rel="external nofollow noopener" target="_blank">Clozapine-induced transcriptional changes in the zebrafish brain</a></div> <div class="author"> Joana Viana, Nick Wildman, Eilis Hannon, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Audrey Farbos, Paul O’Neill, Karen Moore, Ronny Van Aerle, Greg Paull, Eduarda Santos, Jonathan Mill' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>NPJ SCHIZOPHRENIA</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41537-019-0092-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41537-019-0092-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Clozapine is an atypical antipsychotic medication that is used to treat schizophrenia patients who are resistant to other antipsychotic drugs. The molecular mechanisms mediating the effects of clozapine are not well understood and its use is often associated with severe side-effects. In this study, we exposed groups of wild-type zebrafish to two doses of clozapine (’low’ (20 mu g/L) and ‘high’ (70 mu g/L)) over a 72-h period, observing dose-dependent effects on behaviour. Using RNA sequencing (RNA-seq) we identified multiple genes differentially expressed in the zebrafish brain following exposure to clozapine. Network analysis identified co-expression modules characterised by striking changes in module connectivity in response to clozapine, and these were enriched for regulatory pathways relevant to the etiology of schizophrenia. Our study highlights the utility of zebrafish as a model for assessing the molecular consequences of antipsychotic medications and identifies genomic networks potentially involved in schizophrenia.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000514593900001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.3389/fgene.2020.00016" rel="external nofollow noopener" target="_blank">Genome-Wide DNA Methylation Patterns in Persistent Attention-Deficit/Hyperactivity Disorder and in Association With Impulsive and Callous Traits</a></div> <div class="author"> Mandy Meijer, Marieke Klein, Eilis Hannon, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Dennis Meer, Catharina Hartman, Jaap Oosterlaan, Dirk Heslenfeld, Pieter J. Hoekstra, Jan Buitelaar, Jonathan Mill, Barbara Franke' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>FRONTIERS IN GENETICS</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.3389/fgene.2020.00016" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fgene.2020.00016" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Attention-deficit/hyperactivity disorder (ADHD) is a neurodevelopmental disorder that often persists into adulthood. ADHD and related personality traits, such as impulsivity and callousness, are caused by genetic and environmental factors and their interplay. Epigenetic modifications of DNA, including methylation, are thought to mediate between such factors and behavior and may behave as biomarkers for disorders. Here, we set out to study DNA methylation in persistent ADHD and related traits. We performed epigenome-wide association studies (EWASs) on peripheral whole blood from participants in the NeuroIMAGE study (age range 12-23 years). We compared participants with persistent ADHD (n = 35) with healthy controls (n = 19) and with participants with remittent ADHD (n = 19). Additionally, we performed EWASs of impulsive and callous traits derived from the Conners Parent Rating Scale and the Callous-Unemotional Inventory, respectively, across all participants. For every EWAS, the linear regression model analyzed included covariates for age, sex, smoking scores, and surrogate variables reflecting blood cell type composition and genetic background. We observed no epigenome-wide significant differences in single CpG site methylation between participants with persistent ADHD and healthy controls or participants with remittent ADHD. However, epigenome-wide analysis of differentially methylated regions provided significant findings showing that hypermethylated regions in the APOB and LPAR5 genes were associated with ADHD persistence compared to ADHD remittance (p = 1.68 * 10(-24) and p = 9.06 * 10(-7), respectively); both genes are involved in cholesterol signaling. Both findings appeared to be linked to genetic variation in cis. We found neither significant epigenome-wide single CpG sites nor regions associated with impulsive and callous traits; the top-hits from these analyses were annotated to genes involved in neurotransmitter release and the regulation of the biological clock. No link to genetic variation was observed for these findings, which thus might reflect environmental influences. In conclusion, in this pilot study with a small sample size, we observed several DNA-methylation-disorder/trait associations of potential significance for ADHD and the related behavioral traits. Although we do not wish to draw conclusions before replication in larger, independent samples, cholesterol signaling and metabolism may be of relevance for the onset and/or persistence of ADHD.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000515871200001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-020-0710-4" rel="external nofollow noopener" target="_blank">Large epigenome-wide association study of childhood ADHD identifies peripheral DNA methylation associated with disease and polygenic risk burden</a></div> <div class="author"> Michael A. Mooney, Peter Ryabinin, Beth Wilmot, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Priya Bhatt, Jonathan Mill, Joel T. Nigg' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-020-0710-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-020-0710-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Epigenetic variation in peripheral tissues is being widely studied as a molecular biomarker of complex disease and disease-related exposures. To date, few studies have examined differences in DNA methylation associated with attention-deficit hyperactivity disorder (ADHD). In this study, we profiled genetic and methylomic variation across the genome in saliva samples from children (age 7-12 years) with clinically established ADHD (N=391) and nonpsychiatric controls (N=213). We tested for differentially methylated positions (DMPs) associated with both ADHD diagnosis and ADHD polygenic risk score, by using linear regression models including smoking, medication effects, and other potential confounders in our statistical models. Our results support previously reported associations between ADHD and DNA methylation levels at sites annotated to VIPR2, and identify several novel disease-associated DMPs (p&lt;1e-5), although none of them were genome-wide significant. The two top-ranked, ADHD-associated DMPs (cg17478313 annotated to SLC7A8 and cg21609804 annotated to MARK2) are also significantly associated with nearby SNPs (p=1.2e-46 and p=2.07e-59), providing evidence that disease-associated DMPs are under genetic control. We also report a genome-wide significant association between ADHD polygenic risk and variable DNA methylation at a site annotated to the promoter of GART and SON (p=6.71E-8). Finally, we show that ADHD-associated SNPs colocalize with SNPs associated with methylation levels in saliva. This is the first large-scale study of DNA methylation in children with ADHD. Our results represent novel epigenetic biomarkers for ADHD that may be useful for patient stratification, reinforce the importance of genetic effects on DNA methylation, and provide plausible molecular mechanisms for ADHD risk variants.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000639431800105" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/braincomms/fcaa167" rel="external nofollow noopener" target="_blank">Genetic risk for Alzheimer’s disease influences neuropathology via multiple biological pathways</a></div> <div class="author"> Eilis Hannon, Gemma L. Shireby, Keeley Brookes, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Johannes Attems, Rebecca Sims, Nigel J. Cairns, Seth Love, Alan J. Thomas, Kevin Morgan, Paul T. Francis, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>BRAIN COMMUNICATIONS</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/braincomms/fcaa167" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/braincomms/fcaa167" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Alzheimer’s disease is a highly heritable, common neurodegenerative disease characterized neuropathologically by the accumulation of beta-amyloid plaques and tau-containing neurofibrillary tangles. In addition to the well-established risk associated with the APOE locus, there has been considerable success in identifying additional genetic variants associated with Alzheimer’s disease. Major challenges in understanding how genetic risk influences the development of Alzheimer’s disease are clinical and neuropathological heterogeneity, and the high level of accompanying comorbidities. We report a multimodal analysis integrating longitudinal clinical and cognitive assessment with neuropathological data collected as part of the Brains for Dementia Research study to understand how genetic risk factors for Alzheimer’s disease influence the development of neuropathology and clinical performance. Six hundred and ninety-three donors in the Brains for Dementia Research cohort with genetic data, semi-quantitative neuropathology measurements, cognitive assessments and established diagnostic criteria were included in this study. We tested the association of APOE genotype and Alzheimer’s disease polygenic risk score-a quantitative measure of genetic burden-with survival, four common neuropathological features in Alzheimer’s disease brains (neurofibrillary tangles, beta-amyloid plaques, Lewy bodies and transactive response DNA-binding protein 43 proteinopathy), clinical status (clinical dementia rating) and cognitive performance (Mini-Mental State Exam, Montreal Cognitive Assessment). The APOE epsilon 4 allele was significantly associated with younger age of death in the Brains for Dementia Research cohort. Our analyses of neuropathology highlighted two independent pathways from APOE epsilon 4, one where beta-amyloid accumulation co-occurs with the development of tauopathy, and a second characterized by direct effects on tauopathy independent of beta-amyloidosis. Although we also detected association between APOE epsilon 4 and dementia status and cognitive performance, these were all mediated by tauopathy, highlighting that they are a consequence of the neuropathological changes. Analyses of polygenic risk score identified associations with tauopathy and beta-amyloidosis, which appeared to have both shared and unique contributions, suggesting that different genetic variants associated with Alzheimer’s disease affect different features of neuropathology to different degrees. Taken together, our results provide insight into how genetic risk for Alzheimer’s disease influences both the clinical and pathological features of dementia, increasing our understanding about the interplay between APOE genotype and other genetic risk factors.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000751652300066" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1002/trc2.12078" rel="external nofollow noopener" target="_blank">Whole transcriptome in silico screening implicates cardiovascular and infectious disease in the mechanism of action underlying atypical antipsychotic side effects</a></div> <div class="author"> Yasaman Malekizadeh, Gareth Williams, Mark Kelson, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'David Whitfield, Jonathan Mill, David A. Collier, Clive Ballard, Aaron R. Jeffries, Byron Creese' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>ALZHEIMERS &amp; DEMENTIA-TRANSLATIONAL RESEARCH &amp; CLINICAL INTERVENTIONS</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/trc2.12078" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/trc2.12078" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Stroke/thromboembolic events, infections, and death are all significantly increased by antipsychotics in dementia but little is known about why they can be harmful. Using a novel application of a drug repurposing paradigm, we aimed to identify potential mechanisms underlying adverse events. Methods: Whole transcriptome signatures were generated for SH-SY5Y cells treated with amisulpride, risperidone, and volinanserin using RNA sequencing. Bioinformatic analysis was performed that scored the association between antipsychotic signatures and expression data from 415,252 samples in the National Center for Biotechnology Information Gene Expression Omnibus (NCBI GEO) repository. Results: Atherosclerosis, venous thromboembolism, and influenza NCBI GEO-derived samples scored positively against antipsychotic signatures. Pathways enriched in antipsychotic signatures were linked to the cardiovascular and immune systems (eg, brain derived neurotrophic factor [BDNF], platelet derived growth factor receptor [PDGFR]-beta, tumor necrosis factor [TNF], transforming growth factor [TGF]-beta, selenoamino acid metabolism, and influenza infection). Conclusions: These findings for the first time mechanistically link antipsychotics to specific cardiovascular and infectious diseases which are known side effects of their use in dementia, providing new information to explain related adverse events.</p> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000511886200001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13059-019-1810-4" rel="external nofollow noopener" target="_blank">Systematic underestimation of the epigenetic clock and age acceleration in older subjects</a></div> <div class="author"> Louis Y. El Khoury, Tyler Gorrie-Stone, Melissa Smart, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Amanda Hughes, Yanchun Bao, Alexandria Andrayas, Joe Burrage, Eilis Hannon, Meena Kumari, Jonathan Mill, Leonard C. Schalkwyk' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>GENOME BIOLOGY</em>, Dec 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-019-1810-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-019-1810-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: The Horvath epigenetic clock is widely used. It predicts age quite well from 353 CpG sites in the DNA methylation profile in unknown samples and has been used to calculate “age acceleration” in various tissues and environments. Results: The model systematically underestimates age in tissues from older people. This is seen in all examined tissues but most strongly in the cerebellum and is consistently observed in multiple datasets. Age acceleration is thus age-dependent, and this can lead to spurious associations. The current literature includes examples of association tests with age acceleration calculated in a wide variety of ways. Conclusions: The concept of an epigenetic clock is compelling, but caution should be taken in interpreting associations with age acceleration. Association tests of age acceleration should include age as a covariate.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000499452500002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13148-019-0755-5" rel="external nofollow noopener" target="_blank">Alzheimer’s disease-associated (hydroxy)methylomic changes in the brain and blood</a></div> <div class="author"> Roy Lardenoije, Janou A. Y. Roubroeks, Ehsan Pishva, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Markus Leber, Holger Wagner, Artemis Iatrou, Adam R. Smith, Rebecca G. Smith, Lars M. T. Eijssen, Luca Kleineidam, Amit Kawalia, Per Hoffmann, Tobias Luck, Steffi Riedel-Heller, Frank Jessen, Wolfgang Maier, Michael Wagner, Rene Hurlemann, Gunter Kenis, Muhammad Ali, Antonio Sol, Diego Mastroeni, Elaine Delvaux, Paul D. Coleman, Jonathan Mill, Bart P. F. Rutten, Katie Lunnon, Alfredo Ramirez, Daniel L. A. Hove' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>CLINICAL EPIGENETICS</em>, Nov 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13148-019-0755-5" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13148-019-0755-5" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background Late-onset Alzheimer’s disease (AD) is a complex multifactorial affliction, the pathogenesis of which is thought to involve gene-environment interactions that might be captured in the epigenome. The present study investigated epigenome-wide patterns of DNA methylation (5-methylcytosine, 5mC) and hydroxymethylation (5-hydroxymethylcytosine, 5hmC), as well as the abundance of unmodified cytosine (UC), in relation to AD. Results We identified epigenetic differences in AD patients (n = 45) as compared to age-matched controls (n = 35) in the middle temporal gyrus, pertaining to genomic regions close to or overlapping with genes such as OXT (- 3.76% 5mC, p(Sidak) = 1.07E-06), CHRNB1 (+ 1.46% 5hmC, p(Sidak) = 4.01E-04), RHBDF2 (- 3.45% UC, p(Sidak) = 4.85E-06), and C3 (- 1.20% UC, p(Sidak) = 1.57E-03). In parallel, in an independent cohort, we compared the blood methylome of converters to AD dementia (n = 54) and non-converters (n = 42), at a preclinical stage. DNA methylation in the same region of the OXT promoter as found in the brain was found to be associated with subsequent conversion to AD dementia in the blood of elderly, non-demented individuals (+ 3.43% 5mC, p(Sidak) = 7.14E-04). Conclusions The implication of genome-wide significant differential methylation of OXT, encoding oxytocin, in two independent cohorts indicates it is a promising target for future studies on early biomarkers and novel therapeutic strategies in AD.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000495630000001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13229-019-0285-1" rel="external nofollow noopener" target="_blank">RNA sequencing of identical twins discordant for autism reveals blood-based signatures implicating immune and transcriptional dysregulation</a></div> <div class="author"> Ayden Saffari, Matt Arno, Eric Nasser, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Angelica Ronald, Chloe C. Y. Wong, Leonard C. Schalkwyk, Jonathan Mill, Frank Dudbridge, Emma L. Meaburn' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>MOLECULAR AUTISM</em>, Nov 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13229-019-0285-1" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13229-019-0285-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>BackgroundA gap exists in our mechanistic understanding of how genetic and environmental risk factors converge at the molecular level to result in the emergence of autism symptoms. We compared blood-based gene expression signatures in identical twins concordant and discordant for autism spectrum condition (ASC) to differentiate genetic and environmentally driven transcription differences, and establish convergent evidence for biological mechanisms involved in ASC.MethodsGenome-wide gene expression data were generated using RNA-seq on whole blood samples taken from 16 pairs of monozygotic (MZ) twins and seven twin pair members (39 individuals in total), who had been assessed for ASC and autism traits at age 12. Differential expression (DE) analyses were performed between (a) affected and unaffected subjects (N=36) and (b) within discordant ASC MZ twin pairs (total N=11) to identify environmental-driven DE. Gene set enrichment and pathway testing was performed on DE gene lists. Finally, an integrative analysis using DNA methylation data aimed to identify genes with consistent evidence for altered regulation in cis.ResultsIn the discordant twin analysis, three genes showed evidence for DE at FDR&lt;10%: IGHG4, EVI2A and SNORD15B. In the case-control analysis, four DE genes were identified at FDR&lt;10% including IGHG4, PRR13P5, DEPDC1B, and ZNF501. We find enrichment for DE of genes curated in the SFARI human gene database. Pathways showing evidence of enrichment included those related to immune cell signalling and immune response, transcriptional control and cell cycle/proliferation. Integrative methylomic and transcriptomic analysis identified a number of genes showing suggestive evidence for cis dysregulation.LimitationsIdentical twins stably discordant for ASC are rare, and as such the sample size was limited and constrained to the use of peripheral blood tissue for transcriptomic and methylomic profiling. Given these primary limitations, we focused on transcript-level analysis.ConclusionsUsing a cohort of ASC discordant and concordant MZ twins, we add to the growing body of transcriptomic-based evidence for an immune-based component in the molecular aetiology of ASC. Whilst the sample size was limited, the study demonstrates the utility of the discordant MZ twin design combined with multi-omics integration for maximising the potential to identify disease-associated molecular signals.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000487249300007" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.biopsych.2019.02.016" rel="external nofollow noopener" target="_blank">Epigenome-wide Association Study of Attention-Deficit/Hyperactivity Disorder Symptoms in Adults</a></div> <div class="author"> Jenny Dongen, Nuno R. Zilhao, Karen Sugden, and <span class="more-authors" title="click to view 60 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '60 more authors' ? 'Eilis J. Hannon, Jonathan Mill, Avshalom Caspi, Jessica Agnew-Blais, Louise Arseneault, David L. Corcoran, Terrie E. Moffitt, Richie Poulton, Barbara Franke, Dorret Boomsma, Bastiaan T. Heijmans, Peter A. G. t’Hoen, Joyce Meurs, Aaron Isaacs, Rick Jansen, Lude Franke, Rene Pool, Jouke J. Hottenga, Marleen M. J. Greevenbroek, Coen D. A. Stehouwer, Carla J. H. Kallen, Casper G. Schalkwijk, Cisca Wijmenga, Sasha Zhernakova, Ettje F. Tigchelaar, P. Eline Slagboom, Marian Beekman, Joris Deelen, Diana Heemst, Jan H. Veldink, Leonard H. Berg, Cornelia M. Duijn, Bert A. Hofman, Andre G. Uitterlinden, P. Mila Jhamai, Michael Verbiest, H. Eka D. Suchiman, Marijn Verkerk, Ruud Breggen, Jeroen Rooij, Nico Lakenberg, Hailiang Mei, Maarten Iterson, Michiel Galen, Jan Bot, Dasha Zhernakova, Peter t’Hof, Patrick Deelen, Irene Nooren, Matthijs Moed, Martijn Vermaat, Rene Luijk, Marc Jan Bonder, Freerk Dijk, Wibowo Arindrarto, Szymon M. Kielbasa, Morris A. Swertz, Erik W. Zwet, Peter-Bram t’Hoen, BIOS Consortium' : '60 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">60 more authors</span> </div> <div class="periodical"> <em>BIOLOGICAL PSYCHIATRY</em>, Oct 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.biopsych.2019.02.016" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.biopsych.2019.02.016" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>BACKGROUND: Previous studies have reported associations between attention-deficit/hyperactivity disorder symptoms and DNA methylation in children. We report the first epigenome-wide association study meta-analysis of adult attention-deficit/hyperactivity disorder symptoms, based on peripheral blood DNA methylation (Infinium HunnanMethylation450K array) in three population-based adult cohorts. METHODS: An epigenome-wide association study was performed in the Netherlands Twin Register (N = 2258, mean age 37 years), Dunedin Multidisciplinary Health and Development Study (N = 800, age 38 years), and Environmental Risk Longitudinal Twin Study (N = 1631, age 18 years), and results were combined through meta-analysis (total sample size N = 4689). Region-based analyses accounting for the correlation between nearby methylation sites were also performed. RESULTS: One epigenome-wide significant differentially methylated position was detected in the Dunedin study, but meta-analysis did not detect differentially methylated positions that were robustly associated across cohorts. In region-based analyses, six significant differentially methylation regions (DMRs) were identified in the Netherlands Twin Register, 19 in the Dunedin study, and none in the Environmental Risk Longitudinal Twin Study. Of these DMRs, 92% were associated with methylation quantitative trait loci, and 68% showed moderate to large blood-brain correlations for DNA methylation levels. DMRs included six nonoverlapping DMRs (three in the Netherlands Twin Register, three in the Dunedin study) in the major histocompatibility complex, which were associated with expression of genes in the major histocompatibility complex, including C4A and C4B, previously implicated in schizophrenia. CONCLUSIONS: Our findings point at new candidate loci involved in immune and neuronal functions that await further replication. Our work also illustrates the need for further research to examine to what extent epigenetic associations with psychiatric traits depend on characteristics such as age, comorbidities, exposures, and genetic background.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000488216600434" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2019.08.247" rel="external nofollow noopener" target="_blank">LARGE-SCALE EWAS AND METHYLATION QTL ANALYSIS IN ADHD</a></div> <div class="author"> Michael Mooney, Peter Ryabinin, Priya Bhatt, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jonathan Mill, Beth Wilmot, Joel Nigg' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2019 </div> <div class="periodical"> 27th World Congress of Psychiatric Genetics (WCPG), Los Angeles, CA, OCT 26-31, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2019.08.247" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2019.08.247" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000488216600144" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2019.07.171" rel="external nofollow noopener" target="_blank">SCHIZOPHRENIA IS CHARACTERIZED BY AGE- AND SEX-SPECIFIC EFFECTS ON EPIGENETIC AGING</a></div> <div class="author"> Anil Ori, Loes Olde Loohuis, Jerry Guintivano, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'David St Clair, Andrew McQuillin, Jonathan Mill, Patrick Sullivan, Rene Kahn, Steve Horvath, Roel Ophoff' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2019 </div> <div class="periodical"> 27th World Congress of Psychiatric Genetics (WCPG), Los Angeles, CA, OCT 26-31, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2019.07.171" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2019.07.171" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000488216600255" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2019.08.051" rel="external nofollow noopener" target="_blank">MAPPING CELL-TYPE SPECIFIC MARKERS OF GENOMIC REGULATION IN THE HUMAN BRAIN</a></div> <div class="author"> Stefania Policicchio, Jonathan Davies, Barry Choiza, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Eilis Hannon, Joe Burrage, Emma Dempster, Jonathan Mill' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2019 </div> <div class="periodical"> 27th World Congress of Psychiatric Genetics (WCPG), Los Angeles, CA, OCT 26-31, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2019.08.051" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2019.08.051" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000488216600344" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2019.08.146" rel="external nofollow noopener" target="_blank">METHYLOMIC CONSEQUENCES OF POLYGENIC BURDEN FOR NEUROPSYCHIATRIC, NEURODEGENERATIVE AND NEUROLOGICAL DISORDERS</a></div> <div class="author"> Gemma Shireby, Eilis Hannon, Emma Dempster, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Paul Francis, Katie Lunnon, Emma Walker, Jonathan Mill' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2019 </div> <div class="periodical"> 27th World Congress of Psychiatric Genetics (WCPG), Los Angeles, CA, OCT 26-31, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2019.08.146" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2019.08.146" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000488216600256" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2019.08.052" rel="external nofollow noopener" target="_blank">RECALIBRATING THE EPIGENETIC CLOCK: APPLICATIONS FOR ASSESSING BIOLOGICAL AGEING IN THE HUMAN BRAIN</a></div> <div class="author"> Gemma Shireby, Eilis Hannon, Paul Francis, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Katie Lunnon, Emma Dempster, Emma Walker, Jonathan Mill' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2019 </div> <div class="periodical"> 27th World Congress of Psychiatric Genetics (WCPG), Los Angeles, CA, OCT 26-31, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2019.08.052" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2019.08.052" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000473730600002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1101/gr.243584.118" rel="external nofollow noopener" target="_blank">Growth disrupting mutations in epigenetic regulatory molecules are associated with abnormalities of epigenetic aging</a></div> <div class="author"> Aaron R. Jeffries, Reza Maroofian, Claire G. Salter, and <span class="more-authors" title="click to view 23 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '23 more authors' ? 'Barry A. Chioza, Harold E. Cross, Michael A. Patton, Emma Dempster, I. Karen Temple, Deborah J. G. Mackay, Faisal I. Rezwan, Lise Aksglaede, Diana Baralle, Tabib Dabir, Matthew F. Hunter, Arveen Kamath, Ajith Kumar, Ruth Newbury-Ecob, Angelo Selicorni, Amanda Springer, Lionel Van Maldergem, Vinod Varghese, Naomi Yachelevich, Katrina Tatton-Brown, Jonathan Mill, Andrew H. Crosby, Emma L. Baple' : '23 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">23 more authors</span> </div> <div class="periodical"> <em>GENOME RESEARCH</em>, Jul 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1101/gr.243584.118" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/gr.243584.118" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Germline mutations in fundamental epigenetic regulatory molecules including DNA methyltransferase 3 alpha (DNMT3A) are commonly associated with growth disorders, whereas somatic mutations are often associated with malignancy. We profiled genome-wide DNA methylation patterns in DNMT3A c.2312G &gt; A; p.(Arg771Gln) carriers in a large Amish sibship with Tatton-Brown-Rahman syndrome (TBRS), their mosaic father, and 15 TBRS patients with distinct pathogenic de novo DNMT3A variants. This defined widespread DNA hypomethylation at specific genomic sites enriched at locations annotated as genes involved in morphogenesis, development, differentiation, and malignancy predisposition pathways. TBRS patients also displayed highly accelerated DNA methylation aging. These findings were most marked in a carrier of the AML-associated driver mutation p.Arg882Cys. Our studies additionally defined phenotype-related accelerated and decelerated epigenetic aging in two histone methyltransferase disorders: NSD1 Sotos syndrome overgrowth disorder and KMT2D Kabuki syndrome growth impairment. Together, our findings provide fundamental new insights into aberrant epigenetic mechanisms, the role of epigenetic machinery maintenance, and determinants of biological aging in these growth disorders.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000470933400001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-019-0489-3" rel="external nofollow noopener" target="_blank">Methylation age acceleration does not predict mortality in schizophrenia</a></div> <div class="author"> Kaarina Kowalec, His Hannon, Georgina Mansell, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Joe Burrage, Anil P. S. Ori, Roel A. Ophoff, Jonathan Mill, Patrick F. Sullivan' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-019-0489-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-019-0489-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Schizophrenia (SCZ) is associated with high mortality. DNA methylation levels vary over the life course, and pre-selected combinations of methylation array probes can be used to estimate “methylation age” (mAge). mAge correlates highly with chronological age but when it differs, termed mAge acceleration, it has been previously associated with all-cause mortality. We tested the association between mAge acceleration and mortality in SCZ and controls. We selected 190 SCZ cases and 190 controls from the Sweden Schizophrenia Study. Cases were identified from the Swedish Hospital Discharge Register with &gt;= 5 specialist treatment contacts and &gt;= 5 antipsychotic prescriptions. Controls had no psychotic disorder or antipsychotics. Subjects were selected if they had died or survived during follow-up (2:1 oversampling). Extracted DNA was assayed on the Illumina MethylationEPIC array. mAge was regressed on age at sampling to obtain mAge acceleration. Using Cox proportional hazards regression, the association between mAge acceleration and mortality was tested. After quality control, the following were available: n = 126 SCZ died, 63 SCZ alive, 127 controls died, 62 controls alive. In the primary analyses, we did not find a significant association between mAge acceleration and SCZ mortality (adjusted p &gt; 0.005). Sensitivity analyses excluding SCZ cases with pre-existing cancer demonstrated a significant association between the Hannum mAge acceleration and mortality (hazard ratio = 1.13, 95% confidence interval = 1.04-1.22, p = 0.005). Per our pre-specified criteria, we did not confirm our primary hypothesis that mAge acceleration would predict subsequent mortality in people with SCZ, but we cannot rule out smaller effects or effects in patient subsets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000471237800014" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41562-019-0562-1" rel="external nofollow noopener" target="_blank">Genetics and the geography of health, behaviour and attainment</a></div> <div class="author"> Daniel W. Belsky, Avshalom Caspi, Louise Arseneault, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'David L. Corcoran, Benjamin W. Domingue, Kathleen Mullan Harris, Renate M. Houts, Jonathan S. Mill, Terrie E. Moffitt, Joseph Prinz, Karen Sugden, Jasmin Wertz, Benjamin Williams, Candice L. Odgers' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>NATURE HUMAN BEHAVIOUR</em>, Jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41562-019-0562-1" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41562-019-0562-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Young people’s life chances can be predicted by characteristics of their neighbourhood’. Children growing up in disadvantaged neighbourhoods exhibit worse physical and mental health and suffer poorer educational and economic outcomes than children growing up in advantaged neighbourhoods. Increasing recognition that aspects of social inequalities tend, in fact, to be geographical inequalities 2-5 is stimulating research and focusing policy interest on the role of place in shaping health, behaviour and social outcomes. Where neighbourhood effects are causal, neighbourhood-level interventions can be effective. Where neighbourhood effects reflect selection of families with different characteristics into different neighbourhoods, interventions should instead target families or individuals directly. To test how selection may affect different neighbourhood-linked problems, we linked neighbourhood data with genetic, health and social outcome data for &gt;7,000 European-descent UK and US young people in the E-Risk and Add Health studies. We tested selection/concentration of genetic risks for obesity, schizophrenia, teen pregnancy and poor educational outcomes in high-risk neighbourhoods, including genetic analysis of neighbourhood mobility. Findings argue against genetic selection/concentration as an explanation for neighbourhood gradients in obesity and mental health problems. By contrast, modest genetic selection/concentration was evident for teen pregnancy and poor educational outcomes, suggesting that neighbourhood effects for these outcomes should be interpreted with care.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000472661000347" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.biopsych.2019.03.363" rel="external nofollow noopener" target="_blank">Alzheimer’s Disease DNA (Hydroxy)Methylome in the Brain and Blood: Evidence for OXT Methylation as a Preclinical Marker</a></div> <div class="author"> Roy Lardenoije, Janou Roubroeks, Ehsan Pishva, and <span class="more-authors" title="click to view 25 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '25 more authors' ? 'Markus Leber, Holger Wagner, Artemis Iatrou, Adam Smith, Rebecca Smith, Lars Eijssen, Luca Kleineidam, Amit Kawalia, Per Hoffmann, Tobias Luck, Steffi Riedel-Heller, Frank Jessen, Wolfgang Maier, Michael Wagner, Gunter Kenis, Muhammad Ali, Antonio Sol, Diego Mastroeni, Elaine Delvaux, Paul Coleman, Jonathan Mill, Bart Rutten, Katie Lunnon, Alfredo Ramirez, Daniel Hove' : '25 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">25 more authors</span> </div> <div class="periodical"> <em>BIOLOGICAL PSYCHIATRY</em>, May 2019 </div> <div class="periodical"> 74th Annual Meeting of the Society-of-Biological-Psychiatry (SOBP), Chicago, IL, MAY 16-18, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.biopsych.2019.03.363" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.biopsych.2019.03.363" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000468050300008" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s12864-019-5761-7" rel="external nofollow noopener" target="_blank">Guidance for DNA methylation studies: statistical insights from the Illumina EPIC array</a></div> <div class="author"> Georgina Mansell, Tyler J. Gorrie-Stone, Yanchun Bao, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Meena Kumari, Leonard S. Schalkwyk, Jonathan Mill, Eilis Hannon' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>BMC GENOMICS</em>, May 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s12864-019-5761-7" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-019-5761-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>BackgroundThere has been a steady increase in the number of studies aiming to identify DNA methylation differences associated with complex phenotypes. Many of the challenges of epigenetic epidemiology regarding study design and interpretation have been discussed in detail, however there are analytical concerns that are outstanding and require further exploration. In this study we seek to address three analytical issues. First, we quantify the multiple testing burden and propose a standard statistical significance threshold for identifying DNA methylation sites that are associated with an outcome. Second, we establish whether linear regression, the chosen statistical tool for the majority of studies, is appropriate and whether it is biased by the underlying distribution of DNA methylation data. Finally, we assess the sample size required for adequately powered DNA methylation association studies.ResultsWe quantified DNA methylation in the Understanding Society cohort (n=1175), a large population based study, using the Illumina EPIC array to assess the statistical properties of DNA methylation association analyses. By simulating null DNA methylation studies, we generated the distribution of p-values expected by chance and calculated the 5% family-wise error for EPIC array studies to be 9x10(-8). Next, we tested whether the assumptions of linear regression are violated by DNA methylation data and found that the majority of sites do not satisfy the assumption of normal residuals. Nevertheless, we found no evidence that this bias influences analyses by increasing the likelihood of affected sites to be false positives. Finally, we performed power calculations for EPIC based DNA methylation studies, demonstrating that existing studies with data on similar to 1000 samples are adequately powered to detect small differences at the majority of sites.ConclusionWe propose that a significance threshold of P&lt;9x10(-8) adequately controls the false positive rate for EPIC array DNA methylation studies. Moreover, our results indicate that linear regression is a valid statistical methodology for DNA methylation studies, despite the fact that the data do not always satisfy the assumptions of this test. These findings have implications for epidemiological-based studies of DNA methylation and provide a framework for the interpretation of findings from current and future studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000460487000011" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1098/rstb.2018.0120" rel="external nofollow noopener" target="_blank">Variable DNA methylation in neonates mediates the association between prenatal smoking and birth weight</a></div> <div class="author"> Eilis Hannon, Diana Schendel, Christine Ladd-Acosta, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Jakob Grove, Christine Soholm Hansen, David Michael Hougaard, Michaeline Bresnahan, Ole Mors, Mads Vilhelm Hollegaard, Marie Baekvad-Hansen, Mady Hornig, Preben Bo Mortensen, Anders D. Borglum, Thomas Werge, Marianne Giortz Pedersen, Merete Nordentoft, Joseph D. Buxbaum, M. Daniele Fallin, Jonas Bybjerg-Grauholm, Abraham Reichenberg, Jonathan Mill, ASD Grp' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>PHILOSOPHICAL TRANSACTIONS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1098/rstb.2018.0120" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1098/rstb.2018.0120" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>There is great interest in the role epigenetic variation induced by non-genetic exposures may play in the context of health and disease. In particular, DNA methylation has previously been shown to be highly dynamic during the earliest stages of development and is influenced by in utero exposures such as maternal smoking and medication. In this study we sought to identify the specific DNA methylation differences in blood associated with prenatal and birth factors, including birth weight, gestational age and maternal smoking. We quantified neonatal methylomic variation in 1263 infants using DNA isolated from a unique collection of archived blood spots taken shortly after birth (mean = 6.08 days; s.d. = 3.24 days). An epigenome-wide association study (EWAS) of gestational age and birth weight identified 4299 and 18 differentially methylated positions (DMPs) respectively, at an experiment-wide significance threshold of p &lt; 1 x 10(-7). Our EWAS of maternal smoking during pregnancy identified 110 DMPs in neonatal blood, replicating previously reported genomic loci, including AHRR. Finally, we tested the hypothesis that DNA methylation mediates the relationship between maternal smoking and lower birth weight, finding evidence that methylomic variation at three DMPs may link exposure to outcome. These findings complement an expanding literature on the epigenomic consequences of prenatal exposures and obstetric factors,</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000466725400084" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/schbul/sbz022.082" rel="external nofollow noopener" target="_blank">DNA METHYLATION PROFILING MIGHT SHED LIGHT ON THE BIOLOGY OF CANNABIS ASSOCIATED PSYCHOSIS</a></div> <div class="author"> Marta Di Forti, Emma Dempster, Diego Quattrone, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Giada Tripoli, Radhika Kandaswamy, Craig Morgan, Jim Os, Bart Rutten, Robin Murray, Jonathan Mill, Chloe Wong, Rajiv Radhakrishnan' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>SCHIZOPHRENIA BULLETIN</em>, Apr 2019 </div> <div class="periodical"> Congress of the Schizophrenia-International-Research-Society (SRIS), Orlando, FL, APR 10-14, 2019 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/schbul/sbz022.082" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/schbul/sbz022.082" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462257900001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13148-019-0636-y" rel="external nofollow noopener" target="_blank">Parallel profiling of DNA methylation and hydroxymethylation highlights neuropathology-associated epigenetic variation in Alzheimer’s disease</a></div> <div class="author"> Adam R. Smith, Rebecca G. Smith, Ehsan Pishva, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Eilis Hannon, Janou A. Y. Roubroeks, Joe Burrage, Claire Troakes, Safa Al-Sarraj, Carolyn Sloan, Jonathan Mill, Daniel L. Hove, Katie Lunnon' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>CLINICAL EPIGENETICS</em>, Mar 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13148-019-0636-y" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13148-019-0636-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Alzheimer’s disease is a progressive neurodegenerative disorder that is hypothesized to involve epigenetic dysfunction. Previous studies of DNA modifications in Alzheimer’s disease have been unable to distinguish between DNA methylation and DNA hydroxymethylation. DNA hydroxymethylation has been shown to be enriched in the human brain, although its role in Alzheimer’s disease has not yet been fully explored. Here, we utilize oxidative bisulfite conversion, in conjunction with the Illumina Infinium Human Methylation 450K microarray, to identify neuropathology-associated differential DNA methylation and DNA hydroxymethylation in the entorhinal cortex. Results: We identified one experiment-wide significant differentially methylated position residing in the WNT5B gene. Next, we investigated pathology-associated regions consisting of multiple adjacent loci. We identified one significant differentially hydroxymethylated region consisting of four probes spanning 104 bases in the FBXL16 gene. We also identified two significant differentially methylated regions: one consisting of two probes in a 93 base-pair region in the ANK1 gene and the other consisting of six probes in a 99-base pair region in the ARID5B gene. We also highlighted three regions that show alterations in unmodified cytosine: two probes in a 39-base pair region of ALLC, two probes in a 69-base pair region in JAG2, and the same six probes in ARID5B that were differentially methylated. Finally, we replicated significant ANK1 disease-associated hypermethylation and hypohydroxymethylation patterns across eight CpG sites in an extended 118-base pair region in an independent cohort using oxidative-bisulfite pyrosequencing. Conclusions: Our study represents the first epigenome-wide association study of both DNA methylation and hydroxymethylation in Alzheimer’s disease entorhinal cortex. We demonstrate that previous estimates of DNA hypermethylation in ANK1 in Alzheimer’s disease were underestimates as it is confounded by hypohydroxymethylation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462709200010" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/bioinformatics/bty713" rel="external nofollow noopener" target="_blank">Bigmelon: tools for analysing large DNA methylation datasets</a></div> <div class="author"> Tyler J. Gorrie-Stone, Melissa C. Smart, Ayden Saffari, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Karim Malki, Eilis Hannon, Joe Burrage, Jonathan Mill, Meena Kumari, Leonard C. Schalkwyk' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>BIOINFORMATICS</em>, Mar 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/bioinformatics/bty713" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/bty713" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Motivation The datasets generated by DNA methylation analyses are getting bigger. With the release of the HumanMethylationEPIC micro-array and datasets containing thousands of samples, analyses of these large datasets using R are becoming impractical due to large memory requirements. As a result there is an increasing need for computationally efficient methodologies to perform meaningful analysis on high dimensional data. Results Here we introduce the bigmelon R package, which provides a memory efficient workflow that enables users to perform the complex, large scale analyses required in epigenome wide association studies (EWAS) without the need for large RAM. Building on top of the CoreArray Genomic Data Structure file format and libraries packaged in the gdsfmt package, we provide a practical workflow that facilitates the reading-in, preprocessing, quality control and statistical analysis of DNA methylation data. We demonstrate the capabilities of the bigmelon package using a large dataset consisting of 1193 human blood samples from the Understanding Society: UK Household Longitudinal Study, assayed on the EPIC micro-array platform. Availability and implementation The bigmelon package is available on Bioconductor (http://bioconductor.org/packages/bigmelon/). The Understanding Society dataset is available at https://www.understandingsociety.ac.uk/about/health/data upon request. Supplementary information Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000459835300002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-019-0430-9" rel="external nofollow noopener" target="_blank">Establishing a generalized polyepigenetic biomarker for tobacco smoking</a></div> <div class="author"> Karen Sugden, Eilis J. Hannon, Louise Arseneault, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Daniel W. Belsky, Jonathan M. Broadbent, David L. Corcoran, Robert J. Hancox, Renate M. Houts, Terrie E. Moffitt, Richie Poulton, Joseph A. Prinz, W. Murray Thomson, Benjamin S. Williams, Chloe C. Y. Wong, Jonathan Mill, Avshalom Caspi' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-019-0430-9" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-019-0430-9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Large-scale epigenome-wide association meta-analyses have identified multiple ‘signatures” of smoking. Drawing on these findings, we describe the construction of a polyepigenetic DNA methylation score that indexes smoking behavior and that can be utilized for multiple purposes in population health research. To validate the score, we use data from two birth cohort studies: The Dunedin Longitudinal Study, followed to age-38 years, and the Environmental Risk Study, followed to age-18 years. Longitudinal data show that changes in DNA methylation accumulate with increased exposure to tobacco smoking and attenuate with quitting. Data from twins discordant for smoking behavior show that smoking influences DNA methylation independently of genetic and environmental risk factors. Physiological data show that changes in DNA methylation track smoking-related changes in lung function and gum health over time. Moreover, DNA methylation changes predict corresponding changes in gene expression in pathways related to inflammation, immune response, and cellular trafficking. Finally, we present prospective data about the link between adverse childhood experiences (ACEs) and epigenetic modifications; these findings document the importance of controlling for smoking-related DNA methylation changes when studying biological embedding of stress in life-course research. We introduce the polyepigenetic DNA methylation score as a tool both for discovery and theory-guided research in epigenetic epidemiology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000459834400010" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-019-0407-8" rel="external nofollow noopener" target="_blank">Longitudinal investigation of DNA methylation changes preceding adolescent psychotic experiences</a></div> <div class="author"> Susanna Roberts, Matthew Suderman, Stanley Zammit, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Sarah H. Watkins, Eilis Hannon, Jonathan Mill, Caroline Relton, Louise Arseneault, Chloe C. Y. Wong, Helen L. Fisher' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-019-0407-8" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-019-0407-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Childhood psychotic experiences (PEs), such as seeing or hearing things that others do not, or extreme paranoia, are relatively common with around 1 in 20 children reporting them at age 12. Childhood PEs are often distressing and can be predictive of schizophrenia, other psychiatric disorders, and suicide attempts in adulthood, particularly if they persist during adolescence. Previous research has demonstrated that methylomic signatures in blood could be potential biomarkers of psychotic phenomena. This study explores the association between DNA methylation (DNAm) and the emergence, persistence, and remission of PEs in childhood and adolescence. DNAm profiles were obtained from the ALSPAC cohort at birth, age 7, and age 15/17 (n = 901). PEs were assessed through interviews with participants at ages 12 and 18. We identified PE-associated probes (p &lt; 5x10-5) and regions (corrected p &lt; 0.05) at ages 12 and 18. Several of the differentially methylated probes were also associated with the continuity of PEs across adolescence. One probe (cg16459265), detected consistently at multiple timepoints in the study sample, was replicated in an independent sample of twins (n = 1658). Six regions, including those spanning the HLA-DBP2 and GDF7 genes, were consistently differentially methylated at ages 7 and 15-17. Findings from this large, population-based study suggest that DNAm at multiple stages of development may be associated with PEs in late childhood and adolescence, though further replication is required. Research uncovering biomarkers associated with pre-clinical PEs is important as it has the potential to facilitate early identification of individuals at increased risk who could benefit from preventive interventions.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000460620100030" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/aje/kwy247" rel="external nofollow noopener" target="_blank">RE: “SOCIOECONOMIC POSITION AND DNA METHYLATION AGE ACCELERATION ACROSS THE LIFE COURSE” REPLY</a></div> <div class="author"> Amanda Hughes, Yanchun Bao, Melissa Smart, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Tyler Gorrie-Stone, Eilis Hannon, Jonathan Mill, Joe Burrage, Leo Schalkwyk, Meena Kumari' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>AMERICAN JOURNAL OF EPIDEMIOLOGY</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/aje/kwy247" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/aje/kwy247" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000460484300016" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.cobeha.2018.12.003" rel="external nofollow noopener" target="_blank">Epigenetic studies of schizophrenia: current status and future directions</a></div> <div class="author"> Anna Migdalska-Richards, and <em>Jonathan Mill</em> </div> <div class="periodical"> <em>CURRENT OPINION IN BEHAVIORAL SCIENCES</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.cobeha.2018.12.003" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cobeha.2018.12.003" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Schizophrenia is a chronic neuropsychiatric disorder characterized by episodes of psychosis and altered cognitive functioning. Epigenetic processes, which developmentally regulate gene expression via modifications to DNA, histone proteins, and chromatin, have been widely heralded as the ‘missing piece’ of the etiological puzzle for many neuropsychiatric and neurodevelopmental disorders, including schizophrenia. In this review, we discuss the current state of knowledge of epigenetic mechanisms in schizophrenia, highlighting how future work integrating these data with recent genetic findings will facilitate our understanding of schizophrenia pathophysiology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000455193900007" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neurobiolaging.2018.09.024" rel="external nofollow noopener" target="_blank">A cross-brain regions study of ANK1 DNA methylation in different neurodegenerative diseases</a></div> <div class="author"> Adam R. Smith, Rebecca G. Smith, Joe Burrage, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Claire Troakes, Safa Al-Sarraj, Rajesh N. Kalaria, Carolyn Sloan, Andrew C. Robinson, Jonathan Mill, Katie Lunnon' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF AGING</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2018.09.024" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2018.09.024" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Recent epigenome-wide association studies in Alzheimer’s disease have highlighted consistent robust neuropathology-associated DNA hypermethylation of the ankyrin 1 (ANK1) gene in the cortex. The extent to which altered ANK1 DNA methylation is also associated with other neurodegenerative diseases is not currently known. In the present study, we used bisulfite pyrosequencing to quantify DNA methylation across 8 CpG sites within a 118 bp region of the ANK1 gene across multiple brain regions in Alzheimer’s disease, Vascular dementia, Dementia with Lewy bodies, Huntington’s disease, and Parkinson’s disease. We demonstrate disease-associated ANK1 hypermethylation in the entorhinal cortex in Alzheimer’s disease, Huntington’s disease, and Parkinson’s disease, whereas in donors with Vascular dementia and Dementia with Lewy bodies, we observed elevated ANK1 DNA methylation only in individuals with coexisting Alzheimer’s disease pathology. We did not observe any disease-associated differential ANK1 DNA methylation in the striatum in Huntington’s disease or the substantia nigra in Parkinson’s disease. Our data suggest that ANK1 is characterized by region and disease-specific differential DNA methylation in multiple neurodegenerative diseases. (C) 2018 Elsevier Inc. All rights reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000455193900023" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neurobiolaging.2018.09.012" rel="external nofollow noopener" target="_blank">Association of NIPA1 repeat expansions with amyotrophic lateral sclerosis in a large international cohort</a></div> <div class="author"> Gij S. H. P. Tazelaar, Annelot M. Dekker, Joke J. F. A. Vugt, and <span class="more-authors" title="click to view 78 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '78 more authors' ? 'Rick A. Spek, Henk-Jan Westeneng, Lindy J. B. G. Kool, Kevin P. Kenna, Wouter Rheenen, Sara L. Pulit, Russell L. McLaughlin, William Sproviero, Alfredo Iacoangeli, Annemarie Huebers, David Brenner, Karen E. Morrison, Pamela J. Shaw, Christopher E. Shaw, Monica Povedano Panades, Jesus S. Mora Pardina, Jonathan D. Glass, Orla Hardiman, Ammar Al-Chalabi, Philip Damme, Wim Robberecht, John E. Landers, Albert C. Ludolph, Jochen H. Weishaupt, Leonard H. Berg, Jan H. Veldink, Michael A. Es, Fulya Akcimen, Ahmad Al Khleifat, Peter Andersen, A. Nazli Basak, Denis C. Bauer, Ian Blair, William J. Brands, Ross P. Byrne, Andrea Calvo, Yolanda Campos Gonzalez, Adriano Chio, Jonothan Cooper-Knock, Philippe Corcia, Philippe Couratier, Mamede Carvalho, Vivian E. Drory, Chen Eitan, Alberto Garcia Redondo, Cinzia Gellera, Marc Gotkine, Eran Hornstein, Brendan Kenna, Matthew C. Kiernan, Cemile Kocoglu, Maarten Kooyman, Victoria Lopez Alonso, Bas Middelkoop, Jonathan Mill, Miguel Mitne-Neto, Matthieu Moisse, Susana C. Pinto, Antonia Ratti, Raymond D. Schellevis, Aleksey Shatunov, Vincenzo Silani, Christine Staiger, Gijs H. P. Tazelaar, Nicola Ticozzi, Ceren Tunca, Nathalie A. Twine, Perry T. C. Doormaal, Kristel R. Eijk, Peter M. Visscher, Patrick Vourch, Markus Weber, Kelly L. Williams, Naomi Wray, Jian Yang, Mayana Zatz, Katharine Zhang, Project MinE ALS Sequencing Consor' : '78 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">78 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF AGING</em>, Feb 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2018.09.012" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2018.09.012" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>NIPA1 (nonimprinted in Prader-Willi/Angelman syndrome 1) mutations are known to cause hereditary spastic paraplegia type 6, a neurodegenerative disease that phenotypically overlaps to some extent with amyotrophic lateral sclerosis (ALS). Previously, a genomewide screen for copy number variants found an association with rare deletions in NIPA1 and ALS, and subsequent genetic analyses revealed that long (or expanded) polyalanine repeats in NIPA1 convey increased ALS susceptibility. We set out to perform a large-scale replication study to further investigate the role of NIPA1 polyalanine expansions with ALS, in which we characterized NIPA1 repeat size in an independent international cohort of 3955 patients with ALS and 2276 unaffected controls and combined our results with previous reports. Meta-analysis on a total of 6245 patients with ALS and 5051 controls showed an overall increased risk of ALS in those with expanded (&gt;8) GCG repeat length (odds ratio = 1.50, p = 3.8x10(-5)). Together with previous reports, these findings provide evidence for an association of an expanded polyalanine repeat in NIPA1 and ALS. (C) 2018 Elsevier Inc. All rights reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000459816900001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13041-019-0429-4" rel="external nofollow noopener" target="_blank">Genetic risk variants for brain disorders are enriched in cortical H3K27ac domains</a></div> <div class="author"> Eilis Hannon, Sarah J. Marzi, Leonard S. Schalkwyk, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jonathan Mill' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>MOLECULAR BRAIN</em>, Jan 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13041-019-0429-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13041-019-0429-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Most variants associated with complex phenotypes in genome-wide association studies (GWAS) do not directly index coding changes affecting protein structure. Instead they are hypothesized to influence gene regulation, with common variants associated with disease being enriched in regulatory domains including enhancers and regions of open chromatin. There is interest, therefore, in using epigenomic annotation data to identify the specific regulatory mechanisms involved and prioritize risk variants. We quantified lysine H3K27 acetylation (H3K27ac) - a robust mark of active enhancers and promoters that is strongly correlated with gene expression and transcription factor binding - across the genome in entorhinal cortex samples using chromatin immunoprecipitation followed by highly parallel sequencing (ChIP-seq). H3K27ac peaks were called using high quality reads combined across all samples and formed the basis of partitioned heritability analysis using LD score regression along with publicly-available GWAS results for seven psychiatric and neurodegenerative traits. Heritability for all seven brain traits was significantly enriched in these H3K27ac peaks (enrichment ranging from 1.09-2.13) compared to regions of the genome containing other active regulatory and functional elements across multiple cell types and tissues. The strongest enrichments were for amyotrophic lateral sclerosis (ALS) (enrichment = 2.19; 95% CI = 2.12-2.27), autism (enrichment = 2.11; 95% CI = 2.05-2.16) and major depressive disorder (enrichment = 2.04; 95% CI = 1.92-2.16). Much lower enrichments were observed for 14 non-brain disorders, although we identified enrichment in cortical H3K27ac domains for body mass index (enrichment = 1.16; 95% CI = 1.13-1.19), ever smoked (enrichment = 2.07; 95% CI = 2.04-2.10), HDL (enrichment = 1.53; 95% CI = 1.45-1.62) and trigylcerides (enrichment = 1.33; 95% CI = 1.24-1.42). These results indicate that risk alleles for brain disorders are preferentially located in regions of regulatory/enhancer function in the cortex, further supporting the hypothesis that genetic variants for these phenotypes influence gene regulation in the brain.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400485" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.404" rel="external nofollow noopener" target="_blank">DNA METHYLATION AND INFLAMMATION MARKER PROFILES ASSOCIATED WITH A HISTORY OF DEPRESSION</a></div> <div class="author"> Bethany Crawford, Zoe Craig, Georgina Mansell, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Isobel White, Adam Smith, Steve Spaull, Jennifer Imm, Eilis Hannon, Andrew Wood, Hanieh Yaghootkar, Yingjie Ji, Niamh Mullins, Cathryn Lewis, Jonathan Mill, Therese Murphy, Major Depressive Disorder Working' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.404" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.404" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400365" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.283" rel="external nofollow noopener" target="_blank">CAN DNA METHYLATION PROFILING SHED LIGHT ON THE BIOLOGY OF CANNABIS ASSOCIATED PSYCHOSIS? PRELIMINARY DATA ON THE EWAS CANNABIS USE SIGNATURE IN THE EUGEI CASE-CONTROL STUDY</a></div> <div class="author"> Marta Di Forti, Emma Dempster, Diego Quattrone, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Eilis Hannon, Joe Burrage, Georgina Mansell, Bart Rutten, Jim Os, Michael O’Donovan, Alexander Richards, Craig Morgan, Robin Murray, Chloe Chung Yi Wong, Jonathan Mill' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.283" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.283" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400150" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.064" rel="external nofollow noopener" target="_blank">METHYLOMIC BIOMARKERS OF SCHIZOPHRENIA AND ANTIPSYCHOTIC MEDICATION EXPOSURE</a></div> <div class="author"> Eilis Hannon, Emma Dempster, Georgina Mansell, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Leo Schalkwyk, Robin Murray, Andrew McQuillin, Kaarina Kowalec, David St Clair, Derek Morris, Patrick Sullivan, Michael O’Donovan, James MacCabe, David Collier, Jonathan Mill, CRESTAR Consortium' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.064" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.064" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400225" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.141" rel="external nofollow noopener" target="_blank">GENETIC RISK VARIANTS FOR BRAIN DISORDERS ARE SIGNIFICANTLY ENRICHED IN ACTIVE ENHANCERS IN THE CORTEX</a></div> <div class="author"> Eilis Hannon, Sarah Marzi, Leo Schalkwyk, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jonathan Mill' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.141" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.141" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400491" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.362" rel="external nofollow noopener" target="_blank">PLEIOTROPIC EFFECTS OF GENETIC VARIATION ASSOCIATED WITH PSYCHIATRIC DISORDERS ON DNA METHYLATION</a></div> <div class="author"> Eilis Hannon, Nick Bray, Micheal Weedon, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Tyler Gorrie-Stone, Melissa Smart, Meena Kumari, Leo Schalkwyk, Michael O’Donovan, Jonathan Mill' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.362" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.362" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400138" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.006" rel="external nofollow noopener" target="_blank">AN INTEGRATED GENETIC-EPIGENETIC ANALYSIS OF SCHIZOPHRENIA: EVIDENCE FOR CO-LOCALIZATION OF GENETIC ASSOCIATIONS AND DIFFERENTIAL DNA METHYLATION FROM A LARGE META-ANALYSIS OF WHOLE BLOOD DNA</a></div> <div class="author"> Eilis Hannon, Emma Dempster, Joe Burrage, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Andrew McQuillin, David St Clair, Derek Morris, Marta Di Forti, Fiona Gaughran, James MacCabe, Gerome Breen, David Collier, Robin Murray, Leo Schalkwyk, Jonathan Mill' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.006" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.006" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400375" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.293" rel="external nofollow noopener" target="_blank">ALTERNATIVE SPLICING AND ISOFOROM DIVERSITY IN THE DEVELOPING BRAIN: RELEVANCE FOR NEUROPSYCHIATRIC DISEASE</a></div> <div class="author"> Aaron Jeffries, Nick Bray, Paul O’Neill, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jeremie Poschmann, Eilis Hannon, Jonathan Mill' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.293" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.293" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400369" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.287" rel="external nofollow noopener" target="_blank">DNA METHYLOME MARKS OF EXPOSURE TO PSYCHOSOCIAL STRESS DURING ADOLESCENCE: ANALYSIS OF A NOVEL LONGITUDINAL MZ DISCORDANT TWIN STUDY</a></div> <div class="author"> Radhika Kandaswamy, Eilis Hannon, Georgina Mansell, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Ben Williams, Joe Burrage, Susanna Roberts, Andrea Danese, Jonathan Mill, Louise Arseneault, Helen Fisher, Chloe Chung Yi Wong' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.287" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.287" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400377" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.295" rel="external nofollow noopener" target="_blank">SITES OF ACTIVE GENE REGULATION IN THE DEVELOPING HUMAN BRAIN AND THEIR ROLE IN NEUROPSYCHIATRIC DISORDERS</a></div> <div class="author"> Manuela Kouakou, <em>Jonathan Mill</em>, Matthew Hill, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Nick Bray' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.295" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.295" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400497" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.368" rel="external nofollow noopener" target="_blank">SITES OF ACTIVE GENE REGULATION IN THE DEVELOPING HUMAN BRAIN AND THEIR ROLE IN NEUROPSYCHIATRIC DISORDERS</a></div> <div class="author"> Manuela Kouakou, <em>Jonathan Mill</em>, Matthew Hill, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Nick Bray' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.368" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.368" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400565" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.487" rel="external nofollow noopener" target="_blank">PREDICTION OF MORTALITY USING DNA METHYLATION AGE IN SCHIZOPHRENIA</a></div> <div class="author"> Kaarina Kowalec, Eilis Hannon, Georgina Mansell, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Joe Burrage, Jonathan Mill, Patrick Sullivan' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.487" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.487" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400121" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.06.137" rel="external nofollow noopener" target="_blank">REGULATORY GENOMIC VARIATION IN THE DEVELOPING BRAIN: RELEVANCE TO NEUROPSYCHIATRIC PHENOTYPES</a></div> <div class="author"> <em>Jonathan Mill</em>, Eilis Hannon, Helen Spiers, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Chloe Chung Yi Wong, Joana Viana, Michael O’Donovan, Leo Schalkwyk, Nick Bray' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.06.137" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.06.137" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400119" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.06.135" rel="external nofollow noopener" target="_blank">FUNCTIONAL GENOMIC REGULATION IN THE BRAIN: (EPI) GENETIC VARIATION, NEURODEVELOPMENT AND PSYCHIATRIC DISEASE</a></div> <div class="author"> Derek Morris, <em>Jonathan Mill</em>, and Andrew Jaffe</div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.06.135" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.06.135" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400148" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.062" rel="external nofollow noopener" target="_blank">EXPRESSION QUANTITATIVE TRAIT LOCI IN THE DEVELOPING HUMAN BRAIN AND THEIR ENRICHMENT IN NEUROPSYCHIATRIC DISORDERS</a></div> <div class="author"> Heath O’Brien, Eilis Hannon, Matthew Hill, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Carolina Toste, Matthew Robertson, Joanne Morgan, Gemma McLaughlin, Cathryn Lewis, Leonard Schalkwyk, Antonio Pardinas, Michael Owen, Michael O’Donovan, Jonathan Mill, Nick Bray' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.062" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.062" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400506" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.425" rel="external nofollow noopener" target="_blank">METHYLOMIC ANALYSIS OF SCHIZOPHRENIA IN 22Q11.2 DELETION SYNDROME</a></div> <div class="author"> Ehsan Pishva, Therese Amelsvoort, Eilis Hannon, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Laurens J. M. Evers, Jonathan Mill, Emma Dempster' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.425" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.425" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400372" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.290" rel="external nofollow noopener" target="_blank">SUICIDE-ASSOCIATED DNA METHYLATION CHANGES IN POST-MORTEM BRAIN SAMPLES: A META-ANALYSIS</a></div> <div class="author"> Stefania Policicchio, Joana Viana, Artemis Iatrou, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Eilis Hannon, Joe Burrage, Gustavo Turecki, Zachary Kaminsky, Jonathan Mill, Emma Dempster, Therese Murphy' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.290" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.290" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400255" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.126" rel="external nofollow noopener" target="_blank">DNA METHYLATION AND ADOLESCENT TRAJECTORIES OF PSYCHOTIC SYMPTOMS</a></div> <div class="author"> Susanna Roberts, Matthew Suderman, Stanley Zammit, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Sarah H. Watkins, Eilis Hannon, Caroline Relton, Jonathan Mill, Helen Fisher' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.126" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.126" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400373" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.291" rel="external nofollow noopener" target="_blank">GENETIC AND EPIGENETIC SIGNATURES OF AGGRESSION IN THREE DISTINCT RODENT MODELS</a></div> <div class="author"> Joana Viana, Georgina Mansell, Amanda Jager, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Floriana Mogavero, Sophie Walker, Damien Huzard, Francesca Zoratto, Simone Macri, Eilis Hannon, Giovanni Laviola, Carmen Sandi, Jeffrey Glennon, Jonathan Mill' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.291" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.291" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400545" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.467" rel="external nofollow noopener" target="_blank">A ZEBRAFISH MODEL OF CLOZAPINE EXPOSURE: DRUG-INDUCED TRANSCRIPTOMIC CHANGES IN THE BRAIN</a></div> <div class="author"> Joana Viana, Nick Wildman, Eilis Hannon, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Audrey Farbos, Paul O’Neill, Karen Moore, Konrad Paszkiewicz, Ronny Aerle, Gregory Paull, Eduarda Santos, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.467" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.467" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400137" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.005" rel="external nofollow noopener" target="_blank">A ZEBRAFISH MODEL OF CLOZAPINE EXPOSURE: DRUG-INDUCED TRANSCRIPTOMIC CHANGES IN THE BRAIN</a></div> <div class="author"> Joana Viana, Eilis Hannon, Ronny Aerle, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Emma Dempster, Gregory Paull, Eduarda Santos, Jonathan Mill' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.005" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.005" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000477708400178" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2018.08.093" rel="external nofollow noopener" target="_blank">FUNCTIONAL ANALYSIS OF THE SCHIZOPHRENIA ASSOCIATED GENE AS3MT IN SH-SY5Y NEUROBLASTOMA CELLS</a></div> <div class="author"> Sam Washer, Asami Oguro-Ando, Joe Burrage, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jonathan Mill, Aaron Jeffries, Emma Dempster' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 26th World Congress of Psychiatric Genetics (WCPG), Glasgow, SCOTLAND, OCT 11-15, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2018.08.093" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2018.08.093" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400494" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.365" rel="external nofollow noopener" target="_blank">FUNCTIONAL ANALYSIS OF THE SCHIZOPHRENIA ASSOCIATED GENE AS3MT IN SH-SY5Y NEUROBLASTOMA CELLS</a></div> <div class="author"> Sam Washer, Joe Burrage, <em>Jonathan Mill</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Aaron Jeffries, Emma Dempster' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.365" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.365" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000462156400488" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2017.08.359" rel="external nofollow noopener" target="_blank">CONVERGENT METHYLOMIC SIGNATURES BETWEEN AUTISM ASSOCIATED WITH DUPLICATIONS OF CHROMOSOME 15Q AND IDIOPATHIC AUTISM</a></div> <div class="author"> Chloe Chung Yi Wong, Rebecca G. Smith, Eilis Hannon, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Elham Assary, Neelroop Parikshak, Shyam Prabhakar, Daniel Geschwind, Jonathan Mill' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Jan 2019 </div> <div class="periodical"> 25th World Congress of Psychiatric Genetics (WCPG), Orlando, FL, OCT 13-17, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2017.08.359" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2017.08.359" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> </ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000451827200003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.jalz.2018.01.017" rel="external nofollow noopener" target="_blank">Elevated DNA methylation across a 48-kb region spanning the HOXA gene cluster is associated with Alzheimer’s disease neuropathology</a></div> <div class="author"> Rebecca G. Smith, Eilis Hannon, Philip L. De Jager, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Lori Chibnik, Simon J. Lott, Daniel Condliffe, Adam R. Smith, Vahram Haroutunian, Claire Troakes, Safa Al-Sarraj, David A. Bennett, John Powell, Simon Lovestone, Leonard Schalkwyk, Jonathan Mill, Katie Lunnon' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>ALZHEIMERS &amp; DEMENTIA</em>, Dec 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.jalz.2018.01.017" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jalz.2018.01.017" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Introduction: Alzheimer’s disease is a neurodegenerative disorder that is hypothesized to involve epigenetic dysregulation of gene expression in the brain. Methods: We performed an epigenome-wide association study to identify differential DNA methylation associated with neuropathology in prefrontal cortex and superior temporal gyrus samples from 147 individuals, replicating our findings in two independent data sets (N = 117 and 740). Results: We identify elevated DNA methylation associated with neuropathology across a 48-kb region spanning 208 CpG sites within the HOXA gene cluster. A meta-analysis of the top-ranked probe within the HOXA3 gene (cg22962123) highlighted significant hypermethylation across all three cohorts (P = 3.11 X 10(-18)). Discussion: We present robust evidence for elevated DNA methylation associated with Alzheimer’s disease neuropathology spanning the HOXA gene cluster on chromosome 7. These data add to the growing evidence highlighting a role for epigenetic variation in Alzheimer’s disease, implicating the HOX gene family as a target for future investigation. (C) 2018 The Authors. Published by Elsevier Inc. on behalf of the Alzheimer’s Association.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000451083400003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13059-018-1567-1" rel="external nofollow noopener" target="_blank">Expression quantitative trait loci in the developing human brain and their enrichment in neuropsychiatric disorders</a></div> <div class="author"> Heath E. O’Brien, Eilis Hannon, Matthew J. Hill, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Carolina C. Toste, Matthew J. Robertson, Joanne E. Morgan, Gemma McLaughlin, Cathryn M. Lewis, Leonard C. Schalkwyk, Lynsey S. Hall, Antonio F. Pardinas, Michael J. Owen, Michael C. O’Donovan, Jonathan Mill, Nicholas J. Bray' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>GENOME BIOLOGY</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-018-1567-1" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-018-1567-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Genetic influences on gene expression in the human fetal brain plausibly impact upon a variety of postnatal brain-related traits, including susceptibility to neuropsychiatric disorders. However, to date, there have been no studies that have mapped genome-wide expression quantitative trait loci (eQTL) specifically in the human prenatal brain. Results: We performed deep RNA sequencing and genome-wide genotyping on a unique collection of 120 human brains from the second trimester of gestation to provide the first eQTL dataset derived exclusively from the human fetal brain. We identify high confidence cis-acting eQTL at the individual transcript as well as whole gene level, including many mapping to a common inversion polymorphism on chromosome 17q21. Fetal brain eQTL are enriched among risk variants for postnatal conditions including attention deficit hyperactivity disorder, schizophrenia, and bipolar disorder. We further identify changes in gene expression within the prenatal brain that potentially mediate risk for neuropsychiatric traits, including increased expression of C4A in association with genetic risk for schizophrenia, increased expression of LRRC57 in association with genetic risk for bipolar disorder, and altered expression of multiple genes within the chromosome 17q21 inversion in association with variants influencing the personality trait of neuroticism. Conclusions: We have mapped eQTL operating in the human fetal brain, providing evidence that these confer risk to certain neuropsychiatric disorders, and identifying gene expression changes that potentially mediate susceptibility to these conditions.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000448942100003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.ajhg.2018.09.007" rel="external nofollow noopener" target="_blank">Leveraging DNA-Methylation Quantitative-Trait Loci to Characterize the Relationship between Methylomic Variation, Gene Expression, and Complex Traits</a></div> <div class="author"> Eilis Hannon, Tyler J. Gorrie-Stone, Melissa C. Smart, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Joe Burrage, Amanda Hughes, Yanchun Bao, Meena Kumari, Leonard C. Schalkwyk, Jonathan Mill' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>AMERICAN JOURNAL OF HUMAN GENETICS</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.ajhg.2018.09.007" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ajhg.2018.09.007" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Characterizing the complex relationship between genetic, epigenetic, and transcriptomic variation has the potential to increase understanding about the mechanisms underpinning health and disease phenotypes. We undertook a comprehensive analysis of common genetic variation on DNA methylation (DNAm) by using the Illumina EPIC array to profile samples from the UK Household Longitudinal study. We identified 12,689,548 significant DNA methylation quantitative trait loci (mQTL) associations (p &lt; 6.52 x 10(-14)) occurring between 2,907,234 genetic variants and 93,268 DNAm sites, including a large number not identified by previous DNAm-profiling methods. We demonstrate the utility of these data for interpreting the functional consequences of common genetic variation associated with &gt; 60 human traits by using summary-data-based Mendelian randomization (SMR) to identify 1,662 pleiotropic associations between 36 complex traits and 1,246 DNAm sites. We also use SMR to characterize the relationship between DNAm and gene expression and thereby identify 6,798 pleiotropic associations between 5,420 DNAm sites and the transcription of 1,702 genes. Our mQTL database and SMR results are available via a searchable online database as a resource to the research community.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000452399200007" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41380-017-0008-y" rel="external nofollow noopener" target="_blank">Meta-analysis of epigenome-wide association studies of cognitive abilities</a></div> <div class="author"> Riccardo E. Marioni, Allan F. McRae, Jan Bressler, and <span class="more-authors" title="click to view 44 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '44 more authors' ? 'Elena Colicino, Eilis Hannon, Shuo Li, Diddier Prada, Jennifer A. Smith, Letizia Trevisi, Pei-Chien Tsai, Dina Vojinovic, Jeannette Simino, Daniel Levy, Chunyu Liu, Michael Mendelson, Claudia L. Satizabal, Qiong Yang, Min A. Jhun, Sharon L. R. Kardia, Wei Zhao, Stefania Bandinelli, Luigi Ferrucci, Dena G. Hernandez, Andrew B. Singleton, Sarah E. Harrisl, John M. Starr, Douglas P. Kie, Robert R. McLean, Allan C. Just, Joel Schwartz, Avron Spiro, Pantel Vokonas, Najaf Amin, M. Arfan Ikram, Andre G. Uitterlinden, Joyce B. J. Meurs, Tim D. Spector, Claire Steves, Andrea A. Baccarelli, Jordana T. Bell, Cornelia M. Duijn, Myriam Fornage, Yi-Hsiang Hsu, Jonathan Mill, Thomas H. Mosley, Sudha Seshadri, Ian J. Deary' : '44 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">44 more authors</span> </div> <div class="periodical"> <em>MOLECULAR PSYCHIATRY</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41380-017-0008-y" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41380-017-0008-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Cognitive functions are important correlates of health outcomes across the life-course. Individual differences in cognitive functions are partly heritable. Epigenetic modifications, such as DNA methylation, are susceptible to both genetic and environmental factors and may provide insights into individual differences in cognitive functions. Epigenome-wide meta-analyses for blood-based DNA methylation levels at similar to 420,000 CpG sites were performed for seven measures of cognitive functioning using data from 11 cohorts. CpGs that passed a Bonferroni correction, adjusting for the number of CpGs and cognitive tests, were assessed for: longitudinal change; being under genetic control (methylation QTLs); and associations with brain health (structural MRI), brain methylation and Alzheimer’s disease pathology. Across the seven measures of cognitive functioning (meta-analysis n range: 2557-6809), there were epigenome-wide significant (P &lt; 1.7 x 10(-8)) associations for global cognitive function (cg21450381, P = 1.6 x 10(-8)), and phonemic verbal fluency (cg12507869, P = 2.5 x 10(-9)). The CpGs are located in an intergenic region on chromosome 12 and the INPP5A gene on chromosome 10, respectively. Both probes have moderate correlations (similar to 0.4) with brain methylation in Brodmann area 20 (ventral temporal cortex). Neither probe showed evidence of longitudinal change in late-life or associations with white matter brain MRI measures in one cohort with these data. A methylation QTL analysis suggested that rs113565688 was a cis methylation QTL for cg12507869 (P = 5 x 10(-5) and 4 x 10(-13) in two lookup cohorts). We demonstrate a link between blood-based DNA methylation and measures of phonemic verbal fluency and global cognitive ability. Further research is warranted to understand the mechanisms linking genomic regulatory changes with cognitive function to health and disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000448319500016" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41593-018-0253-7" rel="external nofollow noopener" target="_blank">A histone acetylome-wide association study of Alzheimer’s disease identifies disease-associated H3K27ac differences in the entorhinal cortex</a></div> <div class="author"> Sarah J. Marzi, Szi Kay Leung, Teodora Ribarska, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Eilis Hannon, Adam R. Smith, Ehsan Pishva, Jeremie Poschmann, Karen Moore, Claire Troakes, Safa Al-Sarraj, Stephan Beck, Stuart Newman, Katie Lunnon, Leonard C. Schalkwyk, Jonathan Mill' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>NATURE NEUROSCIENCE</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41593-018-0253-7" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41593-018-0253-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>We quantified genome-wide patterns of lysine H3K27 acetylation (H3K27ac) in entorhinal cortex samples from Alzheimer’s disease (AD) cases and matched controls using chromatin immunoprecipitation and highly parallel sequencing. We observed widespread acetylomic variation associated with AD neuropathology, identifying 4,162 differential peaks (false discovery rate &lt; 0.05) between AD cases and controls. Differentially acetylated peaks were enriched in disease-related biological pathways and included regions annotated to genes involved in the progression of amyloid-beta and tau pathology (for example, APP, PSEN1, PSEN2, and MAPT), as well as regions containing variants associated with sporadic late-onset AD. Partitioned heritability analysis highlighted a highly significant enrichment of AD risk variants in entorhinal cortex H3K27ac peak regions. AD-associated variable H3K27ac was associated with transcriptional variation at proximal genes including CR1, GPR22, KMO, PIM3, PSEN1, and RGCC. In addition to identifying molecular pathways associated with AD neuropathology, we present a framework for genome-wide studies of histone modifications in complex disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000444554000001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41467-018-05714-3" rel="external nofollow noopener" target="_blank">Autosomal genetic variation is associated with DNA methylation in regions variably escaping X-chromosome inactivation</a></div> <div class="author"> Rene Luijk, Haoyu Wu, Cavin K. Ward-Caviness, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Eilis Hannon, Elena Carnero-Montoro, Josine L. Min, Pooja Mandaviya, Martina Mueller-Nurasyid, Hailiang Mei, Silvere M. Maarel, Caroline Relton, Jonathan Mill, Melanie Waldenberger, Jordana T. Bell, Rick Jansen, Alexandra Zhernakova, Lude Franke, Peter A. C. Hoen, Dorret I. Boomsma, Cornelia M. Duijn, Marleen M. J. Greevenbroek, Jan H. Veldink, Cisca Wijmenga, Joyce Meurs, Lucia Daxinger, P. Eline Slagboom, Erik W. Zwet, Bastiaan T. Heijmans, BIOS Consortium' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Sep 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41467-018-05714-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-018-05714-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>X-chromosome inactivation (XCI), i.e., the inactivation of one of the female X chromosomes, restores equal expression of X-chromosomal genes between females and males. However, similar to 10% of genes show variable degrees of escape from XCI between females, although little is known about the causes of variable XCI. Using a discovery data-set of 1867 females and 1398 males and a replication sample of 3351 females, we show that genetic variation at three autosomal loci is associated with female-specific changes in X-chromosome methylation. Through cis-eQTL expression analysis, we map these loci to the genes SMCHD1/METTL4, TRIM6/HBG2, and ZSCAN9. Low-expression alleles of the loci are predominantly associated with mild hypomethylation of CpG islands near genes known to variably escape XCI, implicating the autosomal genes in variable XCI. Together, these results suggest a genetic basis for variable escape from XCI and highlight the potential of a population genomics approach to identify genes involved in XCI.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000441102800006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/hmg/ddy199" rel="external nofollow noopener" target="_blank">DNA methylation and inflammation marker profiles associated with a history of depression</a></div> <div class="author"> Bethany Crawford, Zoe Craig, Georgina Mansell, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Isobel White, Adam Smith, Steve Spaull, Jennifer Imm, Eilis Hannon, Andrew Wood, Hanieh Yaghootkar, Yingjie Ji, Niamh Mullins, Cathryn M. Lewis, Jonathan Mill, Therese M. Murphy, Major Depressive Disorder Working' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>HUMAN MOLECULAR GENETICS</em>, Aug 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/hmg/ddy199" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/hmg/ddy199" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Depression is a common and disabling disorder, representing a major social and economic health issue. Moreover, depression is associated with the progression of diseases with an inflammatory etiology including many inflammatory related disorders. At the molecular level, the mechanisms by which depression might promote the onset of these diseases and associated immune-dysfunction are not well understood. In this study we assessed genome-wide patterns of DNA methylation in whole blood-derived DNA obtained from individuals with a self-reported history of depression (n = 100) and individuals without a history of depression (n = 100) using the Illumine 450K microarray. Our analysis identified six significant (Sidak corrected P &lt; 0.05) depression-associated differentially methylated regions (DMRs); the top-ranked DMR was located in exon 1 of the LTB4R2 gene (Sidel corrected P = 1.27 x 10(-14)). Polygenic risk scores (PRS) for depression were generated and known biological markers of inflammation, telomere length (TL) and IL-6, were measured in DNA and serum samples, respectively. Next, we employed a systems-level approach to identify networks of co-methylated loci associated with a history of depression, in addition to depression PRS, TL and IL-6 levels. Our analysis identified one depression-associated co-methylation module (P = 0.04). Interestingly, the depression-associated module was highly enriched for pathways related to immune function and was also associated with TL and IL-6 cytokine levels. In summary, our genome-wide DNA methylation analysis of individuals with and without a self-reported history of depression identified several candidate DMRs of potential relevance to the pathogenesis of depression and its associated immune-dysfunction phenotype.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000443389100015" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1371/journal.pgen.1007544" rel="external nofollow noopener" target="_blank">Characterizing genetic and environmental influences on variable DNA methylation using monozygotic and dizygotic twins</a></div> <div class="author"> Eilis Hannon, Olivia Knox, Karen Sugden, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Joe Burrage, Chloe C. Y. Wong, Daniel W. Belsky, David L. Corcoran, Louise Arseneault, Terrie E. Moffitt, Avshalom Caspi, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>PLOS GENETICS</em>, Aug 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1371/journal.pgen.1007544" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pgen.1007544" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Variation in DNA methylation is being increasingly associated with health and disease outcomes. Although DNA methylation is hypothesized to be a mechanism by which both genetic and non-genetic factors can influence the regulation of gene expression, little is known about the extent to which DNA methylation at specific sites is influenced by heritable as well as environmental factors. We quantified DNA methylation in whole blood at age 18 in a birth cohort of 1,464 individuals comprising 426 monozygotic (MZ) and 306 same-sex dizygotic (DZ) twin pairs. Site-specific levels of DNA methylation were more strongly correlated across the genome between MZ than DZ twins. Structural equation models revealed that although the average contribution of additive genetic influences on DNA methylation across the genome was relatively low, it was notably elevated at the highly variable sites characterized by intermediate levels of DNAm that are most relevant for epigenetic epidemiology. Sites at which variable DNA methylation was most influenced by genetic factors were significantly enriched for DNA methylation quantitative trait loci (mQTL) effects, and overlapped with sites where inter-individual variation correlates across tissues. Finally, we show that DNA methylation at sites robustly associated with environmental exposures such as tobacco smoking and obesity is also influenced by additive genetic effects, highlighting the need to control for genetic background in analyses of exposure-associated DNA methylation differences. Estimates of the contribution of genetic and environmental influences to DNA methylation at all sites profiled in this study are available as a resource for the research community (http://www.epigenomicslab.com/online-data-resources).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000439344300002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13148-018-0530-z" rel="external nofollow noopener" target="_blank">Genome-wide methylomic analysis in individuals with HNF1B intragenic mutation and 17q12 microdeletion</a></div> <div class="author"> Rhian L. Clissold, Beth Ashfield, Joe Burrage, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Eilis Hannon, Coralie Bingham, Jonathan Mill, Andrew Hattersley, Emma L. Dempster' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>CLINICAL EPIGENETICS</em>, Jul 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13148-018-0530-z" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13148-018-0530-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Heterozygous mutation of the transcription factor HNF1B is the most common cause of monogenetic developmental renal disease. Disease-associated mutations fall into two categories: HNF1B intragenic mutations and a 1.3 Mb deletion at chromosome 17q12. An increase in neurodevelopmental disorders has been observed in individuals harbouring the 17q12 deletion but not in patients with HNF1B coding mutations. Previous investigations have concentrated on identifying a genetic cause for the increase in behavioural problems seen in 17q12 deletion carriers. We have taken the alternative approach of investigating the DNA methylation profile of these two HNF1B genotype groups along with controls matched for age, gender and diabetes status using the Illumina 450K DNA methylation array (total sample n = 60). We identified a number of differentially methylated probes (DMPs) that were associated with HNF1B-associated disease and passed our stringent experiment-wide significance threshold. These associations were largely driven by the deletion patients and the majority of the significant probes mapped to the 17q12 deletion locus. The observed changes in DNA methylation at this locus were not randomly dispersed and occurred in clusters, suggesting a regulatory mechanism reacting to haploinsufficiency across the entire deleted region. Along with these deletion-specific changes in DNA methylation, we also identified a shared DNA methylation signature in both mutation and deletion patient groups indicating that haploinsufficiency of HNF1B impacts on the methylome of a number of genes, giving further insight to the role of HNF1B.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000437025600002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41398-018-0167-x" rel="external nofollow noopener" target="_blank">Prenatal immune activation alters the adult neural epigenome but can be partly stabilised by a n-3 polyunsaturated fatty acid diet</a></div> <div class="author"> Paul Basil, Qi Li, Hongsheng Gui, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Tomy C. K. Hui, Vicki H. M. Ling, Chloe C. Y. Wong, Jonathan Mill, Grainne M. McAlonan, Pak-Chung Sham' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>TRANSLATIONAL PSYCHIATRY</em>, Jul 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41398-018-0167-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41398-018-0167-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>An unstable epigenome is implicated in the pathophysiology of neurodevelopmental disorders such as schizophrenia and autism. This is important because the epigenome is potentially modifiable. We have previously reported that adult offspring exposed to maternal immune activation (MIA) prenatally have significant global DNA hypomethylation in the hypothalamus. However, what genes had altered methylation state, their functional effects on gene expression and whether these changes can be moderated, have not been addressed. In this study, we used next-generation sequencing (NGS) for methylome profiling in a MIA rodent model of neurodevelopmental disorders. We assessed whether differentially methylated regions (DMRs) affected the chromatin state by mapping known DNase I hypersensitivity sites (DHSs), and selected overlapping genes to confirm a functional effect of MIA on gene expression using qPCR. Finally, we tested whether methylation differences elicited by MIA could be limited by post-natal dietary (omega) n-3 polyunsaturated fatty acid (PUFA) supplementation. These experiments were conducted using hypothalamic brain tissue from 12-week-old offspring of mice injected with viral analogue PolyI:C on gestation day 9 of pregnancy or saline on gestation day 9. Half of the animals from each group were fed a diet enriched with n-3 PUFA from weaning (MIA group, n = 12 units, n = 39 mice; Control group, n = 12 units, n = 38 mice). The results confirmed our previous finding that adult offspring exposed to MIA prenatally had significant global DNA hypomethylation. Furthermore, genes linked to synaptic plasticity were over-represented among differentially methylated genes following MIA. More than 80% of MIA-induced hypomethylated sites, including those affecting chromatin state and MECP2 binding, were stabilised by the n-3 PUFA intervention. MIA resulted in increased expression of two of the ‘top five’ genes identified from an integrated analysis of DMRs, DHSs and MECP2 binding sites, namely Abat (t = 2.46, p &lt; 0.02) and Gnas9 (t = 2.96, p &lt; 0.01), although these changes were not stabilised by dietary intervention. Thus, prenatal MIA exposure impacts upon the epigenomic regulation of gene pathways linked to neurodevelopmental conditions; and many of the changes can be attenuated by a low-cost dietary intervention.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000437316000009" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1176/appi.ajp.2017.17060693" rel="external nofollow noopener" target="_blank">Analysis of DNA Methylation in Young People: Limited Evidence for an Association Between Victimization Stress and Epigenetic Variation in Blood</a></div> <div class="author"> Sarah J. Marzi, Karen Sugden, Louise Arseneault, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Daniel W. Belsky, Joe Burrage, David L. Corcoran, Andrea Danese, Helen L. Fisher, Eilis Hannon, Terrie E. Moffitt, Candice L. Odgers, Carmine Pariante, Richie Poulton, Benjamin S. Williams, Chloe C. Y. Wong, Jonathan Mill, Avshalom Caspi' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>AMERICAN JOURNAL OF PSYCHIATRY</em>, Jun 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1176/appi.ajp.2017.17060693" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1176/appi.ajp.2017.17060693" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Objective: DNA methylation has been proposed as an epigenetic mechanism by which early-life experiences become “embedded” in the genome and alter transcriptional processes to compromise health. The authors sought to investigate whether early-life victimization stress is associated with genome-wide DNA methylation. Method: The authors tested the hypothesis that victimization is associated with DNA methylation in the Environmental Risk (E-Risk) Longitudinal Study, a nationally representative 1994-1995 birth cohort of 2,232 twins born in England and Wales and assessed at ages 5, 7, 10, 12, and 18 years. Multiple forms of victimization were ascertained in childhood and adolescence (including physical, sexual, and emotional abuse; neglect; exposure to intimate-partner violence; bullying; cyber-victimization; and crime). Results: Epigenome-wide analyses of polyvictimization across childhood and adolescence revealed few significant associations with DNA methylation in peripheral blood at age 18, but these analyses were confounded by tobacco smoking and/or did not survive co-twin control tests. Secondary analyses of specific forms of victimization revealed sparse associations with DNA methylation that did not replicate across different operationalizations of the same putative victimization experience. Hypothesis-driven analyses of six candidate genes in the stress response (NR3C1, FKBP5, BDNF, AVP, CRHR1, SLC6A4) did not reveal predicted associations with DNA methylation in probes annotated to these genes. Conclusions: Findings from this epidemiological analysis of the epigenetic effects of early-life stress do not support the hypothesis of robust changes in DNA methylation in victimized young people. We need to come to terms with the possibility that epigenetic epidemiology is not yet well matched to experimental, nonhuman models in uncovering the biological embedding of stress.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000432201400006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.2217/epi-2017-0132" rel="external nofollow noopener" target="_blank">Peripheral DNA methylation, cognitive decline and brain aging: pilot findings from the Whitehall II imaging study</a></div> <div class="author"> Leonidas Chouliaras, Ehsan Pishva, Rita Haapakoski, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Eniko Zsoldos, Abda Mahmood, Nicola Filippini, Joe Burrage, Jonathan Mill, Mika Kivimaki, Katie Lunnon, Klaus P. Ebmeier' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EPIGENOMICS</em>, May 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.2217/epi-2017-0132" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.2217/epi-2017-0132" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Aim: The present study investigated the link between peripheral DNA methylation (DNAm), cognitive impairment and brain aging. Methods: We tested the association between blood genome-wide DNAm profiles using the Illumina 450K arrays, cognitive dysfunction and brain MRI measures in selected participants of the Whitehall II imaging sub-study. Results: Eight differentially methylated regions were associated with cognitive impairment. Accelerated aging based on the Hannum epigenetic clock was associated with mean diffusivity and global fractional anisotropy. We also identified modules of co-methylated loci associated with white matter hyperintensities. These co-methylation modules were enriched among pathways relevant to beta-amyloid processing and glutamatergic signaling. Conclusion: Our data support the notion that blood DNAm changes may have utility as a biomarker for cognitive dysfunction and brain aging.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000431394900010" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/s41588-018-0090-3" rel="external nofollow noopener" target="_blank">Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression</a></div> <div class="author"> Naomi R. Wray, Stephan Ripke, Manuel Mattheisen, and <span class="more-authors" title="click to view 215 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '215 more authors' ? 'Maciej Trzaskowski, Enda M. Byrne, Abdel Abdellaoui, Mark J. Adams, Esben Agerbo, Tracy M. Air, Till M. F. Andlauer, Silviu-Alin Bacanu, Marie Baekvad-Hansen, Aartjan F. T. Beekman, Tim B. Bigdeli, Elisabeth B. Binder, Douglas R. H. Blackwood, Julien Bryois, Henriette N. Buttenschon, Jonas Bybjerg-Grauholm, Na Cai, Enrique Castelao, Jane Hvarregaard Christensen, Toni-Kim Clarke, Jonathan I. R. Coleman, Lucia Colodro-Conde, Baptiste Couvy-Duchesne, Nick Craddock, Gregory E. Crawford, Cheynna A. Crowley, Hassan S. Dashti, Gail Davies, Ian J. Deary, Franziska Degenhardt, Eske M. Derks, Nese Direk, Conor V. Dolan, Erin C. Dunn, Thalia C. Eley, Nicholas Eriksson, Valentina Escott-Price, Farnush Hassan Farhadi Kiadeh, Hilary K. Finucane, Andreas J. Forstner, Josef Frank, Helena A. Gaspar, Michael Gill, Paola Giusti-Rodriguez, Fernando S. Goes, Scott D. Gordon, Jakob Grove, Lynsey S. Hall, Eilis Hannon, Christine Soholm Hansen, Thomas F. Hansen, Stefan Herms, Ian B. Hickie, Per Hoffmann, Georg Homuth, Carsten Horn, Jouke-Jan Hottenga, David M. Hougaard, Ming Hu, Craig L. Hyde, Marcus Ising, Rick Jansen, Fulai Jin, Eric Jorgenson, James A. Knowles, Isaac S. Kohane, Julia Kraft, Warren W. Kretzschmar, Jesper Krogh, Zoltan Kutalik, Jacqueline M. Lane, Yihan Li, Yun Li, Penelope A. Lind, Xiaoxiao Liu, Leina Lu, Donald J. MacIntyre, Dean F. MacKinnon, Robert M. Maier, Wolfgang Maier, Jonathan Marchini, Hamdi Mbarek, Patrick McGrath, Peter McGuffin, Sarah E. Medland, Divya Mehta, Christel M. Middeldorp, Evelin Mihailov, Yuri Milaneschi, Lili Milani, Jonathan Mill, Francis M. Mondimore, Grant W. Montgomery, Sara Mostafavi, Niamh Mullins, Matthias Nauck, Bernard Ng, Michel G. Nivard, Dale R. Nyholt, Paul F. O’Reilly, Hogni Oskarsson, Michael J. Owen, Jodie N. Painter, Carsten Bocker Pedersen, Marianne Giortz Pedersen, Roseann E. Peterson, Erik Pettersson, Wouter J. Peyrot, Giorgio Pistis, Danielle Posthuma, Shaun M. Purcell, Jorge A. Quiroz, Per Qvist, John P. Rice, Brien P. Riley, Margarita Rivera, Saira Saeed Mirza, Richa Saxena, Robert Schoevers, Eva C. Schulte, Ling Shen, Jianxin Shi, Stanley I. Shyn, Engilbert Sigurdsson, Grant B. C. Sinnamon, Johannes H. Smit, Daniel J. Smith, Hreinn Stefansson, Stacy Steinberg, Craig A. Stockmeier, Fabian Streit, Jana Strohmaier, Katherine E. Tansey, Henning Teismann, Alexander Teumer, Wesley Thompson, Pippa A. Thomson, Thorgeir E. Thorgeirsson, Chao Tian, Matthew Traylor, Jens Treutlein, Vassily Trubetskoy, Andre G. Uitterlinden, Daniel Umbricht, Sandra Auwera, Albert M. Hemert, Alexander Viktorin, Peter M. Visscher, Yunpeng Wang, Bradley T. Webb, Shantel Marie Weinsheimer, Juegen Wellmann, Gonneke Willemsen, Stephanie H. Witt, Yang Wu, Hualin S. Xi, Jian Yang, Futao Zhang, Volker Arolt, Bernhard T. Baune, Klaus Berger, Dorret I. Boomsma, Sven Cichon, Udo Dannlowski, E. C. J. Geus, J. Raymond DePaulo, Enrico Domenici, Katharina Domschke, Tonu Esko, Hans J. Grabe, Steven P. Hamilton, Caroline Hayward, Andrew C. Heath, David A. Hinds, Kenneth S. Kendler, Stefan Kloiber, Glyn Lewis, Qingqin S. Li, Susanne Lucae, Pamela F. A. Madden, Patrik K. Magnusson, Nicholas G. Martin, Andrew M. McIntosh, Andres Metspalu, Ole Mors, Preben Bo Mortensen, Bertram Mueller-Myhsok, Merete Nordentoft, Markus M. Noethen, Michael C. O’Donovan, Sara A. Paciga, Nancy L. Pedersen, Brenda W. J. H. Penninx, Roy H. Perlis, David J. Porteous, James B. Potash, Martin Preisig, Marcella Rietschel, Catherine Schaefer, Thomas G. Schulze, Jordan W. Smoller, Kari Stefansson, Henning Tiemeier, Rudolf Uher, Henry Voelzke, Myrna M. Weissman, Thomas Werge, Ashley R. Winslow, Cathryn M. Lewis, Douglas F. Levinson, Gerome Breen, Anders D. Borglum, Patrick F. Sullivan, EQTLGEN, Major Depressive Disorder Working' : '215 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">215 more authors</span> </div> <div class="periodical"> <em>NATURE GENETICS</em>, May 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/s41588-018-0090-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41588-018-0090-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Major depressive disorder (MDD) is a common illness accompanied by considerable morbidity, mortality, costs, and heightened risk of suicide. We conducted a genome-wide association meta-analysis based in 135,458 cases and 344,901 controls and identified 44 independent and significant loci. The genetic findings were associated with clinical features of major depression and implicated brain regions exhibiting anatomical differences in cases. Targets of antidepressant medications and genes involved in gene splicing were enriched for smaller association signal. We found important relationships of genetic risk for major depression with educational attainment, body mass, and schizophrenia: lower educational attainment and higher body mass were putatively causal, whereas major depression and schizophrenia reflected a partly shared biological etiology. All humans carry lesser or greater numbers of genetic risk factors for major depression. These findings help refine the basis of major depression and imply that a continuous measure of risk underlies the clinical phenotype.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000429541800062" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/schbul/sby014.060" rel="external nofollow noopener" target="_blank">EPIGENETIC SIGNATURES OF CHILDHOOD AND ADOLESCENT VICTIMISATION USING A GENETICALLY SENSITIVE LONGITUDINAL COHORT STUDY</a></div> <div class="author"> Helen Fisher, Sarah Marzi, Louise Arseneault, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Chloe Wong, Radhika Kandaswamy, Terrie Moffitt, Susanna Roberts, Jonathan Mill, Avshalom Caspi' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>SCHIZOPHRENIA BULLETIN</em>, Apr 2018 </div> <div class="periodical"> 6th Biennial Conference of the Schizophrenia-International-Research-Society (SIRS), Florence, ITALY, APR 04-08, 2018 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/schbul/sby014.060" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/schbul/sby014.060" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000428906200001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13073-018-0527-4" rel="external nofollow noopener" target="_blank">Elevated polygenic burden for autism is associated with differential DNA methylation at birth</a></div> <div class="author"> Eilis Hannon, Diana Schendel, Christine Ladd-Acosta, and <span class="more-authors" title="click to view 20 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '20 more authors' ? 'Jakob Grove, Christine Soholm Hansen, Shan V. Andrews, David Michael Hougaard, Michaeline Bresnahan, Ole Mors, Mads Vilhelm Hollegaard, Marie Baekvad-Hansen, Mady Hornig, Preben Bo Mortensen, Anders D. Borglum, Thomas Werge, Marianne Giortz Pedersen, Merete Nordentoft, Joseph Buxbaum, M. Daniele Fallin, Jonas Bybjerg-Grauholm, Abraham Reichenberg, Jonathan Mill, ASD Grp' : '20 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">20 more authors</span> </div> <div class="periodical"> <em>GENOME MEDICINE</em>, Mar 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13073-018-0527-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13073-018-0527-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Autism spectrum disorder (ASD) is a severe neurodevelopmental disorder characterized by deficits in social communication and restricted, repetitive behaviors, interests, or activities. The etiology of ASD involves both inherited and environmental risk factors, with epigenetic processes hypothesized as one mechanism by which both genetic and non-genetic variation influence gene regulation and pathogenesis. The aim of this study was to identify DNA methylation biomarkers of ASD detectable at birth. Methods: We quantified neonatal methylomic variation in 1263 infants-of whom similar to 50% went on to subsequently develop ASD-using DNA isolated from archived blood spots taken shortly after birth. We used matched genotype data from the same individuals to examine the molecular consequences of ASD-associated genetic risk variants, identifying methylomic variation associated with elevated polygenic burden for ASD. In addition, we performed DNA methylation quantitative trait loci (mQTL) mapping to prioritize target genes from ASD GWAS findings. Results: We identified robust epigenetic signatures of gestational age and prenatal tobacco exposure, confirming the utility of DNA methylation data generated from neonatal blood spots. Although we did not identify specific loci showing robust differences in neonatal DNA methylation associated with later ASD, there was a significant association between increased polygenic burden for autism and methylomic variation at specific loci. Each unit of elevated ASD polygenic risk score was associated with a mean increase in DNA methylation of -0.14% at two CpG sites located proximal to a robust GWAS signal for ASD on chromosome 8. Conclusions: This study is the largest analysis of DNA methylation in ASD undertaken and the first to integrate genetic and epigenetic variation at birth. We demonstrate the utility of using a polygenic risk score to identify molecular variation associated with disease, and of using mQTL to refine the functional and regulatory variation associated with ASD risk variants.</p> </div> </div> </div> </li> </ol> <h2 class="year">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800470" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">GENOME-WIDE METHYLOMIC ANALYSIS OF MONOZYGOTIC TWINS DISCORDANT FOR SCHIZOPHRENIA</a></div> <div class="author"> Emma Dempster, Eilis Hannon, Leo Schalkwyk, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Marc Bohlken, Jaakko Kaprio, Timothea Toulopoulou, Tim Spector, Hilleke Hulshoffpol, Igor Nenadic, Tyrone Cannon, Christina Hultman, Robin Murray, Jonathan Mill' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413847300165" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.euroneuro.2016.09.538" rel="external nofollow noopener" target="_blank">LONGITUDINAL EPIGENETIC ANALYSIS OF CLOZAPINE USE IN TREATMENT-RESISTANT SCHIZOPHRENIA: DATA FROM THE CRESTAR CONSORTIUM</a></div> <div class="author"> Amy Gillespie, Eilis Hannon, Emma Dempster, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'David Collier, Alice Egerton, Jonathan Mill, James MacCabe' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 24th World Congress of Psychiatric Genetics (WCPG), Jerusalem, ISRAEL, OCT 30-NOV 04, 2016 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.euroneuro.2016.09.538" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.euroneuro.2016.09.538" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800392" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">METHYLATION QUANTITATIVE TRAIT LOCI (MQTL) IN THE DEVELOPING HUMAN BRAIN AND THEIR ENRICHMENT IN GENOMIC REGIONS ASSOCIATED WITH SCHIZOPHRENIA</a></div> <div class="author"> Eilis Hannon, Helen Spiers, Joana Viana, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Ruth Pidsley, Joe Burrage, Therese Murphy, Claire Troakes, Gustavo Turecki, Michael O’Donovan, Leonard Schalkwyk, Nicholas J. Bray, Jonathan Mill' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800333" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">CHARACTERIZING INTER-INDIVIDUAL VARIATION IN DNA METHYLATION BETWEEN BLOOD AND BRAIN: IMPLICATIONS FOR EPIGENETIC STUDIES OF PSYCHIATRIC PHENOTYPES</a></div> <div class="author"> Eilis Hannon, Katie Lunnon, Leonard Schalkwyk, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Jonathan Mill' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800136" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">AN EPIGENOME-WIDE ASSOCIATION STUDY OF CLOZAPINE USE IN TREATMENT-RESISTANT SCHIZOPHRENIA</a></div> <div class="author"> Eilis Hannon, Emma Dempster, Joe Burrage, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Charles Curtis, Amy Gillespie, David Dempster, Cerisse Gunasinghe, Leonard Schalkwyk, Fiona Gaughran, Robin Murray, Marta Di Forti, David Collier, Gerome Breen, Jonathan Mill, CRESTAR Consortium' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800393" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">AN INTEGRATED GENETIC-EPIGENETIC ANALYSIS OF SCHIZOPHRENIA</a></div> <div class="author"> Ellis Hannon, Emma Dempster, Joe Burrage, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Adam Smith, Hugh Gurling, Nicholas Basso, Andrew McQuillin, Leo Schalkwyk, Jonathan Mill' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800250" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">METHYLOME CELLULAR SIGNATURES INDUCED BY ACUTE EXPOSURE TO TWELVE ANTIPSYCHOTIC AND NEUROTRANSMITTER DRUGS</a></div> <div class="author"> Aaron Jeffries, Eilis Hannon, Emma Dempster, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Joe Burrage, David Evans, David Collier, Jonathan Mill' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800328" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">GENOME-WIDE METHYLATION ANALYSIS IN THE BRAINS OF DEPRESSED SUICIDE COMPLETERS</a></div> <div class="author"> Therese Murphy, Emma Dempster, Eilis Hannon, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Joe Burrage, Gustavo Turecki, Jonathan Mill' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800396" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">EPIGENETIC ALTERATIONS THROUGH SURGERY AND POSTOPERATIVE DELIRIUM: PILOT STUDY</a></div> <div class="author"> Ryoichi Sadahiro, Knight Bridget, Neil Smart, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'John Charity, Aaron Jeffries, Joe Burrage, Eilis Hannon, Emma Dempster, Therese Murphy, Katie Lunnon, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800397" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">DNA HYDROXYMETHYLATION ASSOCIATED WITH HUMAN FETAL BRAIN DEVELOPMENT</a></div> <div class="author"> Helen Spiers, Ellis Hannon, Leo Schalkwyk, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Nick Bray, Jonathan Mill' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800309" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">INVESTIGATING EPIGENOMIC REGULATION IN SCHIZOPHRENIA</a></div> <div class="author"> Joana Viana, Ruth Pidsley, Eilis Hannon, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Helen Spiers, Claire Troakes, Safa Al-Saraj, Naguib Mechawar, Gustavo Turecki, Leo Schalkwyk, Nicholas J. Bray, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800193" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">BEHAVIOURAL AND TRANSCRIPTOMIC CORRELATES WITH CLOZAPINE RESPONSE IN ZEBRAFISH</a></div> <div class="author"> Joana Viana, Nick Wildman, Gregory Paull, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Eduarda Santos, Jonathan Mill' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800207" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">METHYLOMIC ANALYSIS OF SALIVARY DNA IN CHILDHOOD ADHD IDENTIFIES ALTERED DNA METHYLATION IN VIPR2</a></div> <div class="author"> Beth Wilmot, Rebecca Fry, Lisa Smeester, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Erica Musser, Jonathan Mill, Joel Nigg Nigg' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800253" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">METHYLOMIC ANALYSIS OF AUTISM BRAIN: DISEASE ASSOCIATED VARIATION AND SUPPORT FOR A NEURODEVELOPMENTAL COMPONENT TO ETIOLOGY</a></div> <div class="author"> Chloe Chung Yi Wong, Neel Parikshak, Helen Spiers, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Nicholas J. Brays, Laura Lysenko, Claire Troakes, Joana Viana, Eilis Hannon, Leo Schalkwyk, Daniel Geschwind, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000413843800377" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">AN INVESTIGATION OF X-CHROMOSOME INACTIVATION PROFILES USING A GENETICALLY SENSITIVE DISCORDANT TWIN DESIGN</a></div> <div class="author"> Baocong Xia, Agnieszka Kepa, Emma Colvert, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Emma Mea-burn, Angelica Ronald, Leonard Schalkwyk, Jonathan Mill, Robert Plomin, Francesca Happe, Chloe Chung Yi Wong' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>EUROPEAN NEUROPSYCHOPHARMACOLOGY</em>, Oct 2017 </div> <div class="periodical"> 23rd Annual World Congress of Psychiatric Genetics (WCPG), Toronto, CANADA, OCT 16-20, 2015 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000410996700006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s12864-017-4091-x" rel="external nofollow noopener" target="_blank">5-hydroxymethylcytosine is highly dynamic across human fetal brain development</a></div> <div class="author"> Helen Spiers, Eilis Hannon, Leonard C. Schalkwyk, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Nicholas J. Bray, Jonathan Mill' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>BMC GENOMICS</em>, Sep 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s12864-017-4091-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-017-4091-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Epigenetic processes play a key role in orchestrating transcriptional regulation during the development of the human central nervous system. We previously described dynamic changes in DNA methylation (5mC) occurring during human fetal brain development, but other epigenetic processes operating during this period have not been extensively explored. Of particular interest is DNA hydroxymethylation (5hmC), a modification that is enriched in the human brain and hypothesized to play an important role in neuronal function, learning and memory. In this study, we quantify 5hmC across the genome of 71 human fetal brain samples spanning 23 to 184 days post-conception. Results: We identify widespread changes in 5hmC occurring during human brain development, notable sex-differences in 5hmC in the fetal brain, and interactions between 5mC and 5hmC at specific sites. Finally, we identify loci where 5hmC in the fetal brain is associated with genetic variation. Conclusions: This study represents the first systematic analysis of dynamic changes in 5hmC across human neurodevelopment and highlights the potential importance of this modification in the human brain.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000405649600002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1371/journal.pone.0177814" rel="external nofollow noopener" target="_blank">ANK1 is up-regulated in laser captured microglia in Alzheimer’s brain; the importance of addressing cellular heterogeneity</a></div> <div class="author"> Diego Mastroeni, Shobana Sekar, Jennifer Nolz, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Elaine Delvaux, Katie Lunnon, Jonathan Mill, Winnie S. Liang, Paul D. Coleman' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>PLOS ONE</em>, Jul 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1371/journal.pone.0177814" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0177814" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Recent epigenetic association studies have identified a new gene, ANK1, in the pathogenesis of Alzheimer’s disease (AD). Although strong associations were observed, brain homogenates were used to generate the data, introducing complications because of the range of cell types analyzed. In order to address the issue of cellular heterogeneity in homogenate samples we isolated microglial, astrocytes and neurons by laser capture microdissection from CA1 of hippocampus in the same individuals with a clinical and pathological diagnosis of AD and matched control cases. Using this unique RNAseq data set, we show that in the hippocampus, ANK1 is significantly (p&lt;0.0001) up-regulated 4-fold in AD microglia, but not in neurons or astrocytes from the same individuals. These data provide evidence that microglia are the source of ANK1 differential expression previously identified in homogenate samples in AD.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000402700600010" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.ajhg.2017.04.013" rel="external nofollow noopener" target="_blank">Pleiotropic Effects of Trait-Associated Genetic Variation on DNA Methylation: Utility for Refining GWAS Loci</a></div> <div class="author"> Eilis Hannon, Mike Weedon, Nicholas Bray, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Michael O’Donovan, Jonathan Mill' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>AMERICAN JOURNAL OF HUMAN GENETICS</em>, Jun 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.ajhg.2017.04.013" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ajhg.2017.04.013" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Most genetic variants identified in genome-wide association studies (GWASs) of complex traits are thought to act by affecting gene regulation rather than directly altering the protein product. As a consequence, the actual genes involved in disease are not necessarily the most proximal to the associated variants. By integrating data from GWAS analyses with those from genetic studies of regulatory variation, it is possible to identify variants pleiotropically associated with both a complex trait and measures of gene regulation. In this study, we used summary-data-based Mendelian randomization (SMR), a method developed to identify variants pleiotropically associated with both complex traits and gene expression, to identify variants associated with complex traits and DNA methylation. We used large DNA methylation quantitative trait locus (mQTL) datasets generated from two different tissues (blood and fetal brain) to prioritize genes for &gt;40 complex traits with robust GWAS data and found considerable overlap with the results of SMR analyses performed with expression QTL (eQTL) data. We identified multiple examples of variable DNA methylation associated with GWAS variants for a range of complex traits, demonstrating the utility of this approach for refining genetic association signals.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000401509300001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/ncomms15353" rel="external nofollow noopener" target="_blank">Genetic architecture of epigenetic and neuronal ageing rates in human brain regions</a></div> <div class="author"> Ake T. Lu, Eilis Hannon, Morgan E. Levine, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Eileen M. Crimmins, Katie Lunnon, Jonathan Mill, Daniel H. Geschwind, Steve Horvath' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, May 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/ncomms15353" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/ncomms15353" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Identifying genes regulating the pace of epigenetic ageing represents a new frontier in genome-wide association studies (GWASs). Here using 1,796 brain samples from 1,163 individuals, we carry out a GWAS of two DNA methylation-based biomarkers of brain age: the epigenetic ageing rate and estimated proportion of neurons. Locus 17q11.2 is significantly associated (P = 4.5 x 10(-9)) with the ageing rate across five brain regions and harbours a cis-expression quantitative trait locus for EFCAB5 (P = 3.4 x 10(-20)). Locus 1p36.12 is significantly associated (P = 2.2 x 10(-8)) with epigenetic ageing of the prefrontal cortex, independent of the proportion of neurons. Our GWAS of the proportion of neurons identified two genome-wide significant loci (10q26 and 12p13.31) and resulted in a gene set that overlaps significantly with sets found by GWAS of age-related macular degeneration (P = 1.4 x 10(-12)), ulcerative colitis (P &lt; 1.0 x 10(-20)), type 2 diabetes (P = 2.8 x 10(-13)), hip/waist circumference in men (P = 1.1 x 10(-9)), schizophrenia (P = 1.6 x 10(-9)), cognitive decline (P = 5.3 x 10(-4)) and Parkinson’s disease (P = 8.6 x 10(-3)).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000400790400001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13148-017-0337-3" rel="external nofollow noopener" target="_blank">Regional differences in mitochondrial DNA methylation in human post-mortem brain tissue</a></div> <div class="author"> Matthew Devall, Rebecca G. Smith, Aaron Jeffries, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Eilis Hannon, Matthew N. Davies, Leonard Schalkwyk, Jonathan Mill, Michael Weedon, Katie Lunnon' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>CLINICAL EPIGENETICS</em>, May 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13148-017-0337-3" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13148-017-0337-3" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: DNA methylation is an important epigenetic mechanism involved in gene regulation, with alterations in DNA methylation in the nuclear genome being linked to numerous complex diseases. Mitochondrial DNA methylation is a phenomenon that is receiving ever-increasing interest, particularly in diseases characterized by mitochondrial dysfunction; however, most studies have been limited to the investigation of specific target regions. Analyses spanning the entire mitochondrial genome have been limited, potentially due to the amount of input DNA required. Further, mitochondrial genetic studies have been previously confounded by nuclear-mitochondrial pseudogenes. Methylated DNA Immunoprecipitation Sequencing is a technique widely used to profile DNA methylation across the nuclear genome; however, reads mapped to mitochondrial DNA are often discarded. Here, we have developed an approach to control for nuclear-mitochondrial pseudogenes within Methylated DNA Immunoprecipitation Sequencing data. We highlight the utility of this approach in identifying differences in mitochondrial DNA methylation across regions of the human brain and pre-mortem blood. Results: We were able to correlate mitochondrial DNA methylation patterns between the cortex, cerebellum and blood. We identified 74 nominally significant differentially methylated regions (p &lt; 0.05) in the mitochondrial genome, between anatomically separate cortical regions and the cerebellum in matched samples (N = 3 matched donors). Further analysis identified eight significant differentially methylated regions between the total cortex and cerebellum after correcting for multiple testing. Using unsupervised hierarchical clustering analysis of the mitochondrial DNA methylome, we were able to identify tissue-specific patterns of mitochondrial DNA methylation between blood, cerebellum and cortex. Conclusions: Our study represents a comprehensive analysis of the mitochondrial methylome using pre-existing Methylated DNA Immunoprecipitation Sequencing data to identify brain region-specific patterns of mitochondrial DNA methylation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000400417800005" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.jaac.2017.02.006" rel="external nofollow noopener" target="_blank">Paternal Age Alters Social Development in Offspring</a></div> <div class="author"> Magdalena Janecka, Claire M. A. Haworth, Angelica Ronald, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Eva Krapohl, Francesca Happe, Jonathan Mill, Leonard C. Schalkwyk, Cathy Fernandes, Abraham Reichenberg, Fruhling Rijsdijk' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>JOURNAL OF THE AMERICAN ACADEMY OF CHILD AND ADOLESCENT PSYCHIATRY</em>, May 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.jaac.2017.02.006" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jaac.2017.02.006" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Objective: Advanced paternal age (APA) at conception has been linked with autism and schizophrenia in offspring, neurodevelopmental disorders that affect social functioning. The current study explored the effects of paternal age on social development in the general population. Method: We used multilevel growth modeling to investigate APA effects on socioemotional development from early childhood until adolescence, as measured by the Strengths and Difficulties Questionnaire (SDQ) in the Twins Early Development Study (TEDS) sample. We also investigated genetic and environmental underpinnings of the paternal age effects on development, using the Additive genetics, Common environment, unique Environment (ACE) and gene environment (GxE) models. Results: In the general population, both very young and advanced paternal ages were associated with altered trajectory of social development (intercept: p = .01; slope: p = .03). No other behavioral domain was affected by either young or advanced age at fatherhood, suggesting specificity of paternal age effects. Increased importance of genetic factors in social development was recorded in the offspring of older but not very young fathers, suggesting distinct underpinnings of the paternal age effects at these two extremes. Conclusion: Our findings highlight that the APA-related deficits that lead to autism and schizophrenia are likely continuously distributed in the population.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000397126200548" class="col-sm-8"> <div class="title"><a href="https://doi.org/" rel="external nofollow noopener" target="_blank">LONGITUDINAL EPIGENETIC ANALYSIS OF CLOZAPINE USE IN TREATMENT-RESISTANT SCHIZOPHRENIA</a></div> <div class="author"> Amy Gillespie, Eilis Hannon, Emma Dempster, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'David Collier, Alice Egerton, Jonathan Mill, James MacCabe' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>SCHIZOPHRENIA BULLETIN</em>, Mar 2017 </div> <div class="periodical"> 16th International Congress on Schizophrenia Research (ICOSR), San Diego, CA, MAR 24-28, 2017 </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000397280100008" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1002/ana.24840" rel="external nofollow noopener" target="_blank">Genetic Variation at 16q24.2 Is Associated With Small Vessel Stroke</a></div> <div class="author"> Matthew Traylor, Rainer Malik, Mike A. Nalls, and <span class="more-authors" title="click to view 69 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '69 more authors' ? 'Ioana Cotlarciuc, Farid Radmanesh, Gudmar Thorleifsson, Ken B. Hanscombe, Carl Langefeld, Danish Saleheen, Natalia S. Rost, Idil Yet, Tim D. Spector, Jordana T. Bell, Eilis Hannon, Jonathan Mill, Ganesh Chauhan, Stephanie Debette, Joshua C. Bis, W. T. Longstreth, M. Arfan Ikram, Lenore J. Launer, Sudha Seshadri, Monica Anne Hamilton-Bruce, Jordi Jimenez-Conde, John W. Cole, Reinhold Schmidt, Agnieszka Slowik, Robin Lemmens, Arne Lindgren, Olle Melander, Raji P. Grewal, Ralph L. Sacco, Tatjana Rundek, Kathryn Rexrode, Donna K. Arnett, Julie A. Johnson, Oscar R. Benavente, Sylvia Wasssertheil-Smoller, Jin-Moo Lee, Sara L. Pulit, Quenna Wong, Stephen S. Rich, Paul I. W. Bakker, Patrick F. McArdle, Daniel Woo, Christopher D. Anderson, Huichun Xu, Laura Heitsch, Myriam Fornage, Christina Jern, Kari Stefansson, Unnur Thorsteinsdottir, Solveig Gretarsdottir, Cathryn M. Lewis, Pankaj Sharma, Cathie L. M. Sudlow, Peter M. Rothwell, Giorgio B. Boncoraglio, Vincent Thijs, Chris Levi, James F. Meschia, Jonathan Rosand, Steven J. Kittner, Braxton D. Mitchell, Martin Dichgans, Bradford B. Worrall, Hugh S. Markus, METASTROKE, UK Young Lacunar DNA Study, NINDS Stroke Genetics Network, Neurology Working Grp CHARGE Conso, Int Stroke Genetics Consortium' : '69 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">69 more authors</span> </div> <div class="periodical"> <em>ANNALS OF NEUROLOGY</em>, Mar 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1002/ana.24840" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1002/ana.24840" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Objective: Genome-wide association studies (GWAS) have been successful at identifying associations with stroke and stroke subtypes, but have not yet identified any associations solely with small vessel stroke (SVS). SVS comprises one quarter of all ischemic stroke and is a major manifestation of cerebral small vessel disease, the primary cause of vascular cognitive impairment. Studies across neurological traits have shown that younger-onset cases have an increased genetic burden. We leveraged this increased genetic burden by performing an age-at-onset informed GWAS metaanalysis, including a large younger-onset SVS population, to identify novel associations with stroke. Methods: We used a three-stage age-at-onset informed GWAS to identify novel genetic variants associated with stroke. On identifying a novel locus associated with SVS, we assessed its influence on other small vessel disease phenotypes, as well as on messenger RNA (mRNA) expression of nearby genes, and on DNA methylation of nearby CpG sites in whole blood and in the fetal brain. Results: We identified an association with SVS in 4,203 cases and 50,728 controls on chromosome 16q24.2 (odds ratio [OR; 95% confidence interval {CI}] = 1.16 [1.10-1.22]; p=3.2x10(-9)). The lead single-nucleotide polymorphism (rs12445022) was also associated with cerebral white matter hyperintensities (OR [95% CI] = 1.10 [1.05-1.16]; p=5.3x10(-5); N=3,670), but not intracerebral hemorrhage (OR [95% CI] 50.97 [0.84-1.12]; p=0.71; 1,545 cases, 1,481 controls). rs12445022 is associated with mRNA expression of ZCCHC14 in arterial tissues (p=9.4x10(-7)) and DNA methylation at probe cg16596957 in whole blood (p=5.3x10(-6)). Interpretation: 16q24.2 is associated with SVS. Associations of the locus with expression of ZCCHC14 and DNA methylation suggest the locus acts through changes to regulatory elements.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000396524000006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.biopsych.2016.10.019" rel="external nofollow noopener" target="_blank">Profiling Regulatory Variation in the Brain: Methods for Exploring the Neuronal Epigenome</a></div> <div class="author"> Aaron R. Jeffries, and <em>Jonathan Mill</em> </div> <div class="periodical"> <em>BIOLOGICAL PSYCHIATRY</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.biopsych.2016.10.019" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.biopsych.2016.10.019" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000394383700001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s12888-016-1177-y" rel="external nofollow noopener" target="_blank">Is treatment-resistant schizophrenia categorically distinct from treatment-responsive schizophrenia? a systematic review</a></div> <div class="author"> Amy L. Gillespie, Ruta Samanaite, <em>Jonathan Mill</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Alice Egerton, James H. MacCabe' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>BMC PSYCHIATRY</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s12888-016-1177-y" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12888-016-1177-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Schizophrenia is a highly heterogeneous disorder, and around a third of patients are treatment-resistant. The only evidence-based treatment for these patients is clozapine, an atypical antipsychotic with relatively weak dopamine antagonism. It is plausible that varying degrees of response to antipsychotics reflect categorically distinct illness subtypes, which would have significant implications for research and clinical practice. If these subtypes could be distinguished at illness onset, this could represent a first step towards personalised medicine in psychiatry. This systematic review investigates whether current evidence supports conceptualising treatment-resistant and treatment-responsive schizophrenoa as categorically distinct subtypes. Method: A systematic literature search was conducted, using PubMed, EMBASE, PsycInfo, CINAHL and OpenGrey databases, to identify all studies which compared treatment-resistant schizophrenia (defined as either a lack of response to two antipsychotic trials or clozapine prescription) to treatment-responsive schizophrenia (defined as known response to non-clozapine antipsychotics). Results: Nineteen studies of moderate quality met inclusion criteria. The most robust findings indicate that treatment-resistant patients show glutamatergic abnormalities, a lack of dopaminergic abnormalities, and significant decreases in grey matter compared to treatment-responsive patients. Treatment-resistant patients were also reported to have higher familial loading; however, no individual gene-association study reported their findings surviving correction for multiple comparisons. Conclusions: Tentative evidence supports conceptualising treatment-resistant schizophrenia as a categorically different illness subtype to treatment-responsive schizophrenia. However, research is limited and confirmation will require replication and rigorously controlled studies with large sample sizes and prospective study designs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000390556400006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1159/000445875" rel="external nofollow noopener" target="_blank">Rapid Down-Regulation of Glucocorticoid Receptor Gene Expression in the Dentate Gyrus after Acute Stress in vivo: Role of DNA Methylation and MicroRNA Activity</a></div> <div class="author"> Karen R. Mifsud, Emily A. Saunderson, Helen Spiers, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Sylvia D. Carter, Alexandra F. Trollope, Jonathan Mill, Johannes M. H. M. Reul' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>NEUROENDOCRINOLOGY</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1159/000445875" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1159/000445875" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Although glucocorticoid receptors (GRs) in the hippocampus play a vital role in the regulation of physiological and behavioural responses to stress, the regulation of receptor expression remains unclear. This work investigates the molecular mechanisms underpinning stress-induced changes in hippocampal GR mRNA levels in vivo. Methods: Male Wistar rats were killed either under baseline conditions or after forced swim stress (FSS; 15 min in 25 degrees C water). Rat hippocampi were micro-dissected (for mRNA, microRNA, and DNA methylation analysis) or frozen whole (for chromatin immunoprecipitation). In an additional experiment, rats were pre-treated with RU486 (a GR antagonist) or vehicle. Results: FSS evoked a dentate gyrus-specific reduction in GR mRNA levels. This was related to an increased DNMT3a protein association with a discreet region of the Nr3c1 (GR gene) promoter, shown here to undergo increased DNA methylation after FSS. FSS also caused a time-dependent increase in the expression of miR-124a, a microRNA known to reduce GR mRNA expression, which was inversely correlated with a reduction in GR mRNA levels 30 min after FSS. FSS did not affect GR binding to a putative negative glucocorticoid response element within the Nr3c1 gene. Conclusions: Acute stress results in decreased GR mRNA expression specifically in the dentate gyrus. Our results indicate that a complex interplay of multiple molecular mechanisms - including increased DNA methylation of discrete CpG residues within the Nr3c1 gene, most likely facilitated by DNMT3a, and increased expression of miR-124a - could be responsible for these changes. (C) 2016 S. Karger AG, Basel.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000397064600017" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/hmg/ddw373" rel="external nofollow noopener" target="_blank">Schizophrenia-associated methylomic variation: molecular signatures of disease and polygenic risk burden across multiple brain regions</a></div> <div class="author"> Joana Viana, Eilis Hannon, Emma Dempster, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Ruth Pidsley, Ruby Macdonald, Olivia Knox, Helen Spiers, Claire Troakes, Safa Al-Saraj, Gustavo Turecki, Leonard C. Schalkwyk, Jonathan Mill' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>HUMAN MOLECULAR GENETICS</em>, Jan 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/hmg/ddw373" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/hmg/ddw373" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Genetic association studies provide evidence for a substantial polygenic component to schizophrenia, although the neurobiological mechanisms underlying the disorder remain largely undefined. Building on recent studies supporting a role for developmentally regulated epigenetic variation in the molecular aetiology of schizophrenia, this study aimed to identify epigenetic variation associated with both a diagnosis of schizophrenia and elevated polygenic risk burden for the disease across multiple brain regions. Genome-wide DNA methylation was quantified in 262 post-mortem brain samples, representing tissue from four brain regions (prefrontal cortex, striatum, hippocampus and cerebellum) from 41 schizophrenia patients and 47 controls. We identified multiple disease-associated and polygenic risk score-associated differentially methylated positions and regions, which are not enriched in genomic regions identified in genetic studies of schizophrenia and do not reflect direct genetic effects on DNA methylation. Our study represents the first analysis of epigenetic variation associated with schizophrenia across multiple brain regions and highlights the utility of polygenic risk scores for identifying molecular pathways associated with aetiological variation in complex disease.</p> </div> </div> </div> </li> </ol> <h2 class="year">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000389100600024" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.jpsychires.2016.09.010" rel="external nofollow noopener" target="_blank">Epigenetic signatures of childhood abuse and neglect: Implications for psychiatric vulnerability</a></div> <div class="author"> Charlotte A. M. Cecil, Rebecca G. Smith, Esther Walton, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jonathan Mill, Eamon J. McCrory, Essi Viding' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>JOURNAL OF PSYCHIATRIC RESEARCH</em>, Dec 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.jpsychires.2016.09.010" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jpsychires.2016.09.010" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Childhood maltreatment is a key risk factor for poor mental and physical health. Recently, variation in epigenetic processes, such as DNA methylation, has emerged as a potential pathway mediating this association; yet, the extent to which different forms of maltreatment may be characterized by unique vs shared epigenetic signatures is currently unknown. In this study, we quantified DNA methylation across the genome in buccal epithelial cell samples from a high-risk sample of inner-city youth (n = 124; age = 16-24; 53% female), 68% of whom reported experiencing at least one form of maltreatment while growing up. Our analyses aimed to identify methylomic variation associated with exposure to five major types of childhood maltreatment. We found that: (i) maltreatment types differ in the extent to which they associate with methylomic variation, with physical exposures showing the strongest associations; (ii) many of the identified loci are annotated to genes previously implicated in stress-related outcomes, including psychiatric and physical disorders (e.g. GABBR1, GRIN2D, CACNA2D4, PSEN2); and (iii) based on gene ontology analyses, maltreatment types not only show unique methylation patterns enriched for specific biological processes (e.g. physical abuse and cardiovascular function), but also share a ‘common’ epigenetic signature enriched for biological processes related to neural development and organismal growth. A stringent set of sensitivity analyses were also run to identify high-confidence associations. Together, findings lend novel insights into epigenetic signatures of childhood abuse and neglect, point to novel potential biomarkers for future investigation and support a molecular link between maltreatment and poor health outcomes. Nevertheless, it will be important in future to replicate findings, as the use of cross-sectional data and high rates of polyvictimization in our study make it difficult to fully disentangle the shared vs unique epigenetic signatures of maltreatment types. Furthermore, studies will be needed to test the role of potential moderators in the identified associations, including age of onset and chronicity of maltreatment exposure. (C) 2016 Elsevier Ltd. All rights reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000389470100026" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.cell.2016.10.031" rel="external nofollow noopener" target="_blank">Histone Acetylome-wide Association Study of Autism Spectrum Disorder</a></div> <div class="author"> Wenjie Sun, Jeremie Poschmann, Ricardo Cruz-Herrera Rosario, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Neelroop N. Parikshak, Hajira Shreen Hajan, Vibhor Kumar, Ramalakshmi Ramasamy, T. Grant Belgard, Bavani Elanggovan, Chloe Chung Yi Wong, Jonathan Mill, Daniel H. Geschwind, Shyam Prabhakar' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>CELL</em>, Nov 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.cell.2016.10.031" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2016.10.031" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The association of histone modification changes with autism spectrum disorder (ASD) has not been systematically examined. We conducted a histone acetylome-wide association study (HAWAS) by performing H3K27ac chromatin immunoprecipitation sequencing (ChIP-seq) on 257 postmortem samples from ASD and matched control brains. Despite etiological heterogeneity, &gt;= 68% of syndromic and idiopathic ASD cases shared a common acetylome signature at &gt;5,000 cis-regulatory elements in prefrontal and temporal cortex. Similarly, multiple genes associated with rare genetic mutations in ASD showed common “epi-mutations.” Acetylome aberrations in ASD were not attributable to genetic differentiation at cis-SNPs and highlighted genes involved in synaptic transmission, ion transport, epilepsy, behavioral abnormality, chemokinesis, histone deacetylation, and immunity. By correlating histone acetylation with genotype, we discovered &gt;2,000 histone acetylation quantitative trait loci (haQTLs) in human brain regions, including four candidate causal variants for psychiatric diseases. Due to the relative stability of histone modifications postmortem, we anticipate that the HAWAS approach will be applicable to multiple diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000386976500004" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neurobiolaging.2016.07.008" rel="external nofollow noopener" target="_blank">Increased DNA methylation near TREM2 is consistently seen in the superior temporal gyrus in Alzheimer’s disease brain</a></div> <div class="author"> Adam R. Smith, Rebecca G. Smith, Daniel Condliffe, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Eilis Hannon, Leonard Schalkwyk, Jonathan Mill, Katie Lunnon' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>NEUROBIOLOGY OF AGING</em>, Nov 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neurobiolaging.2016.07.008" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neurobiolaging.2016.07.008" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Although mutations within the TREM2 gene have been robustly associated with Alzheimer’s disease, it is not known whether alterations in the regulation of this gene are also involved in pathogenesis. Here, we present data demonstrating increased DNA methylation in the superior temporal gyrus in Alzheimer’s disease brain at a CpG site located 289 bp upstream of the transcription start site of the TREM2 gene in 3 independent study cohorts using 2 different technologies (Illumina Infinium 450K methylation beadchip and pyrosequencing). A meta-analysis across all 3 cohorts reveals consistent AD-associated hyper-methylation (p = 3.47E-08). This study highlights that extending genetic studies of TREM2 in AD to investigate epigenetic changes may nominate additional mechanisms by which disruption to this gene increases risk. (C) 2016 Elsevier Inc. All rights reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000384706000012" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1261/rna.058347.116" rel="external nofollow noopener" target="_blank">Erasure and reestablishment of random allelic expression imbalance after epigenetic reprogramming</a></div> <div class="author"> Aaron Richard Jeffries, Dafe Aghogho Uwanogho, Graham Cocks, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Leo William Perfect, Emma Dempster, Jonathan Mill, Jack Price' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>RNA</em>, Oct 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1261/rna.058347.116" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1261/rna.058347.116" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Clonal level random allelic expression imbalance and random monoallelic expression provides cellular heterogeneity within tissues by modulating allelic dosage. Although such expression patterns have been observed in multiple cell types, little is known about when in development these stochastic allelic choices are made. We examine allelic expression patterns in human neural progenitor cells before and after epigenetic reprogramming to induced pluripotency, observing that loci previously characterized by random allelic expression imbalance (0.63% of expressed genes) are generally reset to a biallelic state in induced pluripotent stem cells (iPSCs). We subsequently neuralized the iPSCs and profiled isolated clonal neural stem cells, observing that significant random allelic expression imbalance is reestablished at 0.65% of expressed genes, including novel loci not found to show allelic expression imbalance in the original parental neural progenitor cells. Allelic expression imbalance was associated with altered DNA methylation across promoter regulatory regions, with clones characterized by skewed allelic expression being hypermethylated compared to their biallelic sister clones. Our results suggest that random allelic expression imbalance is established during lineage commitment and is associated with increased DNA methylation at the gene promoter.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000395807800016" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/hmg/ddw283" rel="external nofollow noopener" target="_blank">Collapsed methylation quantitative trait loci analysis for low frequency and rare variants</a></div> <div class="author"> Tom G. Richardson, Hashem A. Shihab, Gibran Hemani, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Jie Zheng, Eilis Hannon, Jonathan Mill, Elena Carnero-Montoro, Jordana T. Bell, Oliver Lyttleton, Wendy L. McArdle, Susan M. Ring, Santiago Rodriguez, Colin Campbell, George Davey Smith, Caroline L. Relton, Nicholas J. Timpson, Tom R. Gaunt' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>HUMAN MOLECULAR GENETICS</em>, Oct 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/hmg/ddw283" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/hmg/ddw283" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Single variant approaches have been successful in identifying DNA methylation quantitative trait loci (mQTL), although as with complex traits they lack the statistical power to identify the effects from rare genetic variants. We have undertaken extensive analyses to identify regions of low frequency and rare variants that are associated with DNA methylation levels. Methods: We used repeated measurements of DNA methylation from five different life stages in human blood, taken from the Avon Longitudinal Study of Parents and Children (ALSPAC) cohort. Variants were collapsed across CpG islands and their flanking regions to identify variants collectively associated with methylation, where no single variant was individually responsible for the observed signal. All analyses were undertaken using the sequence kernel association test. Results: For loci where no individual variant mQTL was observed based on a single variant analysis, we identified 95 unique regions where the combined effect of low frequency variants (MAF &lt;= 5%) provided strong evidence of association with methylation. For loci where there was previous evidence of an individual variant mQTL, a further 3 regions provided evidence of association between multiple low frequency variants and methylation levels. Effects were observed consistently across 5 different time points in the lifecourse and evidence of replication in the TwinsUK and Exeter cohorts was also identified. Conclusion: We have demonstrated the potential of this novel approach to mQTL analysis by analysing the combined effect of multiple low frequency or rare variants. Future studies should benefit from applying this approach as a complementary follow up to single variant analyses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000383427100001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13059-016-1041-x" rel="external nofollow noopener" target="_blank">An integrated genetic-epigenetic analysis of schizophrenia: evidence for co-localization of genetic associations and differential DNA methylation</a></div> <div class="author"> Eilis Hannon, Emma Dempster, Joana Viana, and <span class="more-authors" title="click to view 21 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '21 more authors' ? 'Joe Burrage, Adam R. Smith, Ruby Macdonald, David St Clair, Colette Mustard, Gerome Breen, Sebastian Therman, Jaakko Kaprio, Timothea Toulopoulou, Hilleke E. Hulshoff Pol, Marc M. Bohlken, Rene S. Kahn, Igor Nenadic, Christina M. Hultman, Robin M. Murray, David A. Collier, Nick Bass, Hugh Gurling, Andrew McQuillin, Leonard Schalkwyk, Jonathan Mill' : '21 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">21 more authors</span> </div> <div class="periodical"> <em>GENOME BIOLOGY</em>, Aug 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-016-1041-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-016-1041-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: Schizophrenia is a highly heritable, neuropsychiatric disorder characterized by episodic psychosis and altered cognitive function. Despite success in identifying genetic variants associated with schizophrenia, there remains uncertainty about the causal genes involved in disease pathogenesis and how their function is regulated. Results: We performed a multi-stage epigenome-wide association study, quantifying genome-wide patterns of DNA methylation in a total of 1714 individuals from three independent sample cohorts. We have identified multiple differentially methylated positions and regions consistently associated with schizophrenia across the three cohorts; these effects are independent of important confounders such as smoking. We also show that epigenetic variation at multiple loci across the genome contributes to the polygenic nature of schizophrenia. Finally, we show how DNA methylation quantitative trait loci in combination with Bayesian co-localization analyses can be used to annotate extended genomic regions nominated by studies of schizophrenia, and to identify potential regulatory variation causally involved in disease. Conclusions: This study represents the first systematic integrated analysis of genetic and epigenetic variation in schizophrenia, introducing a methodological approach that can be used to inform epigenome-wide association study analyses of other complex traits and diseases. We demonstrate the utility of using a polygenic risk score to identify molecular variation associated with etiological variation, and of using DNA methylation quantitative trait loci to refine the functional and regulatory variation associated with schizophrenia risk variants. Finally, we present strong evidence for the co-localization of genetic associations for schizophrenia and differential DNA methylation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000380633900002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/srep30861" rel="external nofollow noopener" target="_blank">Mapping epigenetic changes to the host cell genome induced by Burkholderia pseudomallei reveals pathogen-specific and pathogengeneric signatures of infection</a></div> <div class="author"> Deniz Cizmeci, Emma L. Dempster, Olivia L. Champion, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Sariqa Wagley, Ozgur E. Akman, Joann L. Prior, Orkun S. Soyer, Jonathan Mill, Richard W. Titball' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>SCIENTIFIC REPORTS</em>, Aug 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/srep30861" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/srep30861" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The potential for epigenetic changes in host cells following microbial infection has been widely suggested, but few examples have been reported. We assessed genome-wide patterns of DNA methylation in human macrophage-like U937 cells following infection with Burkholderia pseudomallei, an intracellular bacterial pathogen and the causative agent of human melioidosis. Our analyses revealed significant changes in host cell DNA methylation, at multiple CpG sites in the host cell genome, following infection. Infection induced differentially methylated probes (iDMPs) showing the greatest changes in DNA methylation were found to be in the vicinity of genes involved in inflammatory responses, intracellular signalling, apoptosis and pathogen-induced signalling. A comparison of our data with reported methylome changes in cells infected with M. tuberculosis revealed commonality of differentially methylated genes, including genes involved in T cell responses (BCL11B, FOXO1, KIF13B, PAWR, SOX4, SYK), actin cytoskeleton organisation (ACTR3, CDC42BPA, DTNBP1, FERMT2, PRKCZ, RAC1), and cytokine production (FOXP1, IRF8, MR1). Overall our findings show that pathogenic-specific and pathogen-common changes in the methylome occur following infection.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000379373900009" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.neulet.2016.02.013" rel="external nofollow noopener" target="_blank">Epigenetic regulation of mitochondrial function in neurodegenerative disease: New insights from advances in genomic technologies</a></div> <div class="author"> Matthew Devall, Janou Roubroeks, <em>Jonathan Mill</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Michael Weedon, Katie Lunnon' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>NEUROSCIENCE LETTERS</em>, Jun 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.neulet.2016.02.013" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.neulet.2016.02.013" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The field of mitochondrial epigenetics has received increased attention in recent years and changes in mitochondrial DNA (mtDNA) methylation has been implicated in a number of diseases, including neurodegenerative diseases such as amyotrophic lateral sclerosis. However, current publications have been limited by the use of global or targeted methods of measuring DNA methylation. In this review, we discuss current findings in mitochondrial epigenetics as well as its potential role as a regulator of mitochondria within the brain. Finally, we summarize the current technologies best suited to capturing mtDNA methylation, and how a move towards whole epigenome sequencing of mtDNA may help to advance our current understanding of the field. (C) 2016 Elsevier Ireland Ltd. All rights reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000378900600002" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13059-016-0958-4" rel="external nofollow noopener" target="_blank">Variation in 5-hydroxymethylcytosine across human cortex and cerebellum (vol 17, 27, 2016)</a></div> <div class="author"> Katie Lunnon, Eilis Hannon, Rebecca G. Smith, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Emma Dempster, Chloe Wong, Joe Burrage, Claire Troakes, Safa Al-Sarraj, Agnieszka Kepa, Leonard Schalkwyk, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>GENOME BIOLOGY</em>, Jun 2016 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-016-0958-4" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-016-0958-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000374748400070" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1073/pnas.1524857113" rel="external nofollow noopener" target="_blank">Stress-induced gene expression and behavior are controlled by DNA methylation and methyl donor availability in the dentate gyrus</a></div> <div class="author"> Emily A. Saunderson, Helen Spiers, Karen R. Mifsud, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Maria Gutierrez-Mecinas, Alexandra F. Trollope, Abeera Shaikh, Jonathan Mill, Johannes M. H. M. Reul' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, Apr 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1073/pnas.1524857113" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1524857113" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Stressful events evoke long-term changes in behavioral responses; however, the underlying mechanisms in the brain are not well understood. Previous work has shown that epigenetic changes and immediate-early gene (IEG) induction in stress-activated dentate gyrus (DG) granule neurons play a crucial role in these behavioral responses. Here, we show that an acute stressful challenge [i.e., forced swimming (FS)] results in DNA demethylation at specific CpG (5’-cytosine-phosphateguanine-3’) sites close to the c-Fos (FBJ murine osteosarcoma viral oncogene homolog) transcriptional start site and within the gene promoter region of Egr-1 (early growth response protein 1) specifically in the DG. Administration of the (endogenous) methyl donor S-adenosyl methionine (SAM) did not affect CpG methylation and IEG gene expression at baseline. However, administration of SAM before the FS challenge resulted in an enhanced CpG methylation at the IEG loci and suppression of IEG induction specifically in the DG and an impaired behavioral immobility response 24 h later. The stressor also specifically increased the expression of the de novo DNA methyltransferase Dnmt3a [DNA (cytosine-5-)-methyltransferase 3 alpha] in this hippocampus region. Moreover, stress resulted in an increased association of Dnmt3a enzyme with the affected CpG loci within the IEG genes. No effects of SAM were observed on stress-evoked histone modifications, including H3S10p-K14ac (histone H3, phosphorylated serine 10 and acetylated lysine-14), H3K4me3 (histone H3, trimethylated lysine-4), H3K9me3 (histone H3, trimethylated lysine-9), and H3K27me3 (histone H3, trimethylated lysine-27). We conclude that the DNA methylation status of IEGs plays a crucial role in FS-induced IEG induction in DG granule neurons and associated behavioral responses. In addition, the concentration of available methyl donor, possibly in conjunction with Dnmt3a, is critical for the responsiveness of dentate neurons to environmental stimuli in terms of gene expression and behavior.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000372497900001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/ncomms10967" rel="external nofollow noopener" target="_blank">Genome-wide DNA methylation levels and altered cortisol stress reactivity following childhood trauma in humans</a></div> <div class="author"> Lotte C. Houtepen, Christiaan H. Vinkers, Tania Carrillo-Roa, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Marieke Hiemstra, Pol A. Lier, Wim Meeus, Susan Branje, Christine M. Heim, Charles B. Nemeroff, Jonathan Mill, Leonard C. Schalkwyk, Menno P. Creyghton, Rene S. Kahn, Marian Joels, Elisabeth B. Binder, Marco P. M. Boks' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Mar 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/ncomms10967" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/ncomms10967" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>DNA methylation likely plays a role in the regulation of human stress reactivity. Here we show that in a genome- wide analysis of blood DNA methylation in 85 healthy individuals, a locus in the Kit ligand gene (KITLG; cg27512205) showed the strongest association with cortisol stress reactivity (P = 5.8 x 10 (-6)). Replication was obtained in two independent samples using either blood (N = 45, P = 0.001) or buccal cells (N = 255, P = 0.004). KITLG methylation strongly mediates the relationship between childhood trauma and cortisol stress reactivity in the discovery sample (32% mediation). Its genomic location, a CpG island shore within an H3K27ac enhancer mark, and the correlation between methylation in the blood and prefrontal cortex provide further evidence that KITLG methylation is functionally relevant for the programming of stress reactivity in the human brain. Our results extend preclinical evidence for epigenetic regulation of stress reactivity to humans and provide leads to enhance our understanding of the neurobiological pathways underlying stress vulnerability.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000372938100003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1016/j.mad.2016.02.001" rel="external nofollow noopener" target="_blank">Age-associated changes in DNA methylation across multiple tissues in an inbred mouse model</a></div> <div class="author"> Helen Spiers, Eilis Hannon, Sara Wells, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Brenda Williams, Cathy Fernandes, Jonathan Mill' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>MECHANISMS OF AGEING AND DEVELOPMENT</em>, Mar 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1016/j.mad.2016.02.001" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.mad.2016.02.001" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Epigenetic disruption has been implicated in many diseases of aging, and age-associated DNA methylation changes at specific genomic loci in humans are strongly correlated with chronological age. The aim of this study was to explore the specificity of selected age-associated differentially methylated positions (aDMPs) identified in human epidemiological studies by quantifying DNA methylation across multiple tissues in homologous regions of the murine genome. We selected four high-confidence aDMPs (located in the vicinity of the ELOVL2, GLRA1, MYOD1 and PDE4C genes) and quantified DNA methylation across these regions in four tissues (blood, lung, cerebellum and hippocampus) from male and female C57BL/6J mice, ranging in age from fetal (embryonic day 17) to 630 days. We observed tissue-specific age-associated changes in DNA methylation that was directionally consistent with those observed in humans. These findings lend further support to the notion that changes in DNA methylation are associated with chronological age and suggest that these processes are often conserved across tissues and between mammalian species. Our data highlight the relevance of utilizing model systems, in which environmental and genetic influences can be carefully controlled, for the further study of these phenomena. (C) 2016 The Authors. Published by Elsevier Ireland Ltd.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000370236100001" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1186/s13059-016-0871-x" rel="external nofollow noopener" target="_blank">Variation in 5-hydroxymethylcytosine across human cortex and cerebellum</a></div> <div class="author"> Katie Lunnon, Eilis Hannon, Rebecca G. Smith, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Emma Dempster, Chloe Wong, Joe Burrage, Claire Troakes, Safa Al-Sarraj, Agnieszka Kepa, Leonard Schalkwyk, Jonathan Mill' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>GENOME BIOLOGY</em>, Feb 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1186/s13059-016-0871-x" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s13059-016-0871-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background: The most widely utilized approaches for quantifying DNA methylation involve the treatment of genomic DNA with sodium bisulfite; however, this method cannot distinguish between 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC). Previous studies have shown that 5hmC is enriched in the brain, although little is known about its genomic distribution and how it differs between anatomical regions and individuals. In this study, we combine oxidative bisulfite (oxBS) treatment with the Illumina Infinium 450K BeadArray to quantify genome-wide patterns of 5hmC in two distinct anatomical regions of the brain from multiple individuals. Results: We identify 37,145 and 65,563 sites passing our threshold for detectable 5hmC in the prefrontal cortex and cerebellum respectively, with 23,445 loci common across both brain regions. Distinct patterns of 5hmC are identified in each brain region, with notable differences in the genomic location of the most hydroxymethylated loci between these brain regions. Tissue-specific patterns of 5hmC are subsequently confirmed in an independent set of prefrontal cortex and cerebellum samples. Conclusions: This study represents the first systematic analysis of 5hmC in the human brain, identifying tissue-specific hydroxymethylated positions and genomic regions characterized by inter-individual variation in DNA hydroxymethylation. This study demonstrates the utility of combining oxBS-treatment with the Illumina 450k methylation array to systematically quantify 5hmC across the genome and the potential utility of this approach for epigenomic studies of brain disorders.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000371014500003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/ncomms10561" rel="external nofollow noopener" target="_blank">Genetic variants near MLST8 and DHX57 affect the epigenetic age of the cerebellum</a></div> <div class="author"> Ake T. Lu, Eilis Hannon, Morgan E. Levine, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Ke Hao, Eileen M. Crimmins, Katie Lunnon, Alexey Kozlenkov, Jonathan Mill, Stella Dracheva, Steve Horvath' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>NATURE COMMUNICATIONS</em>, Feb 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/ncomms10561" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/ncomms10561" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>DNA methylation (DNAm) levels lend themselves for defining an epigenetic biomarker of aging known as the ‘epigenetic clock’. Our genome-wide association study (GWAS) of cerebellar epigenetic age acceleration identifies five significant (P&lt;5.0 x 10(-8)) SNPs in two loci: 2p22.1 (inside gene DHX57) and 16p13.3 near gene MLST8 (a subunit of mTOR complex 1 and 2). We find that the SNP in 16p13.3 has a cis-acting effect on the expression levels of MLST8 (P = 6.9 x 10(-18)) in most brain regions. In cerebellar samples, the SNP in 2p22.1 has a cis-effect on DHX57 (P = 4.4 x 10(-5)). Gene sets found by our GWAS analysis of cerebellar age acceleration exhibit significant overlap with those of Alzheimer’s disease (P = 4.4 x 10(-15)), age-related macular degeneration (P = 6.4 x 10(-6)), and Parkinson’s disease (P = 2.6 x 10(-4)). Overall, our results demonstrate the utility of a new paradigm for understanding aging and age-related diseases: it will be fruitful to use epigenetic tissue age as endophenotype in GWAS.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000373925400005" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1080/15592294.2016.1145329" rel="external nofollow noopener" target="_blank">An epigenome-wide association meta-analysis of prenatal maternal stress in neonates: A model approach for replication</a></div> <div class="author"> Jolien Rijlaarsdam, Irene Pappa, Esther Walton, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Marian J. Bakermans-Kranenburg, Viara R. Mileva-Seitz, Ralph C. A. Rippe, Sabine J. Roza, Vincent W. V. Jaddoe, Frank C. Verhulst, Janine F. Felix, Charlotte A. M. Cecil, Caroline L. Relton, Tom R. Gaunt, Wendy McArdle, Jonathan Mill, Edward D. Barker, Henning Tiemeier, Marinus H. IJzendoorn' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>EPIGENETICS</em>, Feb 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1080/15592294.2016.1145329" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1080/15592294.2016.1145329" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Prenatal maternal stress exposure has been associated with neonatal differential DNA methylation. However, the available evidence in humans is largely based on candidate gene methylation studies, where only a few CpG sites were evaluated. The aim of this study was to examine the association between prenatal exposure to maternal stress and offspring genome-wide cord blood methylation using different methods. First, we conducted a meta-analysis and follow-up pathway analyses. Second, we used novel region discovery methods [i.e., differentially methylated regions (DMRs) analyses]. To this end, we used data from two independent population-based studies, the Generation R Study (n = 912) and the Avon Longitudinal Study of Parents and Children (ALSPAC, n = 828), to (i) measure genome-wide DNA methylation in cord blood and (ii) extract a prenatal maternal stress composite. The meta-analysis (n(total) = 1,740) revealed no epigenome-wide (meta P &lt;1.00e-07) associations of prenatal maternal stress exposure with neonatal differential DNA methylation. Follow-up analyses of the top hits derived from our epigenome-wide meta-analysis (meta P &lt;1.00e-04) indicated an over-representation of the methyltransferase activity pathway. We identified no Bonferroni-corrected (P &lt;1.00e-06) DMRs associated with prenatal maternal stress exposure. Combining data from two independent population-based samples in an epigenome-wide meta-analysis, the current study indicates that there are no large effects of prenatal maternal stress exposure on neonatal DNA methylation. Such replication efforts are essential in the search for robust associations, whether derived from candidate gene methylation or epigenome-wide studies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000368803300006" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1111/jcpp.12457" rel="external nofollow noopener" target="_blank">Methylomic analysis of salivary DNA in childhood ADHD identifies altered DNA methylation in VIPR2</a></div> <div class="author"> Beth Wilmot, Rebecca Fry, Lisa Smeester, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Erica D. Musser, Jonathan Mill, Joel T. Nigg' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>JOURNAL OF CHILD PSYCHOLOGY AND PSYCHIATRY</em>, Feb 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1111/jcpp.12457" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1111/jcpp.12457" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>BackgroundPeripheral epigenetic marks hold promise for understanding psychiatric illness and may represent fingerprints of gene-environment interactions. We conducted an initial examination of CpG methylation variation in children with or without attention-deficit/hyperactivity disorder (ADHD). MethodsChildren age 7-12 were recruited, screened, evaluated and assigned to ADHD or non-ADHD groups by defined research criteria. Two independent age-matched samples were examined, a discovery set (n=92, all boys, half control, half ADHD) and a confirmation set (n=20, half ADHD, all boys). 5-methylcytosine levels were quantified in salivary DNA using the Illumina 450K HumanMethylation array. Genes for which multiple probes were nominally significant and had a beta difference of at least 2% were evaluated for biological relevance and prioritized for confirmation and sequence validation. Gene pathways were explored and described. ResultsTwo genes met the criteria for confirmation testing, VIPR2 and MYT1L; both had multiple probes meeting cutoffs and strong biological relevance. Probes on VIPR2 passed FDR correction in the confirmation set and were confirmed through bisulfite sequencing. Enrichment analysis suggested involvement of gene sets or pathways related to inflammatory processes and modulation of monoamine and cholinergic neurotransmission. ConclusionsAlthough it is unknown to what extent CpG methylation seen in peripheral tissue reflect transcriptomic changes in the brain, these initial results indicate that peripheral DNA methylation markers in ADHD may be promising and suggest targeted hypotheses for future study in larger samples.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000372742300003" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1080/15592294.2015.1127479" rel="external nofollow noopener" target="_blank">Tissue-specific patterns of allelically-skewed DNA methylation</a></div> <div class="author"> Sarah J. Marzi, Emma L. Meaburn, Emma L. Dempster, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Katie Lunnon, Jose L. Paya-Cano, Rebecca G. Smith, Manuela Volta, Claire Troakes, Leonard C. Schalkwyk, Jonathan Mill' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>EPIGENETICS</em>, Jan 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1080/15592294.2015.1127479" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1080/15592294.2015.1127479" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>While DNA methylation is usually thought to be symmetrical across both alleles, there are some notable exceptions. Genomic imprinting and X chromosome inactivation are two well-studied sources of allele-specific methylation (ASM), but recent research has indicated a more complex pattern in which genotypic variation can be associated with allelically-skewed DNA methylation in cis. Given the known heterogeneity of DNA methylation across tissues and cell types we explored inter- and intra-individual variation in ASM across several regions of the human brain and whole blood from multiple individuals. Consistent with previous studies, we find widespread ASM with &gt; 4% of the similar to 220,000 loci interrogated showing evidence of allelically-skewed DNA methylation. We identify ASM flanking known imprinted regions, and show that ASM sites are enriched in DNase I hypersensitivity sites and often located in an extended genomic context of intermediate DNA methylation. We also detect examples of genotype-driven ASM, some of which are tissue-specific. These findings contribute to our understanding of the nature of differential DNA methylation across tissues and have important implications for genetic studies of complex disease. As a resource to the community, ASM patterns across each of the tissues studied are available in a searchable online database: [GRAPHICS] .</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000367254400012" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1038/nn.4182" rel="external nofollow noopener" target="_blank">Methylation QTLs in the developing brain and their enrichment in schizophrenia risk loci</a></div> <div class="author"> Eilis Hannon, Helen Spiers, Joana Viana, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Ruth Pidsley, Joe Burrage, Therese M. Murphy, Claire Troakes, Gustavo Turecki, Michael C. O’Donovan, Leonard C. Schalkwyk, Nicholas J. Bray, Jonathan Mill' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>NATURE NEUROSCIENCE</em>, Jan 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1038/nn.4182" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/nn.4182" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>We characterized DNA methylation quantitative trait loci (mQTLs) in a large collection (n = 166) of human fetal brain samples spanning 56-166 d post-conception, identifying &gt; 16,000 fetal brain mQTLs. Fetal brain mQTLs were primarily cis-acting, enriched in regulatory chromatin domains and transcription factor binding sites, and showed substantial overlap with genetic variants that were also associated with gene expression in the brain. Using tissue from three distinct regions of the adult brain (prefrontal cortex, striatum and cerebellum), we found that most fetal brain mQTLs were developmentally stable, although a subset was characterized by fetal-specific effects. Fetal brain mQTLs were enriched amongst risk loci identified in a recent large-scale genome-wide association study (GWAS) of schizophrenia, a severe psychiatric disorder with a hypothesized neurodevelopmental component. Finally, we found that mQTLs can be used to refine GWAS loci through the identification of discrete sites of variable fetal brain methylation associated with schizophrenia risk variants.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="WOS:000370275900014" class="col-sm-8"> <div class="title"><a href="https://doi.org/10.1093/ecco-jcc/jjv176" rel="external nofollow noopener" target="_blank">DNA Methylation Profiling in Inflammatory Bowel Disease Provides New Insights into Disease Pathogenesis</a></div> <div class="author"> Edel McDermott, Elizabeth J. Ryan, Miriam Tosetto, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'David Gibson, Joe Burrage, Denise Keegan, Kathryn Byrne, Eimear Crowe, Gillian Sexton, Kevin Malone, R. Alan Harris, Richard Kellermayer, Jonathan Mill, Garret Cullen, Glen A. Doherty, Hugh Mulcahy, Therese M. Murphy' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>JOURNAL OF CROHNS &amp; COLITIS</em>, Jan 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-doi="10.1093/ecco-jcc/jjv176" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/ecco-jcc/jjv176" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Background and Aims: Inflammatory bowel diseases (IBDs) are heterogeneous disorders with complex aetiology. Quantitative genetic studies suggest that only a small proportion of the disease variance observed in IBD is accounted for by genetic variation, indicating a potential role for differential epigenetic regulation in disease aetiology. The aim of this study was to assess genome-wide DNA methylation changes specifically associated with ulcerative colitis (UC), Crohn’s disease (CD) and IBD activity. Methods: DNA methylation was quantified in peripheral blood mononuclear cells (PBMCs) from 149 IBD cases (61 UC, 88 CD) and 39 controls using the Infinium HumanMethylation450 BeadChip. Technical and functional validation was performed using pyrosequencing and the real-time polymerase chain reaction. Cross-tissue replication of the top differentially methylated positions (DMPs) was tested in colonic mucosa tissue samples obtained from paediatric IBD cases and controls. Results: A total of 3196 probes were differentially methylated between CD cases and controls, while 1481 probes were differentially methylated between UC cases and controls. There was considerable (45%) overlap between UC and CD DMPs. The top-ranked IBD-associated PBMC differentially methylated region (promoter region of TRIM39-RPP2) was also significantly hypomethylated in colonic mucosa from paediatric UC patients. In addition, we confirmed TRAF6 hypermethylation using pyrosequencing and found reduced TRAF6 gene expression in PBMCs of IBD patients. Conclusions: Our data provide new insights into differential epigenetic regulation of genes and molecular pathways, which may contribute to the pathogenesis and activity of IBD.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2023 Complex Disease Epigenetics Group. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="/assets/js/common.js"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>